<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Renee Matthews" />

<meta name="date" content="2025-03-13" />

<title>sequencing QC</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.33/datatables.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/dt-ext-fixedcolumns-1.13.6/css/fixedColumns.dataTables.min.css" rel="stylesheet" />
<script src="site_libs/dt-ext-fixedcolumns-1.13.6/js/dataTables.fixedColumns.min.js"></script>
<link href="site_libs/dt-ext-scroller-1.13.6/css/scroller.dataTables.min.css" rel="stylesheet" />
<script src="site_libs/dt-ext-scroller-1.13.6/js/dataTables.scroller.min.js"></script>
<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">DXR_project</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/reneeisnowhere/DXR_project">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">sequencing QC</h1>
<h4 class="author">Renee Matthews</h4>
<h4 class="date">2025-03-13</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-04-17
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>DXR_project/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20250313code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20250313)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20250313code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20250313)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereDXRprojecttree749decb869c01565f2b0023f99565cda72a966actargetblank749decba">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/reneeisnowhere/DXR_project/tree/749decb869c01565f2b0023f99565cda72a966ac" target="_blank">749decb</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereDXRprojecttree749decb869c01565f2b0023f99565cda72a966actargetblank749decba"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/reneeisnowhere/DXR_project/tree/749decb869c01565f2b0023f99565cda72a966ac" target="_blank">749decb</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/figure/
    Ignored:    data/first_run_alignment_229andover.RDS
    Ignored:    data/first_run_alignment_specs.RDS
    Ignored:    data/multiqc_fastqc_notrim.txt
    Ignored:    data/multiqc_fastqc_trim.txt
    Ignored:    data/multiqc_genral_stats_notrim.txt
    Ignored:    data/multiqc_genral_stats_trim.txt
    Ignored:    data/prelim_DEG/
    Ignored:    data/sampleinfo.txt

Untracked files:
    Untracked:  DXR_project.Rproj
    Untracked:  analysis/prelim_DEG.Rmd
    Untracked:  multiqc_report_no_cut.html
    Untracked:  multiqc_report_trim.html

Unstaged changes:
    Deleted:    5FU_project.Rproj
    Modified:   analysis/index.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/QC_seq_analysis.Rmd</code>) and
HTML (<code>docs/QC_seq_analysis.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/749decb869c01565f2b0023f99565cda72a966ac/analysis/QC_seq_analysis.Rmd" target="_blank">749decb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-17
</td>
<td>
update table
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/DXR_project/f9af68ba51e3d203f4839ab39f26ef44ecfd044c/docs/QC_seq_analysis.html" target="_blank">f9af68b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/DXR_project/53be528d21140ebc2c5883f7a2ec5ba3151452ec/docs/QC_seq_analysis.html" target="_blank">53be528</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/3851faa5fb34135233f5b5ceaeb5896daefdd2b4/analysis/QC_seq_analysis.Rmd" target="_blank">3851faa</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-16
</td>
<td>
adding fragment counts
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/DXR_project/8e294dc6c2e975cf3b647616f41b03b5a7b72e0c/docs/QC_seq_analysis.html" target="_blank">8e294dc</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/efe5b3bc50f49b513e48002721ff78ef3fefe953/analysis/QC_seq_analysis.Rmd" target="_blank">efe5b3b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-16
</td>
<td>
update project name
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/DXR_project/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/QC_seq_analysis.html" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/7e17cd0c8236534dc276de6dd99473b83d60f141/analysis/QC_seq_analysis.Rmd" target="_blank">7e17cd0</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
<td>
adding updates
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/ccf2cb177c6fbebc6a0d6995b0ae51ba2b500b4b/analysis/QC_seq_analysis.Rmd" target="_blank">ccf2cb1</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-10
</td>
<td>
updates but broken
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/DXR_project/b69b2501b73adf057b40d98e39d89d15d6243e46/docs/QC_seq_analysis.html" target="_blank">b69b250</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-31
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/3073910e58c81c029f87e98277583c4dd6e5e519/analysis/QC_seq_analysis.Rmd" target="_blank">3073910</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-31
</td>
<td>
wflow_publish("analysis/QC_seq_analysis.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/DXR_project/e1f2574321ea7f60075b7087872dda4ec9237fb3/docs/QC_seq_analysis.html" target="_blank">e1f2574</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/fdf6809c54f9e048604fa5c29722720210f94702/analysis/QC_seq_analysis.Rmd" target="_blank">fdf6809</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-28
</td>
<td>
adding in new info
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/7f5695eb1d139344dffd239574aa9113a5aec78b/analysis/QC_seq_analysis.Rmd" target="_blank">7f5695e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-25
</td>
<td>
day of start
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/DXR_project/63d7a250d778a5c0d012d5550d88a6826b6f0375/docs/QC_seq_analysis.html" target="_blank">63d7a25</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/4428396e360070dd1edcd48c8971ecf0f21c9837/analysis/QC_seq_analysis.Rmd" target="_blank">4428396</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
<td>
updates
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/DXR_project/d73698d0e86d481cd5455c1cd6177aff97cb7804/docs/QC_seq_analysis.html" target="_blank">d73698d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d08dbb72d659a404057b5a167e258cfafb37492/analysis/QC_seq_analysis.Rmd" target="_blank">8d08dbb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-19
</td>
<td>
wflow_publish("analysis/QC_seq_analysis.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/DXR_project/343e631ab2f949ff383182eba5d3c2d400f3a4ec/docs/QC_seq_analysis.html" target="_blank">343e631</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-18
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/DXR_project/44d55e8b2cbdf6d6aadd885abe09afe10363508f/docs/QC_seq_analysis.html" target="_blank">44d55e8</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/DXR_project/f7c905588b8f2c897d01b1128f3a524609fea223/docs/QC_seq_analysis.html" target="_blank">f7c9055</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-03-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/DXR_project/614b3c814910a8aa65f93769216d612a6a44304b/docs/QC_seq_analysis.html" target="_blank">614b3c8</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-03-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/1cbf9d20277533df3a204f0dcc456a85f221a637/analysis/QC_seq_analysis.Rmd" target="_blank">1cbf9d2</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-03-17
</td>
<td>
adding in trimmed sequences
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/DXR_project/aaa7046a7e23e3212dd49c9d7326734f85135374/docs/QC_seq_analysis.html" target="_blank">aaa7046</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-03-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/5f1ac0eaee3cfcaca54aa02323db4398fe51bee1/analysis/QC_seq_analysis.Rmd" target="_blank">5f1ac0e</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-03-13
</td>
<td>
update
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<div id="loading-packages" class="section level5">
<h5>Loading Packages</h5>
<pre class="r"><code>library(tidyverse)
library(readr)
library(kableExtra)
library(DT)
library(readr)
library(edgeR)
library(ComplexHeatmap)
library(data.table)
library(genomation)
library(GenomicRanges)</code></pre>
</div>
<div id="data-loading" class="section level3">
<h3>data loading</h3>
<pre class="r"><code>sampleinfo &lt;- read_delim(&quot;data/sampleinfo.txt&quot;, delim = &quot;\t&quot;)

multiqc_gene_stats_notrim &lt;- read_delim(&quot;data/multiqc_genral_stats_notrim.txt&quot;,delim = &quot;\t&quot;)
multiqc_gene_stats_trim &lt;- read_delim(&quot;data/multiqc_genral_stats_trim.txt&quot;,delim = &quot;\t&quot;)

multiqc_fastqc_notrim &lt;- read_delim(&quot;data/multiqc_fastqc_notrim.txt&quot;,delim = &quot;\t&quot;)
multiqc_fastqc_trim &lt;- read_delim(&quot;data/multiqc_fastqc_trim.txt&quot;,delim = &quot;\t&quot;)</code></pre>
<div id="functions" class="section level6">
<h6>functions</h6>
<pre class="r"><code>drug_pal &lt;- c(&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#F1B72B&quot;, &quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;)
pca_plot &lt;-
  function(df,
           col_var = NULL,
           shape_var = NULL,
           title = &quot;&quot;) {
    ggplot(df) + geom_point(aes_string(
      x = &quot;PC1&quot;,
      y = &quot;PC2&quot;,
      color = col_var,
      shape = shape_var
    ),
    size = 5) +
      labs(title = title, x = &quot;PC 1&quot;, y = &quot;PC 2&quot;) +
      scale_color_manual(values = c(
        &quot;#8B006D&quot;,
        &quot;#DF707E&quot;,
        &quot;#F1B72B&quot;,
        &quot;#3386DD&quot;,
        &quot;#707031&quot;,
        &quot;#41B333&quot;
      ))
  }
pca_var_plot &lt;- function(pca) {
  # x: class == prcomp
  pca.var &lt;- pca$sdev ^ 2
  pca.prop &lt;- pca.var / sum(pca.var)
  var.plot &lt;-
    qplot(PC, prop, data = data.frame(PC = 1:length(pca.prop),
                                      prop = pca.prop)) +
    labs(title = &#39;Variance contributed by each PC&#39;,
         x = &#39;PC&#39;, y = &#39;Proportion of variance&#39;)
  plot(var.plot)
}

calc_pca &lt;- function(x) {
  # Performs principal components analysis with prcomp
  # x: a sample-by-gene numeric matrix
  prcomp(x, scale. = TRUE, retx = TRUE)
}

get_regr_pval &lt;- function(mod) {
  # Returns the p-value for the Fstatistic of a linear model
  # mod: class lm
  stopifnot(class(mod) == &quot;lm&quot;)
  fstat &lt;- summary(mod)$fstatistic
  pval &lt;- 1 - pf(fstat[1], fstat[2], fstat[3])
  return(pval)
}

plot_versus_pc &lt;- function(df, pc_num, fac) {
  # df: data.frame
  # pc_num: numeric, specific PC for plotting
  # fac: column name of df for plotting against PC
  pc_char &lt;- paste0(&quot;PC&quot;, pc_num)
  # Calculate F-statistic p-value for linear model
  pval &lt;- get_regr_pval(lm(df[, pc_char] ~ df[, fac]))
  if (is.numeric(df[, f])) {
    ggplot(df, aes_string(x = f, y = pc_char)) + geom_point() +
      geom_smooth(method = &quot;lm&quot;) + labs(title = sprintf(&quot;p-val: %.2f&quot;, pval))
  } else {
    ggplot(df, aes_string(x = f, y = pc_char)) + geom_boxplot() +
      labs(title = sprintf(&quot;p-val: %.2f&quot;, pval))
  }
}
x_axis_labels = function(labels, every_nth = 1, ...) {
  axis(side = 1,
       at = seq_along(labels),
       labels = F)
  text(
    x = (seq_along(labels))[seq_len(every_nth) == 1],
    y = par(&quot;usr&quot;)[3] - 0.075 * (par(&quot;usr&quot;)[4] - par(&quot;usr&quot;)[3]),
    labels = labels[seq_len(every_nth) == 1],
    xpd = TRUE,
    ...
  )
}</code></pre>
<div id="combining-the-dataframes-for-visualization"
class="section level8">
<p class="heading">Combining the dataframes for visualization</p>
<pre class="r"><code>combo_notrim_df &lt;- multiqc_fastqc_notrim %&gt;% 
 extract(., Sample, into = c(&quot;prefix&quot;,&quot;read&quot;), regex= &quot;(.+)_R(\\d+)&quot;, remove=FALSE) %&gt;% 
  mutate(read = paste0(&quot;R&quot;, read)) %&gt;% 
  left_join(., sampleinfo, by =c(&quot;prefix&quot;=&quot;library_ID&quot;)) %&gt;% 
  left_join(., multiqc_gene_stats_notrim, by = c(&quot;Sample&quot; = &quot;Sample&quot;)) %&gt;% 
  mutate(ind=factor(ind, levels = c(&quot;1&quot;,&quot;2&quot;,&quot;3&quot;,&quot;4&quot;,&quot;5&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels = c(&quot;VEH&quot;,&quot;5FU&quot;,&quot;DOX&quot;))) %&gt;% 
  mutate(other_time=factor(other_time, levels=c(&quot;24t&quot;,&quot;24r&quot;,&quot;144r&quot;)))

combo_trim_df &lt;- multiqc_fastqc_trim %&gt;% 
 extract(., Sample, into = c(&quot;prefix&quot;,&quot;read&quot;), regex= &quot;(.+)_R(\\d+)&quot;, remove=FALSE) %&gt;% 
  mutate(read = paste0(&quot;R&quot;, read)) %&gt;% 
  left_join(., sampleinfo, by =c(&quot;prefix&quot;=&quot;library_ID&quot;)) %&gt;% 
  left_join(., multiqc_gene_stats_trim, by = c(&quot;Sample&quot; = &quot;Sample&quot;)) %&gt;% 
  mutate(ind=factor(ind, levels = c(&quot;1&quot;,&quot;2&quot;,&quot;3&quot;,&quot;4&quot;,&quot;5&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels = c(&quot;VEH&quot;,&quot;5FU&quot;,&quot;DOX&quot;))) %&gt;% 
  mutate(other_time=factor(other_time, levels=c(&quot;24t&quot;,&quot;24r&quot;,&quot;144r&quot;)))</code></pre>
</div>
</div>
</div>
<div id="basic-stats" class="section level3">
<h3>Basic Stats</h3>
<pre class="r"><code>combo_notrim_df %&gt;% 
  dplyr::filter(read==&quot;R1&quot;) %&gt;% 
  group_by(trt,other_time,histone_mark) %&gt;% 
  tally() %&gt;% 
  ggplot(., aes(x = other_time, y= n))+
  geom_col(position=&quot;dodge&quot;,aes(fill=trt)) + 
  facet_wrap(~histone_mark)+
  theme(axis.text.x=element_text(angle=90))+
  ylab(&quot;number of samples&quot;)+
  ggtitle(&quot;Breakdown of samples by mark and trt-time&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/Basic%20stats-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Basic-stats-1">
Past versions of "Basic stats-1.png"
</button>
</p>
<div id="fig-no-spaces-Basic-stats-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/614b3c814910a8aa65f93769216d612a6a44304b/docs/figure/QC_seq_analysis.Rmd/Basic stats-1.png" target="_blank">614b3c8</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-03-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>combo_notrim_df %&gt;% 
  dplyr::filter(read==&quot;R1&quot;) %&gt;% 
  group_by(trt,other_time,histone_mark) %&gt;% 
  tally() %&gt;% 
  pivot_wider(., id_cols=c(trt,other_time), names_from = histone_mark, values_from = n) %&gt;% 
  kable(.,caption = (&quot;Sample counts&quot;)) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = FALSE) %&gt;%
  kable_styling(full_width = FALSE,font_size = 16) %&gt;%
  scroll_box(width = &quot;100%&quot;, height = &quot;500px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; overflow-x: scroll; width:100%; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; font-size: 16px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Sample counts
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
trt
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
other_time
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
H3K27ac
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
H3K27me3
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
H3K9me3
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
H3K36me3
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
VEH
</td>
<td style="text-align:left;">
24t
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
VEH
</td>
<td style="text-align:left;">
24r
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
VEH
</td>
<td style="text-align:left;">
144r
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
5FU
</td>
<td style="text-align:left;">
24t
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
5FU
</td>
<td style="text-align:left;">
24r
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
5FU
</td>
<td style="text-align:left;">
144r
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
DOX
</td>
<td style="text-align:left;">
24t
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
DOX
</td>
<td style="text-align:left;">
24r
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
DOX
</td>
<td style="text-align:left;">
144r
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
4
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="visualization-of-counts" class="section level3">
<h3>Visualization of counts</h3>
<pre class="r"><code>combo_notrim_df %&gt;% 
  dplyr::filter(read==&quot;R1&quot;) %&gt;% 
  ggplot(., aes(x = Sample, y= `Total Sequences`))+
  geom_col(aes(fill=histone_mark)) + 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  ylab(&quot;sequene count&quot;)+
  ggtitle(&quot;Read counts by sample and histone mark&quot;)+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/counts%20viz-1.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-counts-viz-1">
Past versions of "counts viz-1.png"
</button>
</p>
<div id="fig-no-spaces-counts-viz-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/d73698d0e86d481cd5455c1cd6177aff97cb7804/docs/figure/QC_seq_analysis.Rmd/counts viz-1.png" target="_blank">d73698d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/614b3c814910a8aa65f93769216d612a6a44304b/docs/figure/QC_seq_analysis.Rmd/counts viz-1.png" target="_blank">614b3c8</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-03-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/aaa7046a7e23e3212dd49c9d7326734f85135374/docs/figure/QC_seq_analysis.Rmd/counts viz-1.png" target="_blank">aaa7046</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-03-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>combo_trim_df %&gt;% 
  dplyr::filter(read==&quot;R1&quot;) %&gt;% 
  ggplot(., aes(x = Sample, y= `Total Sequences`))+
  geom_col(aes(fill=histone_mark)) + 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  ylab(&quot;sequene count&quot;)+
  ggtitle(&quot;Read counts by sample and histone mark trimmed adapters&quot;)+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/counts%20viz-2.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-counts-viz-2">
Past versions of "counts viz-2.png"
</button>
</p>
<div id="fig-no-spaces-counts-viz-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/63d7a250d778a5c0d012d5550d88a6826b6f0375/docs/figure/QC_seq_analysis.Rmd/counts viz-2.png" target="_blank">63d7a25</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/d73698d0e86d481cd5455c1cd6177aff97cb7804/docs/figure/QC_seq_analysis.Rmd/counts viz-2.png" target="_blank">d73698d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/614b3c814910a8aa65f93769216d612a6a44304b/docs/figure/QC_seq_analysis.Rmd/counts viz-2.png" target="_blank">614b3c8</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-03-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/aaa7046a7e23e3212dd49c9d7326734f85135374/docs/figure/QC_seq_analysis.Rmd/counts viz-2.png" target="_blank">aaa7046</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-03-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>combo_notrim_df %&gt;% 
  dplyr::filter(read==&quot;R1&quot;) %&gt;% 
   ggplot(., aes(x = histone_mark, y= `Total Sequences`))+
geom_boxplot(aes(fill=histone_mark)) + 
    geom_point(aes(color=ind))+
  facet_wrap(trt~other_time)+
  ylab(&quot;count&quot;)+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  ggtitle(&quot;Sequence count by trt and time no trim&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/counts%20viz-3.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-counts-viz-3">
Past versions of "counts viz-3.png"
</button>
</p>
<div id="fig-no-spaces-counts-viz-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/63d7a250d778a5c0d012d5550d88a6826b6f0375/docs/figure/QC_seq_analysis.Rmd/counts viz-3.png" target="_blank">63d7a25</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/614b3c814910a8aa65f93769216d612a6a44304b/docs/figure/QC_seq_analysis.Rmd/counts viz-3.png" target="_blank">614b3c8</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-03-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/aaa7046a7e23e3212dd49c9d7326734f85135374/docs/figure/QC_seq_analysis.Rmd/counts viz-3.png" target="_blank">aaa7046</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-03-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>combo_trim_df %&gt;% 
  dplyr::filter(read==&quot;R1&quot;) %&gt;% 
   ggplot(., aes(x = histone_mark, y= `Total Sequences`))+
geom_boxplot(aes(fill=histone_mark)) + 
    geom_point(aes(color=ind))+
  facet_wrap(trt~other_time)+
  ylab(&quot;count&quot;)+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  ggtitle(&quot;Sequence count by trt and time trimmed adapter&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/counts%20viz-4.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-counts-viz-4">
Past versions of "counts viz-4.png"
</button>
</p>
<div id="fig-no-spaces-counts-viz-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/63d7a250d778a5c0d012d5550d88a6826b6f0375/docs/figure/QC_seq_analysis.Rmd/counts viz-4.png" target="_blank">63d7a25</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="trim-information" class="section level3">
<h3>Trim information</h3>
<pre class="r"><code>combo_trim_df %&gt;% 
   ggplot(., aes(x = read, y= avg_sequence_length))+
geom_boxplot(aes(fill=read))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/trim%20info1-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-trim-info1-1">
Past versions of "trim info1-1.png"
</button>
</p>
<div id="fig-no-spaces-trim-info1-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/d73698d0e86d481cd5455c1cd6177aff97cb7804/docs/figure/QC_seq_analysis.Rmd/trim info1-1.png" target="_blank">d73698d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>combo_trim_df %&gt;% 
  ggplot(., aes(x = read, y= avg_sequence_length))+
geom_boxplot(aes(fill=histone_mark)) +
  ggtitle(&quot;Boxplot of trim read length across histone marks&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/trim%20info1-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-trim-info1-2">
Past versions of "trim info1-2.png"
</button>
</p>
<div id="fig-no-spaces-trim-info1-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/d73698d0e86d481cd5455c1cd6177aff97cb7804/docs/figure/QC_seq_analysis.Rmd/trim info1-2.png" target="_blank">d73698d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>combo_trim_df %&gt;% 
  datatable(., options = list(scrollX = TRUE, 
                              scrollY = &quot;400px&quot;,
                              scrollCollapse = TRUE,
                              fixedColumns = list(leftColumns =2),
                              fixedHeader= TRUE),
            extensions = c(&quot;FixedColumns&quot;,&quot;Scroller&quot;),
            class = &quot;display&quot;)</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-0c1484f359f6469957e0" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-0c1484f359f6469957e0">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","Scroller"],"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188"],["MCW_CT_ROA_100_R1_trim","MCW_CT_ROA_100_R2_trim","MCW_CT_ROA_101_R1_trim","MCW_CT_ROA_101_R2_trim","MCW_CT_ROA_102_R1_trim","MCW_CT_ROA_102_R2_trim","MCW_CT_ROA_107_R1_trim","MCW_CT_ROA_107_R2_trim","MCW_CT_ROA_109_R1_trim","MCW_CT_ROA_109_R2_trim","MCW_CT_ROA_112_R1_trim","MCW_CT_ROA_112_R2_trim","MCW_CT_ROA_113_R1_trim","MCW_CT_ROA_113_R2_trim","MCW_CT_ROA_114_R1_trim","MCW_CT_ROA_114_R2_trim","MCW_CT_ROA_115_R1_trim","MCW_CT_ROA_115_R2_trim","MCW_CT_ROA_116_R1_trim","MCW_CT_ROA_116_R2_trim","MCW_CT_ROA_117_R1_trim","MCW_CT_ROA_117_R2_trim","MCW_CT_ROA_119_R1_trim","MCW_CT_ROA_119_R2_trim","MCW_CT_ROA_121_R1_trim","MCW_CT_ROA_121_R2_trim","MCW_CT_ROA_124_R1_trim","MCW_CT_ROA_124_R2_trim","MCW_CT_ROA_125_R1_trim","MCW_CT_ROA_125_R2_trim","MCW_CT_ROA_127_R1_trim","MCW_CT_ROA_127_R2_trim","MCW_CT_ROA_130_R1_trim","MCW_CT_ROA_130_R2_trim","MCW_CT_ROA_138_R1_trim","MCW_CT_ROA_138_R2_trim","MCW_CT_ROA_139_R1_trim","MCW_CT_ROA_139_R2_trim","MCW_CT_ROA_140_R1_trim","MCW_CT_ROA_140_R2_trim","MCW_CT_ROA_141_R1_trim","MCW_CT_ROA_141_R2_trim","MCW_CT_ROA_142_R1_trim","MCW_CT_ROA_142_R2_trim","MCW_CT_ROA_145_R1_trim","MCW_CT_ROA_145_R2_trim","MCW_CT_ROA_146_R1_trim","MCW_CT_ROA_146_R2_trim","MCW_CT_ROA_147_R1_trim","MCW_CT_ROA_147_R2_trim","MCW_CT_ROA_148_R1_trim","MCW_CT_ROA_148_R2_trim","MCW_CT_ROA_149_R1_trim","MCW_CT_ROA_149_R2_trim","MCW_CT_ROA_156_R1_trim","MCW_CT_ROA_156_R2_trim","MCW_CT_ROA_157_R1_trim","MCW_CT_ROA_157_R2_trim","MCW_CT_ROA_158_R1_trim","MCW_CT_ROA_158_R2_trim","MCW_CT_ROA_159_R1_trim","MCW_CT_ROA_159_R2_trim","MCW_CT_ROA_163_R1_trim","MCW_CT_ROA_163_R2_trim","MCW_CT_ROA_167_R1_trim","MCW_CT_ROA_167_R2_trim","MCW_CT_ROA_171_R1_trim","MCW_CT_ROA_171_R2_trim","MCW_CT_ROA_172_R1_trim","MCW_CT_ROA_172_R2_trim","MCW_CT_ROA_173_R1_trim","MCW_CT_ROA_173_R2_trim","MCW_CT_ROA_174_R1_trim","MCW_CT_ROA_174_R2_trim","MCW_CT_ROA_175_R1_trim","MCW_CT_ROA_175_R2_trim","MCW_CT_ROA_177_R1_trim","MCW_CT_ROA_177_R2_trim","MCW_CT_ROA_180_R1_trim","MCW_CT_ROA_180_R2_trim","MCW_CT_ROA_181_R1_trim","MCW_CT_ROA_181_R2_trim","MCW_CT_ROA_183_R1_trim","MCW_CT_ROA_183_R2_trim","MCW_CT_ROA_185_R1_trim","MCW_CT_ROA_185_R2_trim","MCW_CT_ROA_189_R1_trim","MCW_CT_ROA_189_R2_trim","MCW_CT_ROA_192_R1_trim","MCW_CT_ROA_192_R2_trim","MCW_CT_ROA_194_R1_trim","MCW_CT_ROA_194_R2_trim","MCW_CT_ROA_195_R1_trim","MCW_CT_ROA_195_R2_trim","MCW_CT_ROA_202_R1_trim","MCW_CT_ROA_202_R2_trim","MCW_CT_ROA_206_R1_trim","MCW_CT_ROA_206_R2_trim","MCW_CT_ROA_207_R1_trim","MCW_CT_ROA_207_R2_trim","MCW_CT_ROA_208_R1_trim","MCW_CT_ROA_208_R2_trim","MCW_CT_ROA_209_R1_trim","MCW_CT_ROA_209_R2_trim","MCW_CT_ROA_210_R1_trim","MCW_CT_ROA_210_R2_trim","MCW_CT_ROA_211_R1_trim","MCW_CT_ROA_211_R2_trim","MCW_CT_ROA_213_R1_trim","MCW_CT_ROA_213_R2_trim","MCW_CT_ROA_215_R1_trim","MCW_CT_ROA_215_R2_trim","MCW_CT_ROA_217_R1_trim","MCW_CT_ROA_217_R2_trim","MCW_CT_ROA_218_R1_trim","MCW_CT_ROA_218_R2_trim","MCW_CT_ROA_221_R1_trim","MCW_CT_ROA_221_R2_trim","MCW_CT_ROA_223_R1_trim","MCW_CT_ROA_223_R2_trim","MCW_CT_ROA_224_R1_trim","MCW_CT_ROA_224_R2_trim","MCW_CT_ROA_226_R1_trim","MCW_CT_ROA_226_R2_trim","MCW_CT_ROA_227_R1_trim","MCW_CT_ROA_227_R2_trim","MCW_CT_ROA_228_R1_trim","MCW_CT_ROA_228_R2_trim","MCW_CT_ROA_229_R1_trim","MCW_CT_ROA_229_R2_trim","MCW_CT_ROA_230_R1_trim","MCW_CT_ROA_230_R2_trim","MCW_CT_ROA_231_R1_trim","MCW_CT_ROA_231_R2_trim","MCW_CT_ROA_232_R1_trim","MCW_CT_ROA_232_R2_trim","MCW_CT_ROA_233_R1_trim","MCW_CT_ROA_233_R2_trim","MCW_CT_ROA_236_R1_trim","MCW_CT_ROA_236_R2_trim","MCW_CT_ROA_238_R1_trim","MCW_CT_ROA_238_R2_trim","MCW_CT_ROA_239_R1_trim","MCW_CT_ROA_239_R2_trim","MCW_CT_ROA_243_R1_trim","MCW_CT_ROA_243_R2_trim","MCW_CT_ROA_245_R1_trim","MCW_CT_ROA_245_R2_trim","MCW_CT_ROA_246_R1_trim","MCW_CT_ROA_246_R2_trim","MCW_CT_ROA_247_R1_trim","MCW_CT_ROA_247_R2_trim","MCW_CT_ROA_249_R1_trim","MCW_CT_ROA_249_R2_trim","MCW_CT_ROA_254_R1_trim","MCW_CT_ROA_254_R2_trim","MCW_CT_ROA_78_R1_trim","MCW_CT_ROA_78_R2_trim","MCW_CT_ROA_79_R1_trim","MCW_CT_ROA_79_R2_trim","MCW_CT_ROA_80_R1_trim","MCW_CT_ROA_80_R2_trim","MCW_CT_ROA_81_R1_trim","MCW_CT_ROA_81_R2_trim","MCW_CT_ROA_83_R1_trim","MCW_CT_ROA_83_R2_trim","MCW_CT_ROA_84_R1_trim","MCW_CT_ROA_84_R2_trim","MCW_CT_ROA_85_R1_trim","MCW_CT_ROA_85_R2_trim","MCW_CT_ROA_86_R1_trim","MCW_CT_ROA_86_R2_trim","MCW_CT_ROA_87_R1_trim","MCW_CT_ROA_87_R2_trim","MCW_CT_ROA_88_R1_trim","MCW_CT_ROA_88_R2_trim","MCW_CT_ROA_89_R1_trim","MCW_CT_ROA_89_R2_trim","MCW_CT_ROA_94_R1_trim","MCW_CT_ROA_94_R2_trim","MCW_CT_ROA_95_R1_trim","MCW_CT_ROA_95_R2_trim","MCW_CT_ROA_96_R1_trim","MCW_CT_ROA_96_R2_trim","MCW_CT_ROA_97_R1_trim","MCW_CT_ROA_97_R2_trim","MCW_CT_ROA_99_R1_trim","MCW_CT_ROA_99_R2_trim"],["MCW_CT_ROA_100","MCW_CT_ROA_100","MCW_CT_ROA_101","MCW_CT_ROA_101","MCW_CT_ROA_102","MCW_CT_ROA_102","MCW_CT_ROA_107","MCW_CT_ROA_107","MCW_CT_ROA_109","MCW_CT_ROA_109","MCW_CT_ROA_112","MCW_CT_ROA_112","MCW_CT_ROA_113","MCW_CT_ROA_113","MCW_CT_ROA_114","MCW_CT_ROA_114","MCW_CT_ROA_115","MCW_CT_ROA_115","MCW_CT_ROA_116","MCW_CT_ROA_116","MCW_CT_ROA_117","MCW_CT_ROA_117","MCW_CT_ROA_119","MCW_CT_ROA_119","MCW_CT_ROA_121","MCW_CT_ROA_121","MCW_CT_ROA_124","MCW_CT_ROA_124","MCW_CT_ROA_125","MCW_CT_ROA_125","MCW_CT_ROA_127","MCW_CT_ROA_127","MCW_CT_ROA_130","MCW_CT_ROA_130","MCW_CT_ROA_138","MCW_CT_ROA_138","MCW_CT_ROA_139","MCW_CT_ROA_139","MCW_CT_ROA_140","MCW_CT_ROA_140","MCW_CT_ROA_141","MCW_CT_ROA_141","MCW_CT_ROA_142","MCW_CT_ROA_142","MCW_CT_ROA_145","MCW_CT_ROA_145","MCW_CT_ROA_146","MCW_CT_ROA_146","MCW_CT_ROA_147","MCW_CT_ROA_147","MCW_CT_ROA_148","MCW_CT_ROA_148","MCW_CT_ROA_149","MCW_CT_ROA_149","MCW_CT_ROA_156","MCW_CT_ROA_156","MCW_CT_ROA_157","MCW_CT_ROA_157","MCW_CT_ROA_158","MCW_CT_ROA_158","MCW_CT_ROA_159","MCW_CT_ROA_159","MCW_CT_ROA_163","MCW_CT_ROA_163","MCW_CT_ROA_167","MCW_CT_ROA_167","MCW_CT_ROA_171","MCW_CT_ROA_171","MCW_CT_ROA_172","MCW_CT_ROA_172","MCW_CT_ROA_173","MCW_CT_ROA_173","MCW_CT_ROA_174","MCW_CT_ROA_174","MCW_CT_ROA_175","MCW_CT_ROA_175","MCW_CT_ROA_177","MCW_CT_ROA_177","MCW_CT_ROA_180","MCW_CT_ROA_180","MCW_CT_ROA_181","MCW_CT_ROA_181","MCW_CT_ROA_183","MCW_CT_ROA_183","MCW_CT_ROA_185","MCW_CT_ROA_185","MCW_CT_ROA_189","MCW_CT_ROA_189","MCW_CT_ROA_192","MCW_CT_ROA_192","MCW_CT_ROA_194","MCW_CT_ROA_194","MCW_CT_ROA_195","MCW_CT_ROA_195","MCW_CT_ROA_202","MCW_CT_ROA_202","MCW_CT_ROA_206","MCW_CT_ROA_206","MCW_CT_ROA_207","MCW_CT_ROA_207","MCW_CT_ROA_208","MCW_CT_ROA_208","MCW_CT_ROA_209","MCW_CT_ROA_209","MCW_CT_ROA_210","MCW_CT_ROA_210","MCW_CT_ROA_211","MCW_CT_ROA_211","MCW_CT_ROA_213","MCW_CT_ROA_213","MCW_CT_ROA_215","MCW_CT_ROA_215","MCW_CT_ROA_217","MCW_CT_ROA_217","MCW_CT_ROA_218","MCW_CT_ROA_218","MCW_CT_ROA_221","MCW_CT_ROA_221","MCW_CT_ROA_223","MCW_CT_ROA_223","MCW_CT_ROA_224","MCW_CT_ROA_224","MCW_CT_ROA_226","MCW_CT_ROA_226","MCW_CT_ROA_227","MCW_CT_ROA_227","MCW_CT_ROA_228","MCW_CT_ROA_228","MCW_CT_ROA_229","MCW_CT_ROA_229","MCW_CT_ROA_230","MCW_CT_ROA_230","MCW_CT_ROA_231","MCW_CT_ROA_231","MCW_CT_ROA_232","MCW_CT_ROA_232","MCW_CT_ROA_233","MCW_CT_ROA_233","MCW_CT_ROA_236","MCW_CT_ROA_236","MCW_CT_ROA_238","MCW_CT_ROA_238","MCW_CT_ROA_239","MCW_CT_ROA_239","MCW_CT_ROA_243","MCW_CT_ROA_243","MCW_CT_ROA_245","MCW_CT_ROA_245","MCW_CT_ROA_246","MCW_CT_ROA_246","MCW_CT_ROA_247","MCW_CT_ROA_247","MCW_CT_ROA_249","MCW_CT_ROA_249","MCW_CT_ROA_254","MCW_CT_ROA_254","MCW_CT_ROA_78","MCW_CT_ROA_78","MCW_CT_ROA_79","MCW_CT_ROA_79","MCW_CT_ROA_80","MCW_CT_ROA_80","MCW_CT_ROA_81","MCW_CT_ROA_81","MCW_CT_ROA_83","MCW_CT_ROA_83","MCW_CT_ROA_84","MCW_CT_ROA_84","MCW_CT_ROA_85","MCW_CT_ROA_85","MCW_CT_ROA_86","MCW_CT_ROA_86","MCW_CT_ROA_87","MCW_CT_ROA_87","MCW_CT_ROA_88","MCW_CT_ROA_88","MCW_CT_ROA_89","MCW_CT_ROA_89","MCW_CT_ROA_94","MCW_CT_ROA_94","MCW_CT_ROA_95","MCW_CT_ROA_95","MCW_CT_ROA_96","MCW_CT_ROA_96","MCW_CT_ROA_97","MCW_CT_ROA_97","MCW_CT_ROA_99","MCW_CT_ROA_99"],["R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2","R1","R2"],["MCW_CT_ROA_100_R1_trim.fastq.gz","MCW_CT_ROA_100_R2_trim.fastq.gz","MCW_CT_ROA_101_R1_trim.fastq.gz","MCW_CT_ROA_101_R2_trim.fastq.gz","MCW_CT_ROA_102_R1_trim.fastq.gz","MCW_CT_ROA_102_R2_trim.fastq.gz","MCW_CT_ROA_107_R1_trim.fastq.gz","MCW_CT_ROA_107_R2_trim.fastq.gz","MCW_CT_ROA_109_R1_trim.fastq.gz","MCW_CT_ROA_109_R2_trim.fastq.gz","MCW_CT_ROA_112_R1_trim.fastq.gz","MCW_CT_ROA_112_R2_trim.fastq.gz","MCW_CT_ROA_113_R1_trim.fastq.gz","MCW_CT_ROA_113_R2_trim.fastq.gz","MCW_CT_ROA_114_R1_trim.fastq.gz","MCW_CT_ROA_114_R2_trim.fastq.gz","MCW_CT_ROA_115_R1_trim.fastq.gz","MCW_CT_ROA_115_R2_trim.fastq.gz","MCW_CT_ROA_116_R1_trim.fastq.gz","MCW_CT_ROA_116_R2_trim.fastq.gz","MCW_CT_ROA_117_R1_trim.fastq.gz","MCW_CT_ROA_117_R2_trim.fastq.gz","MCW_CT_ROA_119_R1_trim.fastq.gz","MCW_CT_ROA_119_R2_trim.fastq.gz","MCW_CT_ROA_121_R1_trim.fastq.gz","MCW_CT_ROA_121_R2_trim.fastq.gz","MCW_CT_ROA_124_R1_trim.fastq.gz","MCW_CT_ROA_124_R2_trim.fastq.gz","MCW_CT_ROA_125_R1_trim.fastq.gz","MCW_CT_ROA_125_R2_trim.fastq.gz","MCW_CT_ROA_127_R1_trim.fastq.gz","MCW_CT_ROA_127_R2_trim.fastq.gz","MCW_CT_ROA_130_R1_trim.fastq.gz","MCW_CT_ROA_130_R2_trim.fastq.gz","MCW_CT_ROA_138_R1_trim.fastq.gz","MCW_CT_ROA_138_R2_trim.fastq.gz","MCW_CT_ROA_139_R1_trim.fastq.gz","MCW_CT_ROA_139_R2_trim.fastq.gz","MCW_CT_ROA_140_R1_trim.fastq.gz","MCW_CT_ROA_140_R2_trim.fastq.gz","MCW_CT_ROA_141_R1_trim.fastq.gz","MCW_CT_ROA_141_R2_trim.fastq.gz","MCW_CT_ROA_142_R1_trim.fastq.gz","MCW_CT_ROA_142_R2_trim.fastq.gz","MCW_CT_ROA_145_R1_trim.fastq.gz","MCW_CT_ROA_145_R2_trim.fastq.gz","MCW_CT_ROA_146_R1_trim.fastq.gz","MCW_CT_ROA_146_R2_trim.fastq.gz","MCW_CT_ROA_147_R1_trim.fastq.gz","MCW_CT_ROA_147_R2_trim.fastq.gz","MCW_CT_ROA_148_R1_trim.fastq.gz","MCW_CT_ROA_148_R2_trim.fastq.gz","MCW_CT_ROA_149_R1_trim.fastq.gz","MCW_CT_ROA_149_R2_trim.fastq.gz","MCW_CT_ROA_156_R1_trim.fastq.gz","MCW_CT_ROA_156_R2_trim.fastq.gz","MCW_CT_ROA_157_R1_trim.fastq.gz","MCW_CT_ROA_157_R2_trim.fastq.gz","MCW_CT_ROA_158_R1_trim.fastq.gz","MCW_CT_ROA_158_R2_trim.fastq.gz","MCW_CT_ROA_159_R1_trim.fastq.gz","MCW_CT_ROA_159_R2_trim.fastq.gz","MCW_CT_ROA_163_R1_trim.fastq.gz","MCW_CT_ROA_163_R2_trim.fastq.gz","MCW_CT_ROA_167_R1_trim.fastq.gz","MCW_CT_ROA_167_R2_trim.fastq.gz","MCW_CT_ROA_171_R1_trim.fastq.gz","MCW_CT_ROA_171_R2_trim.fastq.gz","MCW_CT_ROA_172_R1_trim.fastq.gz","MCW_CT_ROA_172_R2_trim.fastq.gz","MCW_CT_ROA_173_R1_trim.fastq.gz","MCW_CT_ROA_173_R2_trim.fastq.gz","MCW_CT_ROA_174_R1_trim.fastq.gz","MCW_CT_ROA_174_R2_trim.fastq.gz","MCW_CT_ROA_175_R1_trim.fastq.gz","MCW_CT_ROA_175_R2_trim.fastq.gz","MCW_CT_ROA_177_R1_trim.fastq.gz","MCW_CT_ROA_177_R2_trim.fastq.gz","MCW_CT_ROA_180_R1_trim.fastq.gz","MCW_CT_ROA_180_R2_trim.fastq.gz","MCW_CT_ROA_181_R1_trim.fastq.gz","MCW_CT_ROA_181_R2_trim.fastq.gz","MCW_CT_ROA_183_R1_trim.fastq.gz","MCW_CT_ROA_183_R2_trim.fastq.gz","MCW_CT_ROA_185_R1_trim.fastq.gz","MCW_CT_ROA_185_R2_trim.fastq.gz","MCW_CT_ROA_189_R1_trim.fastq.gz","MCW_CT_ROA_189_R2_trim.fastq.gz","MCW_CT_ROA_192_R1_trim.fastq.gz","MCW_CT_ROA_192_R2_trim.fastq.gz","MCW_CT_ROA_194_R1_trim.fastq.gz","MCW_CT_ROA_194_R2_trim.fastq.gz","MCW_CT_ROA_195_R1_trim.fastq.gz","MCW_CT_ROA_195_R2_trim.fastq.gz","MCW_CT_ROA_202_R1_trim.fastq.gz","MCW_CT_ROA_202_R2_trim.fastq.gz","MCW_CT_ROA_206_R1_trim.fastq.gz","MCW_CT_ROA_206_R2_trim.fastq.gz","MCW_CT_ROA_207_R1_trim.fastq.gz","MCW_CT_ROA_207_R2_trim.fastq.gz","MCW_CT_ROA_208_R1_trim.fastq.gz","MCW_CT_ROA_208_R2_trim.fastq.gz","MCW_CT_ROA_209_R1_trim.fastq.gz","MCW_CT_ROA_209_R2_trim.fastq.gz","MCW_CT_ROA_210_R1_trim.fastq.gz","MCW_CT_ROA_210_R2_trim.fastq.gz","MCW_CT_ROA_211_R1_trim.fastq.gz","MCW_CT_ROA_211_R2_trim.fastq.gz","MCW_CT_ROA_213_R1_trim.fastq.gz","MCW_CT_ROA_213_R2_trim.fastq.gz","MCW_CT_ROA_215_R1_trim.fastq.gz","MCW_CT_ROA_215_R2_trim.fastq.gz","MCW_CT_ROA_217_R1_trim.fastq.gz","MCW_CT_ROA_217_R2_trim.fastq.gz","MCW_CT_ROA_218_R1_trim.fastq.gz","MCW_CT_ROA_218_R2_trim.fastq.gz","MCW_CT_ROA_221_R1_trim.fastq.gz","MCW_CT_ROA_221_R2_trim.fastq.gz","MCW_CT_ROA_223_R1_trim.fastq.gz","MCW_CT_ROA_223_R2_trim.fastq.gz","MCW_CT_ROA_224_R1_trim.fastq.gz","MCW_CT_ROA_224_R2_trim.fastq.gz","MCW_CT_ROA_226_R1_trim.fastq.gz","MCW_CT_ROA_226_R2_trim.fastq.gz","MCW_CT_ROA_227_R1_trim.fastq.gz","MCW_CT_ROA_227_R2_trim.fastq.gz","MCW_CT_ROA_228_R1_trim.fastq.gz","MCW_CT_ROA_228_R2_trim.fastq.gz","MCW_CT_ROA_229_R1_trim.fastq.gz","MCW_CT_ROA_229_R2_trim.fastq.gz","MCW_CT_ROA_230_R1_trim.fastq.gz","MCW_CT_ROA_230_R2_trim.fastq.gz","MCW_CT_ROA_231_R1_trim.fastq.gz","MCW_CT_ROA_231_R2_trim.fastq.gz","MCW_CT_ROA_232_R1_trim.fastq.gz","MCW_CT_ROA_232_R2_trim.fastq.gz","MCW_CT_ROA_233_R1_trim.fastq.gz","MCW_CT_ROA_233_R2_trim.fastq.gz","MCW_CT_ROA_236_R1_trim.fastq.gz","MCW_CT_ROA_236_R2_trim.fastq.gz","MCW_CT_ROA_238_R1_trim.fastq.gz","MCW_CT_ROA_238_R2_trim.fastq.gz","MCW_CT_ROA_239_R1_trim.fastq.gz","MCW_CT_ROA_239_R2_trim.fastq.gz","MCW_CT_ROA_243_R1_trim.fastq.gz","MCW_CT_ROA_243_R2_trim.fastq.gz","MCW_CT_ROA_245_R1_trim.fastq.gz","MCW_CT_ROA_245_R2_trim.fastq.gz","MCW_CT_ROA_246_R1_trim.fastq.gz","MCW_CT_ROA_246_R2_trim.fastq.gz","MCW_CT_ROA_247_R1_trim.fastq.gz","MCW_CT_ROA_247_R2_trim.fastq.gz","MCW_CT_ROA_249_R1_trim.fastq.gz","MCW_CT_ROA_249_R2_trim.fastq.gz","MCW_CT_ROA_254_R1_trim.fastq.gz","MCW_CT_ROA_254_R2_trim.fastq.gz","MCW_CT_ROA_78_R1_trim.fastq.gz","MCW_CT_ROA_78_R2_trim.fastq.gz","MCW_CT_ROA_79_R1_trim.fastq.gz","MCW_CT_ROA_79_R2_trim.fastq.gz","MCW_CT_ROA_80_R1_trim.fastq.gz","MCW_CT_ROA_80_R2_trim.fastq.gz","MCW_CT_ROA_81_R1_trim.fastq.gz","MCW_CT_ROA_81_R2_trim.fastq.gz","MCW_CT_ROA_83_R1_trim.fastq.gz","MCW_CT_ROA_83_R2_trim.fastq.gz","MCW_CT_ROA_84_R1_trim.fastq.gz","MCW_CT_ROA_84_R2_trim.fastq.gz","MCW_CT_ROA_85_R1_trim.fastq.gz","MCW_CT_ROA_85_R2_trim.fastq.gz","MCW_CT_ROA_86_R1_trim.fastq.gz","MCW_CT_ROA_86_R2_trim.fastq.gz","MCW_CT_ROA_87_R1_trim.fastq.gz","MCW_CT_ROA_87_R2_trim.fastq.gz","MCW_CT_ROA_88_R1_trim.fastq.gz","MCW_CT_ROA_88_R2_trim.fastq.gz","MCW_CT_ROA_89_R1_trim.fastq.gz","MCW_CT_ROA_89_R2_trim.fastq.gz","MCW_CT_ROA_94_R1_trim.fastq.gz","MCW_CT_ROA_94_R2_trim.fastq.gz","MCW_CT_ROA_95_R1_trim.fastq.gz","MCW_CT_ROA_95_R2_trim.fastq.gz","MCW_CT_ROA_96_R1_trim.fastq.gz","MCW_CT_ROA_96_R2_trim.fastq.gz","MCW_CT_ROA_97_R1_trim.fastq.gz","MCW_CT_ROA_97_R2_trim.fastq.gz","MCW_CT_ROA_99_R1_trim.fastq.gz","MCW_CT_ROA_99_R2_trim.fastq.gz"],["Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls","Conventional base calls"],["Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9","Sanger / Illumina 1.9"],[9646404,9646404,5971251,5971251,8994528,8994528,14094464,14094464,13689268,13689268,12803412,12803412,12211077,12211077,9122059,9122059,11702871,11702871,11303005,11303005,8839709,8839709,10678026,10678026,14847165,14847165,5862953,5862953,13609246,13609246,9069762,9069762,10036729,10036729,10446106,10446106,10762175,10762175,8720469,8720469,12193521,12193521,7383669,7383669,10381840,10381840,12450965,12450965,7627631,7627631,8242133,8242133,16699371,16699371,11950951,11950951,13986120,13986120,8616090,8616090,8134850,8134850,6954866,6954866,10254437,10254437,15049027,15049027,11124138,11124138,12820367,12820367,5638348,5638348,734837,734837,10266750,10266750,14437895,14437895,12982038,12982038,11071957,11071957,10371500,10371500,9018945,9018945,9617315,9617315,13338827,13338827,14457362,14457362,7751943,7751943,8455460,8455460,11006304,11006304,5856849,5856849,11297824,11297824,7400743,7400743,8938017,8938017,13499581,13499581,12237851,12237851,13521248,13521248,15777772,15777772,11674777,11674777,12465025,12465025,7822917,7822917,11120375,11120375,11811856,11811856,5703944,5703944,10611034,10611034,8628302,8628302,7583015,7583015,8665986,8665986,11430178,11430178,9761129,9761129,10348384,10348384,11412974,11412974,10920806,10920806,10824038,10824038,12674448,12674448,11631878,11631878,9780958,9780958,7120797,7120797,8514537,8514537,17230410,17230410,11073255,11073255,10304161,10304161,13873472,13873472,9460829,9460829,17871310,17871310,7129722,7129722,10992015,10992015,6136901,6136901,10476954,10476954,7398484,7398484,9244604,9244604,8197575,8197575,6738747,6738747,8097526,8097526],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],["1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","4-75","4-75","19-75","12-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","8-75","8-75","1-75","1-75","14-75","17-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","9-75","9-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","3-75","3-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75","1-75"],[50,50,50,50,50,50,55,55,55,54,53,53,52,51,49,49,47,47,52,52,49,49,51,51,49,49,52,51,56,55,52,52,50,50,49,49,53,53,51,51,48,48,50,50,52,52,55,55,49,49,50,49,57,57,56,56,57,57,51,50,50,49,53,53,47,47,49,48,47,47,49,48,47,46,56,56,51,51,48,48,48,48,49,49,50,50,49,49,49,48,48,47,50,50,49,49,51,50,50,50,50,50,52,51,48,48,51,51,50,50,51,51,50,50,51,51,53,53,52,52,47,47,52,52,46,46,47,47,50,50,48,48,47,47,49,49,47,47,48,47,50,50,50,50,51,51,52,51,49,49,51,50,50,50,50,50,49,49,56,56,55,54,51,50,52,51,49,49,56,55,51,51,50,50,51,51,50,49,51,50,50,49,50,49,50,50,50,50],[47.63909474909025,51.6693331273103,73.14949944775194,75.0361285239867,54.0162987402237,59.38630337084183,60.46420062679326,63.96276298642483,43.07091311951633,50.9514771211614,70.07233134101696,76.2783104018652,34.85892207044535,39.30779153946359,31.29345509047655,37.24194239522116,46.44576776257633,50.40568751486251,39.66637570305448,46.6495611829191,40.78445338035775,44.91816040723119,42.50806893782864,45.87610874681249,30.10603658988746,33.85906862312132,83.55314188005369,85.68334592129135,72.92556014679748,76.2717961126519,78.16482511683827,82.20030466390439,73.67799378928495,76.7532757959615,68.29459062513759,71.12014241057626,56.06909987218861,63.52562407359469,63.96564393615816,71.35811529839309,61.13050774495667,66.7726286828775,66.83738719604978,70.76859758250798,59.40272255577136,62.79607205912397,57.61164397016968,63.58590751812074,73.24509772608158,78.77926308550482,68.22828739684323,72.07128137741124,56.69568138983762,59.46477118967505,50.15743547973955,53.45074725810279,69.45809375312207,70.43955846902502,77.17846026031701,79.4613453365434,63.92285959574453,67.81219363043918,41.83621591183776,50.35289912166117,37.36075658758444,41.88658627872516,46.87291290617458,49.77030225073058,27.34176373788544,30.42705306414932,46.11331686528144,50.67989315879179,66.44195693710223,67.89464426512491,71.30649950298934,71.49929548025159,37.21526998156284,41.7171421745177,38.30068055180777,41.63128427256419,45.17376542437089,51.47525171125402,36.30695701398286,38.71290255551917,33.28850990481548,36.36352412527602,37.75303594882227,43.61349389278716,30.42447059214892,38.74774606752096,37.58845140470351,40.3526771176922,31.19839651773482,35.86956991625236,29.90943087000396,31.41063604254932,40.52334684044821,46.19523372952927,27.52118974982864,32.10281147137678,41.01129537522176,43.78244621585926,25.24533901862866,29.85357821813432,28.22621173928685,33.48212114958532,34.32866236559902,39.72254851081076,81.34309971400867,81.27833807688926,62.61031319668764,63.25852448918914,67.00784398157424,69.18241161852052,67.69367456830504,67.96638899679297,65.04858009285995,66.1426420888218,56.3472682652025,57.79796700932619,78.73820527475662,80.39181047838548,69.7730515329085,69.93614319600229,76.61750144058831,80.58425761739608,82.1814407542069,83.10551069405311,75.25883956388957,77.8613542143171,72.94732597576554,74.19527677091455,31.5786592448624,40.00209802605045,70.50569518914375,71.39106448749054,68.86897676915822,71.99506440825155,83.29952437954769,84.86354579313988,50.19968564134154,51.53974227429497,75.02821099889972,76.69701335583086,70.6329276119875,71.35948931400321,57.64272549473009,61.66009789258546,60.20118300994973,63.78547970302706,71.69368019648081,76.26437000619097,69.45400687410961,71.44972136867493,70.78755747507552,72.211206517888,49.19865135210898,52.77734625028461,42.55165031344755,48.29103714773905,34.63458769831272,40.09943351556555,66.38003429029841,68.83994861302152,56.5186140320041,62.863790968513,43.59341067877906,46.29364807104758,59.36155711163623,64.52799027153502,69.49501663516327,72.15468121033656,53.5134035153928,56.84928923499481,70.88794514774285,75.22701204935682,57.1216579295816,61.10540734799302,74.06816859074991,75.33823812667868,51.07547825077361,58.38179530783275,50.9284883557616,55.1439986936786,60.01222624404389,62.34438905755707,65.39247735207918,68.28401563102402],[70.99450365130882,70.94970104922,71.6077853702683,71.57006932048243,71.37048947982596,71.34909513873323,67.63537258316457,67.59850094334911,67.27552444732618,67.29941820117774,69.81030447196419,69.79794354817294,67.3891840989947,67.40075588746186,69.97409488362221,69.94473067977306,70.15564761843483,70.14765641695956,70.64911198393702,70.63798609307879,70.52375547656602,70.49129943078442,69.56629137258142,69.55362461188987,69.5789345642754,69.56648073891547,72.05675672310524,72.00678088328526,67.46573719073048,67.43451077304356,71.65190133985875,71.57907671667679,71.91457176934836,71.87581551718692,70.86855446421853,70.83251692065924,68.98765258881221,68.97478455795414,72.12520955008269,72.07273003321266,70.25664055525881,70.24672496155951,72.48310210005351,72.43550950076445,69.24075115779091,69.20096206452806,65.71041859004502,65.71439627370248,71.83580013243954,71.80596963854177,71.78001932266805,71.7426006107885,64.83038492886948,64.80244064282421,61.04070479412057,61.09720506761344,63.5770330870892,63.51813955550217,70.84033964362025,70.81559454462523,70.32276981136714,70.2977670147575,69.95278845056109,69.93736615486193,67.08846892325732,67.1785569505181,68.97556180874683,68.97502702334178,70.10586582079438,70.10302910661483,70.7763869786255,70.75954861510594,71.7617798688552,71.75553158478334,55.45523973343748,55.48629423940275,68.20333484306134,68.21989704629021,68.97772161385022,68.98681795372525,69.68505052904636,69.71416413971366,67.23826040870642,67.2960704236839,69.29508711372512,69.30472583522152,69.79594331709529,69.82365653632436,70.15769078999701,70.17179451853247,69.16238841691252,69.13069027733847,62.89937514188273,62.9960468583411,66.73800284651216,66.7726021721264,69.70043829667458,69.75020140832078,69.60418465635694,69.59359218135353,70.19657156945655,70.2076461250751,68.20027201698309,68.20387324143127,70.70150307881249,70.68481556513989,66.9108794489874,67.02787542247906,62.83196256239361,62.83632017912259,62.3978257293703,62.40976949302618,62.66981006486975,62.74734388423317,61.25034878181786,61.20876331588516,62.51406635004678,62.54887129749887,59.37218561535175,59.43218212558739,71.82758758657417,71.8154026177192,64.97572734732417,64.94042116385464,71.83378971094805,71.8378568109872,71.43267710903193,71.41957634927692,67.46717124834394,67.53056356242003,69.40211202621327,69.4040866905215,70.44186197706321,70.44833723789284,68.69065908945618,68.69845554793189,69.82852078069125,69.81191666481485,71.22680542383981,71.2217691211744,63.76183701725796,63.75337946485171,66.42769991415034,66.50265846570754,68.49809876670275,68.50728197167865,67.06030577497972,67.12089831909311,71.33691250301393,71.31655469334838,70.19097707180217,70.19700670863295,68.86341685548594,68.88315807101922,68.90091432180976,68.9019695407691,66.28385454194397,66.33965041199539,61.02700458085443,61.09410640837914,62.79185984608861,62.88535539008178,69.62583853260833,69.59111916050225,69.6045617131746,69.59570394491011,69.51590965231482,69.5132916999134,65.59186483811203,65.62366261902457,70.86788292727262,70.84767680983914,70.51154115055338,70.47341420112691,70.83638305392249,70.81666398072903,71.93679307936257,71.89850886049514,70.52853152078183,70.5056143934352,71.7617741116872,71.73550862751937,69.64147299658741,69.61672323827473,71.42908882022132,71.40740585749843,70.87750727814891,70.81753636851552],["pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass"],["pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass"],["fail","warn","fail","warn","fail","warn","fail","warn","fail","warn","fail","warn","fail","warn","fail","warn","fail","warn","fail","warn","fail","warn","fail","warn","fail","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","warn","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","warn","pass","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","warn","fail","warn","fail","warn","fail","warn","fail","warn","fail","warn","fail","warn","fail","warn","fail","warn","fail","warn","fail","warn","fail","warn","fail","warn","fail","warn","fail","warn","fail","warn"],["pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass"],["fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail"],["warn","warn","warn","warn","warn","warn","warn","pass","warn","warn","warn","pass","fail","warn","warn","warn","warn","warn","fail","fail","warn","warn","fail","fail","warn","fail","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","warn","warn","warn","warn","warn","warn","warn","pass","pass","pass","fail","fail","warn","warn","warn","warn","warn","warn","fail","warn","warn","warn","warn","warn","warn","warn","fail","fail","warn","warn","fail","fail","warn","warn","warn","fail","fail","fail","warn","warn","fail","fail","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","fail","warn","pass","pass","warn","warn","warn","warn","pass","pass","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","pass","pass","pass","pass","pass","pass","pass","pass","fail","fail","pass","pass","pass","pass","warn","pass","warn","warn","pass","pass","warn","pass","warn","warn","pass","pass","warn","warn","pass","pass","warn","pass","warn","warn","warn","pass","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","fail","warn"],["pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass"],["warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn"],["fail","warn","pass","pass","warn","warn","warn","warn","fail","warn","pass","pass","fail","fail","fail","fail","fail","warn","fail","fail","fail","fail","fail","fail","fail","fail","pass","pass","pass","pass","pass","pass","pass","pass","warn","pass","warn","warn","warn","pass","warn","warn","warn","pass","warn","warn","warn","warn","pass","pass","warn","pass","warn","warn","warn","warn","warn","pass","pass","pass","warn","warn","fail","warn","fail","fail","fail","fail","fail","fail","fail","warn","warn","warn","pass","pass","fail","fail","fail","fail","fail","warn","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","fail","pass","pass","warn","warn","warn","warn","warn","warn","warn","warn","warn","warn","pass","pass","warn","warn","pass","pass","pass","pass","pass","pass","pass","pass","fail","fail","pass","pass","warn","pass","pass","pass","warn","warn","pass","pass","pass","pass","warn","warn","warn","warn","pass","pass","warn","pass","pass","pass","fail","warn","fail","fail","fail","fail","warn","warn","warn","warn","fail","fail","warn","warn","warn","pass","warn","warn","pass","pass","warn","warn","pass","pass","warn","warn","warn","warn","warn","warn","warn","warn"],["pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","warn","warn","pass","pass","pass","pass","pass","pass","warn","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass"],["pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass","pass"],["H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac"],["Ind3_17-3_VEH_24+24","Ind3_17-3_VEH_24+24","Ind3_17-3_5FU_24+144","Ind3_17-3_5FU_24+144","Ind3_17-3_DOX_24+144","Ind3_17-3_DOX_24+144","Ind4_90-1_5FU_24+24","Ind4_90-1_5FU_24+24","Ind4_90-1_VEH_24+24","Ind4_90-1_VEH_24+24","Ind4_90-1_VEH_24+144","Ind4_90-1_VEH_24+144","Ind5_84-1_5FU_24","Ind5_84-1_5FU_24","Ind5_84-1_DOX_24","Ind5_84-1_DOX_24","Ind5_84-1_VEH_24","Ind5_84-1_VEH_24","Ind5_84-1_5FU_24+24","Ind5_84-1_5FU_24+24","Ind5_84-1_DOX_24+24","Ind5_84-1_DOX_24+24","Ind5_84-1_5FU_24+144","Ind5_84-1_5FU_24+144","Ind5_84-1_VEH_24+144","Ind5_84-1_VEH_24+144","Ind1_78-1_VEH_24","Ind1_78-1_VEH_24","Ind1_78-1_5FU_24+24","Ind1_78-1_5FU_24+24","Ind1_78-1_VEH_24+24","Ind1_78-1_VEH_24+24","Ind1_78-1_VEH_24+144","Ind1_78-1_VEH_24+144","Ind2_75-1_DOX_24+144","Ind2_75-1_DOX_24+144","Ind2_75-1_VEH_24+144","Ind2_75-1_VEH_24+144","Ind3_17-3_5FU_24","Ind3_17-3_5FU_24","Ind3_17-3_DOX_24","Ind3_17-3_DOX_24","Ind3_17-3_VEH_24","Ind3_17-3_VEH_24","Ind3_17-3_VEH_24+24","Ind3_17-3_VEH_24+24","Ind3_17-3_5FU_24+144","Ind3_17-3_5FU_24+144","Ind3_17-3_DOX_24+144","Ind3_17-3_DOX_24+144","Ind3_17-3_VEH_24+144","Ind3_17-3_VEH_24+144","Ind4_90-1_5FU_24","Ind4_90-1_5FU_24","Ind4_90-1_DOX_24+144","Ind4_90-1_DOX_24+144","Ind4_90-1_VEH_24+144","Ind4_90-1_VEH_24+144","Ind5_84-1_5FU_24","Ind5_84-1_5FU_24","Ind5_84-1_DOX_24","Ind5_84-1_DOX_24","Ind5_84-1_VEH_24+24","Ind5_84-1_VEH_24+24","Ind1_78-1_5FU_24","Ind1_78-1_5FU_24","Ind1_78-1_DOX_24+24","Ind1_78-1_DOX_24+24","Ind1_78-1_VEH_24+24","Ind1_78-1_VEH_24+24","Ind1_78-1_5FU_24+144","Ind1_78-1_5FU_24+144","Ind1_78-1_DOX_24+144","Ind1_78-1_DOX_24+144","Ind1_78-1_VEH_24+144","Ind1_78-1_VEH_24+144","Ind2_75-1_DOX_24","Ind2_75-1_DOX_24","Ind2_75-1_DOX_24+24","Ind2_75-1_DOX_24+24","Ind2_75-1_VEH_24+24","Ind2_75-1_VEH_24+24","Ind2_75-1_DOX_24+144","Ind2_75-1_DOX_24+144","Ind3_17-3_5FU_24","Ind3_17-3_5FU_24","Ind3_17-3_DOX_24+24","Ind3_17-3_DOX_24+24","Ind3_17-3_DOX_24+144","Ind3_17-3_DOX_24+144","Ind4_90-1_5FU_24","Ind4_90-1_5FU_24","Ind4_90-1_DOX_24","Ind4_90-1_DOX_24","Ind4_90-1_VEH_24+144","Ind4_90-1_VEH_24+144","Ind5_84-1_5FU_24+24","Ind5_84-1_5FU_24+24","Ind5_84-1_DOX_24+24","Ind5_84-1_DOX_24+24","Ind5_84-1_VEH_24+24","Ind5_84-1_VEH_24+24","Ind5_84-1_5FU_24+144","Ind5_84-1_5FU_24+144","Ind5_84-1_DOX_24+144","Ind5_84-1_DOX_24+144","Ind5_84-1_VEH_24+144","Ind5_84-1_VEH_24+144","Ind1_78-1_DOX_24","Ind1_78-1_DOX_24","Ind1_78-1_5FU_24+24","Ind1_78-1_5FU_24+24","Ind1_78-1_VEH_24+24","Ind1_78-1_VEH_24+24","Ind1_78-1_5FU_24+144","Ind1_78-1_5FU_24+144","Ind2_75-1_5FU_24","Ind2_75-1_5FU_24","Ind2_75-1_VEH_24","Ind2_75-1_VEH_24","Ind2_75-1_5FU_24+24","Ind2_75-1_5FU_24+24","Ind2_75-1_VEH_24+24","Ind2_75-1_VEH_24+24","Ind2_75-1_5FU_24+144","Ind2_75-1_5FU_24+144","Ind2_75-1_DOX_24+144","Ind2_75-1_DOX_24+144","Ind2_75-1_VEH_24+144","Ind2_75-1_VEH_24+144","Ind3_17-3_5FU_24","Ind3_17-3_5FU_24","Ind3_17-3_DOX_24","Ind3_17-3_DOX_24","Ind3_17-3_VEH_24","Ind3_17-3_VEH_24","Ind3_17-3_5FU_24+24","Ind3_17-3_5FU_24+24","Ind3_17-3_5FU_24+144","Ind3_17-3_5FU_24+144","Ind3_17-3_VEH_24+144","Ind3_17-3_VEH_24+144","Ind4_90-1_5FU_24","Ind4_90-1_5FU_24","Ind4_90-1_DOX_24+24","Ind4_90-1_DOX_24+24","Ind4_90-1_5FU_24+144","Ind4_90-1_5FU_24+144","Ind4_90-1_DOX_24+144","Ind4_90-1_DOX_24+144","Ind4_90-1_VEH_24+144","Ind4_90-1_VEH_24+144","Ind5_84-1_DOX_24","Ind5_84-1_DOX_24","Ind5_84-1_5FU_24+144","Ind5_84-1_5FU_24+144","Ind1_78-1_DOX_24","Ind1_78-1_DOX_24","Ind1_78-1_VEH_24","Ind1_78-1_VEH_24","Ind1_78-1_5FU_24+24","Ind1_78-1_5FU_24+24","Ind1_78-1_DOX_24+24","Ind1_78-1_DOX_24+24","Ind1_78-1_5FU_24+144","Ind1_78-1_5FU_24+144","Ind1_78-1_DOX_24+144","Ind1_78-1_DOX_24+144","Ind1_78-1_VEH_24+144","Ind1_78-1_VEH_24+144","Ind2_75-1_5FU_24","Ind2_75-1_5FU_24","Ind2_75-1_DOX_24","Ind2_75-1_DOX_24","Ind2_75-1_VEH_24","Ind2_75-1_VEH_24","Ind2_75-1_5FU_24+24","Ind2_75-1_5FU_24+24","Ind2_75-1_VEH_24+144","Ind2_75-1_VEH_24+144","Ind3_17-3_5FU_24","Ind3_17-3_5FU_24","Ind3_17-3_DOX_24","Ind3_17-3_DOX_24","Ind3_17-3_VEH_24","Ind3_17-3_VEH_24","Ind3_17-3_DOX_24+24","Ind3_17-3_DOX_24+24"],["Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind4_90-1","Ind4_90-1","Ind4_90-1","Ind4_90-1","Ind4_90-1","Ind4_90-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind4_90-1","Ind4_90-1","Ind4_90-1","Ind4_90-1","Ind4_90-1","Ind4_90-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind4_90-1","Ind4_90-1","Ind4_90-1","Ind4_90-1","Ind4_90-1","Ind4_90-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind4_90-1","Ind4_90-1","Ind4_90-1","Ind4_90-1","Ind4_90-1","Ind4_90-1","Ind4_90-1","Ind4_90-1","Ind4_90-1","Ind4_90-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind5_84-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind1_78-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind2_75-1","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3","Ind3_17-3"],["3","3","3","3","3","3","4","4","4","4","4","4","5","5","5","5","5","5","5","5","5","5","5","5","5","5","1","1","1","1","1","1","1","1","2","2","2","2","3","3","3","3","3","3","3","3","3","3","3","3","3","3","4","4","4","4","4","4","5","5","5","5","5","5","1","1","1","1","1","1","1","1","1","1","1","1","2","2","2","2","2","2","2","2","3","3","3","3","3","3","4","4","4","4","4","4","5","5","5","5","5","5","5","5","5","5","5","5","1","1","1","1","1","1","1","1","2","2","2","2","2","2","2","2","2","2","2","2","2","2","3","3","3","3","3","3","3","3","3","3","3","3","4","4","4","4","4","4","4","4","4","4","5","5","5","5","1","1","1","1","1","1","1","1","1","1","1","1","1","1","2","2","2","2","2","2","2","2","2","2","3","3","3","3","3","3","3","3"],["VEH","VEH","5FU","5FU","DOX","DOX","5FU","5FU","VEH","VEH","VEH","VEH","5FU","5FU","DOX","DOX","VEH","VEH","5FU","5FU","DOX","DOX","5FU","5FU","VEH","VEH","VEH","VEH","5FU","5FU","VEH","VEH","VEH","VEH","DOX","DOX","VEH","VEH","5FU","5FU","DOX","DOX","VEH","VEH","VEH","VEH","5FU","5FU","DOX","DOX","VEH","VEH","5FU","5FU","DOX","DOX","VEH","VEH","5FU","5FU","DOX","DOX","VEH","VEH","5FU","5FU","DOX","DOX","VEH","VEH","5FU","5FU","DOX","DOX","VEH","VEH","DOX","DOX","DOX","DOX","VEH","VEH","DOX","DOX","5FU","5FU","DOX","DOX","DOX","DOX","5FU","5FU","DOX","DOX","VEH","VEH","5FU","5FU","DOX","DOX","VEH","VEH","5FU","5FU","DOX","DOX","VEH","VEH","DOX","DOX","5FU","5FU","VEH","VEH","5FU","5FU","5FU","5FU","VEH","VEH","5FU","5FU","VEH","VEH","5FU","5FU","DOX","DOX","VEH","VEH","5FU","5FU","DOX","DOX","VEH","VEH","5FU","5FU","5FU","5FU","VEH","VEH","5FU","5FU","DOX","DOX","5FU","5FU","DOX","DOX","VEH","VEH","DOX","DOX","5FU","5FU","DOX","DOX","VEH","VEH","5FU","5FU","DOX","DOX","5FU","5FU","DOX","DOX","VEH","VEH","5FU","5FU","DOX","DOX","VEH","VEH","5FU","5FU","VEH","VEH","5FU","5FU","DOX","DOX","VEH","VEH","DOX","DOX"],["24+24","24+24","24+144","24+144","24+144","24+144","24+24","24+24","24+24","24+24","24+144","24+144","24t","24t","24t","24t","24t","24t","24+24","24+24","24+24","24+24","24+144","24+144","24+144","24+144","24t","24t","24+24","24+24","24+24","24+24","24+144","24+144","24+144","24+144","24+144","24+144","24t","24t","24t","24t","24t","24t","24+24","24+24","24+144","24+144","24+144","24+144","24+144","24+144","24t","24t","24+144","24+144","24+144","24+144","24t","24t","24t","24t","24+24","24+24","24t","24t","24+24","24+24","24+24","24+24","24+144","24+144","24+144","24+144","24+144","24+144","24t","24t","24+24","24+24","24+24","24+24","24+144","24+144","24t","24t","24+24","24+24","24+144","24+144","24t","24t","24t","24t","24+144","24+144","24+24","24+24","24+24","24+24","24+24","24+24","24+144","24+144","24+144","24+144","24+144","24+144","24t","24t","24+24","24+24","24+24","24+24","24+144","24+144","24t","24t","24t","24t","24+24","24+24","24+24","24+24","24+144","24+144","24+144","24+144","24+144","24+144","24t","24t","24t","24t","24t","24t","24+24","24+24","24+144","24+144","24+144","24+144","24t","24t","24+24","24+24","24+144","24+144","24+144","24+144","24+144","24+144","24t","24t","24+144","24+144","24t","24t","24+24","24+24","24+24","24+24","24+144","24+144","24+144","24+144","24+144","24+144","24+144","24+144","24t","24t","24t","24t","24t","24t","24+24","24+24","24+144","24+144","24t","24t","24t","24t","24t","24t","24+24","24+24"],["24r","24r","144r","144r","144r","144r","24r","24r","24r","24r","144r","144r","24t","24t","24t","24t","24t","24t","24r","24r","24r","24r","144r","144r","144r","144r","24t","24t","24r","24r","24r","24r","144r","144r","144r","144r","144r","144r","24t","24t","24t","24t","24t","24t","24r","24r","144r","144r","144r","144r","144r","144r","24t","24t","144r","144r","144r","144r","24t","24t","24t","24t","24r","24r","24t","24t","24r","24r","24r","24r","144r","144r","144r","144r","144r","144r","24t","24t","24r","24r","24r","24r","144r","144r","24t","24t","24r","24r","144r","144r","24t","24t","24t","24t","144r","144r","24r","24r","24r","24r","24r","24r","144r","144r","144r","144r","144r","144r","24t","24t","24r","24r","24r","24r","144r","144r","24t","24t","24t","24t","24r","24r","24r","24r","144r","144r","144r","144r","144r","144r","24t","24t","24t","24t","24t","24t","24r","24r","144r","144r","144r","144r","24t","24t","24r","24r","144r","144r","144r","144r","144r","144r","24t","24t","144r","144r","24t","24t","24t","24t","24r","24r","24r","24r","144r","144r","144r","144r","144r","144r","24t","24t","24t","24t","24t","24t","24r","24r","144r","144r","24t","24t","24t","24t","24t","24t","24r","24r"],[52.36090525090975,48.3306668726897,26.85050055224806,24.9638714760133,45.9837012597763,40.61369662915817,39.53579937320674,36.03723701357517,56.92908688048367,49.0485228788386,29.92766865898304,23.7216895981348,65.14107792955465,60.69220846053641,68.70654490952344,62.75805760477884,53.55423223742367,49.59431248513749,60.33362429694552,53.3504388170809,59.21554661964225,55.08183959276881,57.49193106217136,54.12389125318751,69.89396341011253,66.14093137687868,16.44685811994631,14.31665407870865,27.07443985320252,23.7282038873481,21.83517488316173,17.79969533609561,26.32200621071505,23.2467242040385,31.70540937486241,28.87985758942374,43.93090012781139,36.47437592640531,36.03435606384184,28.64188470160691,38.86949225504333,33.2273713171225,33.16261280395022,29.23140241749202,40.59727744422864,37.20392794087603,42.38835602983032,36.41409248187926,26.75490227391841,21.22073691449518,31.77171260315677,27.92871862258876,43.30431861016238,40.53522881032495,49.84256452026045,46.54925274189721,30.54190624687793,29.56044153097498,22.82153973968299,20.5386546634566,36.07714040425547,32.18780636956082,58.16378408816224,49.64710087833883,62.63924341241556,58.11341372127484,53.12708709382542,50.22969774926942,72.65823626211457,69.57294693585068,53.88668313471856,49.32010684120821,33.55804306289777,32.10535573487509,28.69350049701066,28.50070451974841,62.78473001843716,58.2828578254823,61.69931944819223,58.36871572743581,54.82623457562911,48.52474828874598,63.69304298601714,61.28709744448083,66.71149009518453,63.63647587472398,62.24696405117773,56.38650610721284,69.57552940785108,61.25225393247904,62.41154859529649,59.6473228823078,68.80160348226518,64.13043008374764,70.09056912999604,68.58936395745067,59.47665315955179,53.80476627047073,72.47881025017136,67.89718852862322,58.98870462477824,56.21755378414074,74.75466098137133,70.14642178186568,71.77378826071315,66.51787885041469,65.67133763440097,60.27745148918924,18.65690028599133,18.72166192311074,37.38968680331236,36.74147551081086,32.99215601842576,30.81758838147948,32.30632543169496,32.03361100320703,34.95141990714005,33.8573579111782,43.6527317347975,42.20203299067381,21.26179472524338,19.60818952161452,30.2269484670915,30.06385680399771,23.38249855941169,19.41574238260392,17.8185592457931,16.89448930594689,24.74116043611043,22.1386457856829,27.05267402423446,25.80472322908546,68.42134075513761,59.99790197394955,29.49430481085625,28.60893551250946,31.13102323084178,28.00493559174845,16.70047562045231,15.13645420686012,49.80031435865846,48.46025772570503,24.97178900110028,23.30298664416914,29.3670723880125,28.64051068599679,42.35727450526991,38.33990210741454,39.79881699005027,36.21452029697294,28.30631980351919,23.73562999380903,30.54599312589039,28.55027863132507,29.21244252492448,27.788793482112,50.80134864789102,47.22265374971539,57.44834968655245,51.70896285226095,65.36541230168729,59.90056648443445,33.61996570970159,31.16005138697848,43.4813859679959,37.136209031487,56.40658932122094,53.70635192895242,40.63844288836377,35.47200972846498,30.50498336483673,27.84531878966344,46.4865964846072,43.15071076500519,29.11205485225715,24.77298795064318,42.8783420704184,38.89459265200698,25.9318314092501,24.66176187332132,48.92452174922639,41.61820469216725,49.0715116442384,44.8560013063214,39.98777375595611,37.65561094244293,34.60752264792082,31.71598436897598],[50,50,50,50,50,50,55,55,55,54,53,53,52,51,49,49,47,47,52,52,49,49,51,51,49,49,52,51,56,55,52,52,50,50,49,49,53,53,51,51,48,48,50,50,52,52,55,55,49,49,50,49,57,57,56,56,57,57,51,50,50,49,53,53,47,47,49,48,47,47,49,48,47,46,56,56,51,51,48,48,48,48,49,49,50,50,49,49,49,48,48,47,50,50,49,49,51,50,50,50,50,50,52,51,48,48,51,51,50,50,51,51,50,50,51,51,53,53,52,52,47,47,52,52,46,46,47,47,50,50,48,48,47,47,49,49,47,47,48,47,50,50,50,50,51,51,52,51,49,49,51,50,50,50,50,50,49,49,56,56,55,54,51,50,52,51,49,49,56,55,51,51,50,50,51,51,50,49,51,50,50,49,50,49,50,50,50,50],[70.99450365130882,70.94970104922,71.6077853702683,71.57006932048243,71.37048947982596,71.34909513873323,67.63537258316457,67.59850094334911,67.27552444732618,67.29941820117774,69.81030447196419,69.79794354817294,67.3891840989947,67.40075588746186,69.97409488362221,69.94473067977306,70.15564761843483,70.14765641695956,70.64911198393702,70.63798609307879,70.52375547656602,70.49129943078442,69.56629137258142,69.55362461188987,69.5789345642754,69.56648073891547,72.05675672310524,72.00678088328526,67.46573719073048,67.43451077304356,71.65190133985875,71.57907671667679,71.91457176934836,71.87581551718692,70.86855446421853,70.83251692065924,68.98765258881221,68.97478455795414,72.12520955008269,72.07273003321266,70.25664055525881,70.24672496155951,72.48310210005351,72.43550950076445,69.24075115779091,69.20096206452806,65.71041859004502,65.71439627370248,71.83580013243954,71.80596963854177,71.78001932266805,71.7426006107885,64.83038492886948,64.80244064282421,61.04070479412057,61.09720506761344,63.5770330870892,63.51813955550217,70.84033964362025,70.81559454462523,70.32276981136714,70.2977670147575,69.95278845056109,69.93736615486193,67.08846892325732,67.1785569505181,68.97556180874683,68.97502702334178,70.10586582079438,70.10302910661483,70.7763869786255,70.75954861510594,71.7617798688552,71.75553158478334,55.45523973343748,55.48629423940275,68.20333484306134,68.21989704629021,68.97772161385022,68.98681795372525,69.68505052904636,69.71416413971366,67.23826040870642,67.2960704236839,69.29508711372512,69.30472583522152,69.79594331709529,69.82365653632436,70.15769078999701,70.17179451853247,69.16238841691252,69.13069027733847,62.89937514188273,62.9960468583411,66.73800284651216,66.7726021721264,69.70043829667458,69.75020140832078,69.60418465635694,69.59359218135353,70.19657156945655,70.2076461250751,68.20027201698309,68.20387324143127,70.70150307881249,70.68481556513989,66.9108794489874,67.02787542247906,62.83196256239361,62.83632017912259,62.3978257293703,62.40976949302618,62.66981006486975,62.74734388423317,61.25034878181786,61.20876331588516,62.51406635004678,62.54887129749887,59.37218561535175,59.43218212558739,71.82758758657417,71.8154026177192,64.97572734732417,64.94042116385464,71.83378971094805,71.8378568109872,71.43267710903193,71.41957634927692,67.46717124834394,67.53056356242003,69.40211202621327,69.4040866905215,70.44186197706321,70.44833723789284,68.69065908945618,68.69845554793189,69.82852078069125,69.81191666481485,71.22680542383981,71.2217691211744,63.76183701725796,63.75337946485171,66.42769991415034,66.50265846570754,68.49809876670275,68.50728197167865,67.06030577497972,67.12089831909311,71.33691250301393,71.31655469334838,70.19097707180217,70.19700670863295,68.86341685548594,68.88315807101922,68.90091432180976,68.9019695407691,66.28385454194397,66.33965041199539,61.02700458085443,61.09410640837914,62.79185984608861,62.88535539008178,69.62583853260833,69.59111916050225,69.6045617131746,69.59570394491011,69.51590965231482,69.5132916999134,65.59186483811203,65.62366261902457,70.86788292727262,70.84767680983914,70.51154115055338,70.47341420112691,70.83638305392249,70.81666398072903,71.93679307936257,71.89850886049514,70.52853152078183,70.5056143934352,71.7617741116872,71.73550862751937,69.64147299658741,69.61672323827473,71.42908882022132,71.40740585749843,70.87750727814891,70.81753636851552],[27.27272727272727,9.090909090909092,18.18181818181818,9.090909090909092,18.18181818181818,9.090909090909092,18.18181818181818,9.090909090909092,27.27272727272727,9.090909090909092,18.18181818181818,9.090909090909092,36.36363636363637,18.18181818181818,27.27272727272727,18.18181818181818,27.27272727272727,9.090909090909092,36.36363636363637,27.27272727272727,27.27272727272727,18.18181818181818,36.36363636363637,27.27272727272727,27.27272727272727,27.27272727272727,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,18.18181818181818,9.090909090909092,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,9.090909090909092,27.27272727272727,27.27272727272727,18.18181818181818,18.18181818181818,18.18181818181818,9.090909090909092,18.18181818181818,18.18181818181818,27.27272727272727,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,27.27272727272727,27.27272727272727,18.18181818181818,18.18181818181818,27.27272727272727,27.27272727272727,18.18181818181818,18.18181818181818,18.18181818181818,27.27272727272727,27.27272727272727,27.27272727272727,18.18181818181818,18.18181818181818,27.27272727272727,27.27272727272727,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,27.27272727272727,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,27.27272727272727,27.27272727272727,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,27.27272727272727,27.27272727272727,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,18.18181818181818,27.27272727272727,9.090909090909092,27.27272727272727,18.18181818181818,27.27272727272727,18.18181818181818,18.18181818181818,9.090909090909092,18.18181818181818,9.090909090909092,27.27272727272727,18.18181818181818,18.18181818181818,9.090909090909092,18.18181818181818,9.090909090909092,18.18181818181818,9.090909090909092,18.18181818181818,9.090909090909092,18.18181818181818,9.090909090909092,18.18181818181818,9.090909090909092,18.18181818181818,9.090909090909092,18.18181818181818,9.090909090909092,18.18181818181818,9.090909090909092,27.27272727272727,9.090909090909092],[9646404,9646404,5971251,5971251,8994528,8994528,14094464,14094464,13689268,13689268,12803412,12803412,12211077,12211077,9122059,9122059,11702871,11702871,11303005,11303005,8839709,8839709,10678026,10678026,14847165,14847165,5862953,5862953,13609246,13609246,9069762,9069762,10036729,10036729,10446106,10446106,10762175,10762175,8720469,8720469,12193521,12193521,7383669,7383669,10381840,10381840,12450965,12450965,7627631,7627631,8242133,8242133,16699371,16699371,11950951,11950951,13986120,13986120,8616090,8616090,8134850,8134850,6954866,6954866,10254437,10254437,15049027,15049027,11124138,11124138,12820367,12820367,5638348,5638348,734837,734837,10266750,10266750,14437895,14437895,12982038,12982038,11071957,11071957,10371500,10371500,9018945,9018945,9617315,9617315,13338827,13338827,14457362,14457362,7751943,7751943,8455460,8455460,11006304,11006304,5856849,5856849,11297824,11297824,7400743,7400743,8938017,8938017,13499581,13499581,12237851,12237851,13521248,13521248,15777772,15777772,11674777,11674777,12465025,12465025,7822917,7822917,11120375,11120375,11811856,11811856,5703944,5703944,10611034,10611034,8628302,8628302,7583015,7583015,8665986,8665986,11430178,11430178,9761129,9761129,10348384,10348384,11412974,11412974,10920806,10920806,10824038,10824038,12674448,12674448,11631878,11631878,9780958,9780958,7120797,7120797,8514537,8514537,17230410,17230410,11073255,11073255,10304161,10304161,13873472,13873472,9460829,9460829,17871310,17871310,7129722,7129722,10992015,10992015,6136901,6136901,10476954,10476954,7398484,7398484,9244604,9244604,8197575,8197575,6738747,6738747,8097526,8097526]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Sample<\/th>\n      <th>prefix<\/th>\n      <th>read<\/th>\n      <th>Filename<\/th>\n      <th>File type<\/th>\n      <th>Encoding<\/th>\n      <th>Total Sequences<\/th>\n      <th>Sequences flagged as poor quality<\/th>\n      <th>Sequence length<\/th>\n      <th>%GC<\/th>\n      <th>total_deduplicated_percentage<\/th>\n      <th>avg_sequence_length<\/th>\n      <th>basic_statistics<\/th>\n      <th>per_base_sequence_quality<\/th>\n      <th>per_tile_sequence_quality<\/th>\n      <th>per_sequence_quality_scores<\/th>\n      <th>per_base_sequence_content<\/th>\n      <th>per_sequence_gc_content<\/th>\n      <th>per_base_n_content<\/th>\n      <th>sequence_length_distribution<\/th>\n      <th>sequence_duplication_levels<\/th>\n      <th>overrepresented_sequences<\/th>\n      <th>adapter_content<\/th>\n      <th>histone_mark<\/th>\n      <th>sample_ID<\/th>\n      <th>ind_ID<\/th>\n      <th>ind<\/th>\n      <th>trt<\/th>\n      <th>time<\/th>\n      <th>other_time<\/th>\n      <th>FastQC_mqc-generalstats-fastqc-percent_duplicates<\/th>\n      <th>FastQC_mqc-generalstats-fastqc-percent_gc<\/th>\n      <th>FastQC_mqc-generalstats-fastqc-avg_sequence_length<\/th>\n      <th>FastQC_mqc-generalstats-fastqc-percent_fails<\/th>\n      <th>FastQC_mqc-generalstats-fastqc-total_sequences<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"scrollY":"400px","scrollCollapse":true,"fixedColumns":{"leftColumns":2},"fixedHeader":true,"columnDefs":[{"className":"dt-right","targets":[7,8,10,11,12,31,32,33,34,35]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Sample","targets":1},{"name":"prefix","targets":2},{"name":"read","targets":3},{"name":"Filename","targets":4},{"name":"File type","targets":5},{"name":"Encoding","targets":6},{"name":"Total Sequences","targets":7},{"name":"Sequences flagged as poor quality","targets":8},{"name":"Sequence length","targets":9},{"name":"%GC","targets":10},{"name":"total_deduplicated_percentage","targets":11},{"name":"avg_sequence_length","targets":12},{"name":"basic_statistics","targets":13},{"name":"per_base_sequence_quality","targets":14},{"name":"per_tile_sequence_quality","targets":15},{"name":"per_sequence_quality_scores","targets":16},{"name":"per_base_sequence_content","targets":17},{"name":"per_sequence_gc_content","targets":18},{"name":"per_base_n_content","targets":19},{"name":"sequence_length_distribution","targets":20},{"name":"sequence_duplication_levels","targets":21},{"name":"overrepresented_sequences","targets":22},{"name":"adapter_content","targets":23},{"name":"histone_mark","targets":24},{"name":"sample_ID","targets":25},{"name":"ind_ID","targets":26},{"name":"ind","targets":27},{"name":"trt","targets":28},{"name":"time","targets":29},{"name":"other_time","targets":30},{"name":"FastQC_mqc-generalstats-fastqc-percent_duplicates","targets":31},{"name":"FastQC_mqc-generalstats-fastqc-percent_gc","targets":32},{"name":"FastQC_mqc-generalstats-fastqc-avg_sequence_length","targets":33},{"name":"FastQC_mqc-generalstats-fastqc-percent_fails","targets":34},{"name":"FastQC_mqc-generalstats-fastqc-total_sequences","targets":35}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code># combo_trim_df %&gt;% 
#   dplyr::select(prefix,histone_mark,sample_ID:other_time,`Total Sequences`) %&gt;% 
#   distinct() %&gt;% 
#   mutate(File_name=paste0(prefix,&quot;.bed&quot;))</code></pre>
<pre class="r"><code>combo_trim_df %&gt;% 
  dplyr::filter(read==&quot;R1&quot;) %&gt;% 
  ggplot(., aes(x = Sample, y= avg_sequence_length))+
geom_col(aes(fill=histone_mark)) +
  geom_hline( yintercept = 75)+
  theme_classic()+
  ggtitle(&quot;Graph of average read length across R1 samples&quot;)+ theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/trim%20info2-1.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-trim-info2-1">
Past versions of "trim info2-1.png"
</button>
</p>
<div id="fig-no-spaces-trim-info2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/63d7a250d778a5c0d012d5550d88a6826b6f0375/docs/figure/QC_seq_analysis.Rmd/trim info2-1.png" target="_blank">63d7a25</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/d73698d0e86d481cd5455c1cd6177aff97cb7804/docs/figure/QC_seq_analysis.Rmd/trim info2-1.png" target="_blank">d73698d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>combo_trim_df %&gt;% 
  dplyr::filter(read==&quot;R2&quot;) %&gt;% 
  ggplot(., aes(x = Sample, y= avg_sequence_length))+
geom_col(aes(fill=histone_mark)) +
  geom_hline( yintercept = 75)+
  theme_classic()+
  ggtitle(&quot;Graph of average read length across R2 samples&quot;)+ theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/trim%20info2-2.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-trim-info2-2">
Past versions of "trim info2-2.png"
</button>
</p>
<div id="fig-no-spaces-trim-info2-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/63d7a250d778a5c0d012d5550d88a6826b6f0375/docs/figure/QC_seq_analysis.Rmd/trim info2-2.png" target="_blank">63d7a25</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/d73698d0e86d481cd5455c1cd6177aff97cb7804/docs/figure/QC_seq_analysis.Rmd/trim info2-2.png" target="_blank">d73698d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>percent gc</p>
<pre class="r"><code>combo_notrim_df %&gt;% 
  dplyr::filter(read==&quot;R1&quot;) %&gt;% 
  ggplot(., aes(x = Sample, y= `%GC`))+
  geom_col(aes(fill=histone_mark)) +
  theme_classic()+
  ggtitle(&quot;Graph of %GC for R1 not-trimmed&quot;)+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/GCper-1.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-GCper-1">
Past versions of GCper-1.png
</button>
</p>
<div id="fig-GCper-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/63d7a250d778a5c0d012d5550d88a6826b6f0375/docs/figure/QC_seq_analysis.Rmd/GCper-1.png" target="_blank">63d7a25</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>combo_trim_df %&gt;% 
  dplyr::filter(read==&quot;R1&quot;) %&gt;% 
  ggplot(., aes(x = Sample, y= `%GC`))+
  geom_col(aes(fill=histone_mark)) +
  theme_classic()+
  ggtitle(&quot;Graph of %GC for R1 trimmed&quot;)+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/GCper-2.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-GCper-2">
Past versions of GCper-2.png
</button>
</p>
<div id="fig-GCper-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/63d7a250d778a5c0d012d5550d88a6826b6f0375/docs/figure/QC_seq_analysis.Rmd/GCper-2.png" target="_blank">63d7a25</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>combo_notrim_df %&gt;% 
  dplyr::filter(read==&quot;R2&quot;) %&gt;% 
  ggplot(., aes(x = Sample, y= `%GC`))+
  geom_col(aes(fill=histone_mark)) +
  theme_classic()+
  ggtitle(&quot;Graph of %GC for R2 not-trimmed&quot;)+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/GCper-3.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-GCper-3">
Past versions of GCper-3.png
</button>
</p>
<div id="fig-GCper-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/63d7a250d778a5c0d012d5550d88a6826b6f0375/docs/figure/QC_seq_analysis.Rmd/GCper-3.png" target="_blank">63d7a25</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>combo_trim_df %&gt;% 
  dplyr::filter(read==&quot;R2&quot;) %&gt;% 
  ggplot(., aes(x = Sample, y= `%GC`))+
  geom_col(aes(fill=histone_mark)) +
  theme_classic()+
  ggtitle(&quot;Graph of %GC for R2 trimmed&quot;)+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/GCper-4.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-GCper-4">
Past versions of GCper-4.png
</button>
</p>
<div id="fig-GCper-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/63d7a250d778a5c0d012d5550d88a6826b6f0375/docs/figure/QC_seq_analysis.Rmd/GCper-4.png" target="_blank">63d7a25</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Duplication information</p>
<pre class="r"><code>combo_notrim_df %&gt;% 
  dplyr::filter(read==&quot;R1&quot;) %&gt;% 
  ggplot(., aes(x = histone_mark, y= `FastQC_mqc-generalstats-fastqc-percent_duplicates`))+
  geom_boxplot(aes(fill=histone_mark)) + 
    geom_point(aes(color=ind))+
  facet_wrap(trt~other_time)+
  ylab(&quot;percent duplication&quot;)+
  theme(axis.text.x=element_text(angle=90))+
   ggtitle(&quot;Duplication percentage (R1 no trim)&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/dup%20viz-1.png" width="768" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-dup-viz-1">
Past versions of "dup viz-1.png"
</button>
</p>
<div id="fig-no-spaces-dup-viz-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/63d7a250d778a5c0d012d5550d88a6826b6f0375/docs/figure/QC_seq_analysis.Rmd/dup viz-1.png" target="_blank">63d7a25</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/614b3c814910a8aa65f93769216d612a6a44304b/docs/figure/QC_seq_analysis.Rmd/dup viz-1.png" target="_blank">614b3c8</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-03-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>combo_trim_df %&gt;% 
  dplyr::filter(read==&quot;R1&quot;) %&gt;% 
  ggplot(., aes(x = histone_mark, y= `FastQC_mqc-generalstats-fastqc-percent_duplicates`))+
  geom_boxplot(aes(fill=histone_mark)) + 
    geom_point(aes(color=ind))+
  facet_wrap(trt~other_time)+
  ylab(&quot;percent duplication&quot;)+
  theme(axis.text.x=element_text(angle=90))+
  ggtitle(&quot;Duplication percentage (R1 trimmed)&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/dup%20viz-2.png" width="768" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-dup-viz-2">
Past versions of "dup viz-2.png"
</button>
</p>
<div id="fig-no-spaces-dup-viz-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/63d7a250d778a5c0d012d5550d88a6826b6f0375/docs/figure/QC_seq_analysis.Rmd/dup viz-2.png" target="_blank">63d7a25</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/614b3c814910a8aa65f93769216d612a6a44304b/docs/figure/QC_seq_analysis.Rmd/dup viz-2.png" target="_blank">614b3c8</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-03-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>combo_notrim_df %&gt;% 
  dplyr::filter(read==&quot;R1&quot;) %&gt;% 
  ggplot(., aes(x = interaction(other_time,trt), y= `FastQC_mqc-generalstats-fastqc-percent_duplicates`))+
  geom_boxplot(aes(fill=histone_mark)) + 
  geom_point(aes(color=ind))+
facet_wrap(~histone_mark)+
  ylab(&quot;percent duplication&quot;)+
  theme(axis.text.x=element_text(angle=90))+
  ggtitle(&quot;Duplication percentage (R1 no trim)&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/dup%20viz-3.png" width="768" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-dup-viz-3">
Past versions of "dup viz-3.png"
</button>
</p>
<div id="fig-no-spaces-dup-viz-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/e1f2574321ea7f60075b7087872dda4ec9237fb3/docs/figure/QC_seq_analysis.Rmd/dup viz-3.png" target="_blank">e1f2574</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/63d7a250d778a5c0d012d5550d88a6826b6f0375/docs/figure/QC_seq_analysis.Rmd/dup viz-3.png" target="_blank">63d7a25</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/614b3c814910a8aa65f93769216d612a6a44304b/docs/figure/QC_seq_analysis.Rmd/dup viz-3.png" target="_blank">614b3c8</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-03-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>combo_trim_df %&gt;% 
  dplyr::filter(read==&quot;R1&quot;) %&gt;% 
  ggplot(., aes(x = interaction(other_time,trt), y= `FastQC_mqc-generalstats-fastqc-percent_duplicates`))+
  geom_boxplot(aes(fill=histone_mark)) + 
  geom_point(aes(color=ind))+
facet_wrap(~histone_mark)+
  ylab(&quot;percent duplication&quot;)+
  theme(axis.text.x=element_text(angle=90))+
  ggtitle(&quot;Duplication percentage (R1 trimmed)&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/dup%20viz-4.png" width="768" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-dup-viz-4">
Past versions of "dup viz-4.png"
</button>
</p>
<div id="fig-no-spaces-dup-viz-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/e1f2574321ea7f60075b7087872dda4ec9237fb3/docs/figure/QC_seq_analysis.Rmd/dup viz-4.png" target="_blank">e1f2574</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-28
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/63d7a250d778a5c0d012d5550d88a6826b6f0375/docs/figure/QC_seq_analysis.Rmd/dup viz-4.png" target="_blank">63d7a25</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>combo_notrim_df %&gt;% 
  dplyr::filter(read==&quot;R1&quot;) %&gt;% 
  ggplot(., aes(x = Sample, y= `FastQC_mqc-generalstats-fastqc-percent_duplicates`))+
  geom_col(aes(fill=histone_mark)) +
  theme_classic()+
  ggtitle(&quot;Graph of percent duplicates for R1 not-trimmed&quot;)+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous(limits = c(0,100), expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/Dup%20all%20samp-1.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Dup-all-samp-1">
Past versions of "Dup all samp-1.png"
</button>
</p>
<div id="fig-no-spaces-Dup-all-samp-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/63d7a250d778a5c0d012d5550d88a6826b6f0375/docs/figure/QC_seq_analysis.Rmd/Dup all samp-1.png" target="_blank">63d7a25</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>combo_trim_df %&gt;% 
  dplyr::filter(read==&quot;R1&quot;) %&gt;% 
  ggplot(., aes(x = Sample, y= `FastQC_mqc-generalstats-fastqc-percent_duplicates`))+
  geom_col(aes(fill=histone_mark)) +
  theme_classic()+
  ggtitle(&quot;Graph of percent duplicates for R1 trimmed&quot;)+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( limits = c(0,100),expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/Dup%20all%20samp-2.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Dup-all-samp-2">
Past versions of "Dup all samp-2.png"
</button>
</p>
<div id="fig-no-spaces-Dup-all-samp-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/63d7a250d778a5c0d012d5550d88a6826b6f0375/docs/figure/QC_seq_analysis.Rmd/Dup all samp-2.png" target="_blank">63d7a25</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>combo_notrim_df %&gt;% 
  dplyr::filter(read==&quot;R2&quot;) %&gt;% 
  ggplot(., aes(x = Sample, y= `FastQC_mqc-generalstats-fastqc-percent_duplicates`))+
  geom_col(aes(fill=histone_mark)) +
  theme_classic()+
  ggtitle(&quot;Graph of percent duplicates for R2 not-trimmed&quot;)+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous(limits = c(0,100), expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/Dup%20all%20samp-3.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Dup-all-samp-3">
Past versions of "Dup all samp-3.png"
</button>
</p>
<div id="fig-no-spaces-Dup-all-samp-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/63d7a250d778a5c0d012d5550d88a6826b6f0375/docs/figure/QC_seq_analysis.Rmd/Dup all samp-3.png" target="_blank">63d7a25</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>combo_trim_df %&gt;% 
  dplyr::filter(read==&quot;R2&quot;) %&gt;% 
  ggplot(., aes(x = Sample, y= `FastQC_mqc-generalstats-fastqc-percent_duplicates`))+
  geom_col(aes(fill=histone_mark)) +
  theme_classic()+
  ggtitle(&quot;Graph of percent duplicates for R2 trimmed&quot;)+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous(limits = c(0,100), expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/Dup%20all%20samp-4.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Dup-all-samp-4">
Past versions of "Dup all samp-4.png"
</button>
</p>
<div id="fig-no-spaces-Dup-all-samp-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/63d7a250d778a5c0d012d5550d88a6826b6f0375/docs/figure/QC_seq_analysis.Rmd/Dup all samp-4.png" target="_blank">63d7a25</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>### creating the sample info file from bowtie2 metrics from the first run:

### import files

metric_1 &lt;-  read_delim(&quot;C:/Users/reneels/Documents/Ward Lab/DXR_project/Sample_info/first_run_metrics/metric_1.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE)
metric_2 &lt;- read_delim(&quot;C:/Users/renee/Documents/Ward Lab/DXR_project/Sample_info/first_run_metrics/Second_set_info.txt&quot;, delim = &quot;\t&quot;, escape_double = FALSE, col_names = FALSE, trim_ws = TRUE)
pull_name_file &lt;- read_csv(&quot;C:/Users/renee/Documents/Ward Lab/DXR_project/Sample_info/first_run_metrics/samplesheet.txt&quot;, col_names = FALSE)
pull &lt;- pull_name_file[65:94,]

 fun &lt;- cbind(pull, metric_2[,1:8])
 colnames(fun) &lt;- c(&quot;X1&quot;,&quot;X2&quot;,&quot;X3&quot;,&quot;X4&quot;,&quot;X5&quot;,&quot;X6&quot;,&quot;X7&quot;,&quot;X8&quot;,&quot;X9&quot;)
 
 # almost_fun &lt;- cbind(pull_name_file[1:65,],metric_1)
 
 ##remove 229 from almost_fun
# subset_almost_fun &lt;- almost_fun[1:64,]

alignment_specs &lt;- rbind(metric_1,fun)

colnames(alignment_specs) &lt;- c(&quot;sample_ID&quot;,
                &quot;Paired_Reads&quot;,
                &quot;aligned_pairs&quot;,
                &quot;aligned_concordant_0&quot;,
                &quot;zero&quot;,
                &quot;aligned_concordant_1&quot;,
                &quot;one&quot;,
                &quot;aligned_concordant_g1&quot;,
                &quot;percent_alignment&quot;,
                &quot;N_A&quot;)
colnames(fun) &lt;- c(&quot;sample_ID&quot;,
                &quot;Paired_Reads&quot;,
                &quot;aligned_pairs&quot;,
                &quot;aligned_concordant_0&quot;,
                &quot;zero&quot;,
                &quot;aligned_concordant_1&quot;,
                &quot;one&quot;,
                &quot;aligned_concordant_g1&quot;,
                &quot;percent_alignment&quot;,
                &quot;N_A&quot;)
saveRDS(fun, &quot;data/first_run_alignment_229andover.RDS&quot;)
saveRDS(alignment_specs, &quot;data/first_run_alignment_specs.RDS&quot;)</code></pre>
<pre class="r"><code>alignment_temp&lt;-readRDS(&quot;data/first_run_alignment_specs.RDS&quot;)
for_plots &lt;- alignment_temp %&gt;% 
  left_join(.,sampleinfo, by=c(&quot;sample_ID&quot;=&quot;library_ID&quot;))%&gt;% 
  dplyr::select(sample_ID:aligned_concordant_0,aligned_concordant_1, aligned_concordant_g1,percent_alignment, histone_mark,ind, trt,other_time) %&gt;% 
  distinct()</code></pre>
<p>alignment pair numbers</p>
<pre class="r"><code>for_plots %&gt;% 
  group_by(histone_mark) %&gt;% 
  ggplot(., aes(x=sample_ID, y=Paired_Reads))+
    geom_col(aes(fill = histone_mark)) +
  theme_classic()+
  ggtitle(&quot;Graph of total paired reads&quot;)+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/unnamed-chunk-3-1.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-3-1">
Past versions of unnamed-chunk-3-1.png
</button>
</p>
<div id="fig-unnamed-chunk-3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/unnamed-chunk-3-1.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>for_plots %&gt;% 
  mutate(percent_alignment=parse_number(percent_alignment)) %&gt;% 
  group_by(histone_mark) %&gt;% 
  ggplot(., aes(x=sample_ID, y=percent_alignment))+
    geom_col(aes(fill = histone_mark)) +
  theme_classic()+
  ggtitle(&quot;Percent aligned of paired reads&quot;) + 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous(expand = expansion(mult = c(0, .1)))+
  coord_cartesian(ylim=c(80,100))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/unnamed-chunk-3-2.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-3-2">
Past versions of unnamed-chunk-3-2.png
</button>
</p>
<div id="fig-unnamed-chunk-3-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/unnamed-chunk-3-2.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>for_plots %&gt;% 
  group_by(histone_mark) %&gt;% 
   mutate(percent_aligned_unique = round(aligned_concordant_1 / Paired_Reads * 100, 2)) %&gt;% 
  ggplot(., aes(x=sample_ID, y=percent_aligned_unique))+
    geom_col(aes(fill = histone_mark)) +
  theme_classic()+
  ggtitle(&quot;Percentage of paired reads aligning once&quot;)+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/unnamed-chunk-3-3.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-3-3">
Past versions of unnamed-chunk-3-3.png
</button>
</p>
<div id="fig-unnamed-chunk-3-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/unnamed-chunk-3-3.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="read-analysis" class="section level3">
<h3>Read analysis</h3>
<pre class="r"><code>file_list &lt;- list.files(path=&quot;C:/Users/renee/Documents/DXR_files/metric_files&quot;,
           pattern =&quot;frag_len_count\\.txt$&quot;,full.names = TRUE)
read_and_label &lt;- function(file) {
  df &lt;- read_delim(file, delim = &quot;\t&quot;, col_names = c(&quot;Col1&quot;, &quot;Col2&quot;))  # Adjust delimiter if needed
  df &lt;- df %&gt;%
    mutate(File = basename(file),  # Add filename column
  weight = Col2/sum(Col2))
  return(df)
}
combined_df &lt;- map_df(file_list, read_and_label)

annotated_combo_df &lt;- combined_df %&gt;% 
  mutate(sample = gsub(&quot;_frag_len_count.txt&quot;,&quot;&quot;,File)) %&gt;% 
  left_join(., sampleinfo, by = c(&quot;sample&quot;=&quot;library_ID&quot;))</code></pre>
<pre class="r"><code>annotated_combo_df %&gt;% 
  dplyr::filter(histone_mark==&quot;H3K9me3&quot;) %&gt;% 
  ggplot(., aes(x=Col1, y=Col2, color = sample))+
  geom_line(size=1)+
  # geom_point(size=2,alpha =0.7)+
  scale_x_continuous(breaks = seq(0, max(annotated_combo_df$Col1), by = 50))+
   facet_wrap(~sample)+
  labs(title = &quot;Fragment length, H3K9me3&quot;,
       x = &quot;Fragment Length (bp)&quot;,
       y = &quot;Count&quot;,
       color= &quot;Sample&quot;)+
  theme_minimal()</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/unnamed-chunk-5-1.png" width="1440" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-5-1">
Past versions of unnamed-chunk-5-1.png
</button>
</p>
<div id="fig-unnamed-chunk-5-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/unnamed-chunk-5-1.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>annotated_combo_df %&gt;% 
  dplyr::filter(histone_mark==&quot;H3K9me3&quot;) %&gt;%
  ggplot(., aes(x=sample, y=Col1, weight = weight,fill = histone_mark))+
geom_violin(bw = 5) +
    scale_y_continuous(breaks = seq(0, 800, 50)) +
   theme_bw(base_size = 20) +
    ggpubr::rotate_x_text(angle = 90) +
  ggtitle(&quot;Fragment lengths for H3K9me3&quot;)+
    ylab(&quot;Fragment Length&quot;) +
    xlab(&quot;&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/unnamed-chunk-5-2.png" width="1440" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-5-2">
Past versions of unnamed-chunk-5-2.png
</button>
</p>
<div id="fig-unnamed-chunk-5-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/unnamed-chunk-5-2.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>annotated_combo_df %&gt;% 
  dplyr::filter(histone_mark==&quot;H3K27me3&quot;) %&gt;% 
  ggplot(., aes(x=Col1, y=Col2, color = sample))+
  geom_line(size=1)+
  # geom_point(size=2,alpha =0.7)+
  scale_x_continuous(breaks = seq(0, max(annotated_combo_df$Col1), by = 50))+
   facet_wrap(~sample)+
  labs(title = &quot;Fragment length, H3K27me3&quot;,
       x = &quot;Fragment Length (bp)&quot;,
       y = &quot;Count&quot;,
       color= &quot;Sample&quot;)+
  theme_minimal()</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/unnamed-chunk-5-3.png" width="1440" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-5-3">
Past versions of unnamed-chunk-5-3.png
</button>
</p>
<div id="fig-unnamed-chunk-5-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/unnamed-chunk-5-3.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>annotated_combo_df %&gt;% 
  dplyr::filter(histone_mark==&quot;H3K27me3&quot;) %&gt;%
  ggplot(., aes(x=sample, y=Col1, weight = weight,fill = histone_mark))+
geom_violin(bw = 5) +
    scale_y_continuous(breaks = seq(0, 800, 50)) +
   theme_bw(base_size = 20) +
    ggpubr::rotate_x_text(angle = 90) +
  ggtitle(&quot;Fragment lengths for H3K27me3&quot;)+
    ylab(&quot;Fragment Length&quot;) +
    xlab(&quot;&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/unnamed-chunk-5-4.png" width="1440" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-5-4">
Past versions of unnamed-chunk-5-4.png
</button>
</p>
<div id="fig-unnamed-chunk-5-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/unnamed-chunk-5-4.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>annotated_combo_df %&gt;% 
  dplyr::filter(histone_mark==&quot;H3K27ac&quot;) %&gt;% 
  ggplot(., aes(x=Col1, y=Col2, color = sample))+
  geom_line(size=1)+
  # geom_point(size=2,alpha =0.7)+
  scale_x_continuous(breaks = seq(0, max(annotated_combo_df$Col1), by = 50))+
   facet_wrap(~sample)+
  labs(title = &quot;Fragment length, H3K27ac&quot;,
       x = &quot;Fragment Length (bp)&quot;,
       y = &quot;Count&quot;,
       color= &quot;Sample&quot;)+
  theme_minimal()</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/unnamed-chunk-5-5.png" width="1440" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-5-5">
Past versions of unnamed-chunk-5-5.png
</button>
</p>
<div id="fig-unnamed-chunk-5-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/unnamed-chunk-5-5.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>annotated_combo_df %&gt;% 
  dplyr::filter(histone_mark==&quot;H3K27ac&quot;) %&gt;%
  ggplot(., aes(x=sample, y=Col1, weight = weight,fill = histone_mark))+
geom_violin(bw = 5) +
    scale_y_continuous(breaks = seq(0, 800, 50)) +
   theme_bw(base_size = 20) +
    ggpubr::rotate_x_text(angle = 90) +
  ggtitle(&quot;Fragment lengths for H3K27ac&quot;)+
    ylab(&quot;Fragment Length&quot;) +
    xlab(&quot;&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/unnamed-chunk-5-6.png" width="1440" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-5-6">
Past versions of unnamed-chunk-5-6.png
</button>
</p>
<div id="fig-unnamed-chunk-5-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/unnamed-chunk-5-6.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>annotated_combo_df %&gt;% 
  dplyr::filter(histone_mark==&quot;H3K36me3&quot;) %&gt;% 
  ggplot(., aes(x=Col1, y=Col2, color = sample))+
  geom_line(size=1)+
  # geom_point(size=2,alpha =0.7)+
  scale_x_continuous(breaks = seq(0, max(annotated_combo_df$Col1), by = 50))+
  facet_wrap(~sample)+
  labs(title = &quot;Fragment length, H3K36me3&quot;,
       x = &quot;Fragment Length (bp)&quot;,
       y = &quot;Count&quot;,
       color= &quot;Sample&quot;)+
  theme_minimal()</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/unnamed-chunk-5-7.png" width="1440" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-5-7">
Past versions of unnamed-chunk-5-7.png
</button>
</p>
<div id="fig-unnamed-chunk-5-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/unnamed-chunk-5-7.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>annotated_combo_df %&gt;% 
  dplyr::filter(histone_mark==&quot;H3K36me3&quot;) %&gt;%
  ggplot(., aes(x=sample, y=Col1, weight = weight,fill = histone_mark))+
geom_violin(bw = 5) +
    scale_y_continuous(breaks = seq(0, 800, 50)) +
   theme_bw(base_size = 20) +
    ggpubr::rotate_x_text(angle = 90) +
  ggtitle(&quot;Fragment lengths for H3K36me3&quot;)+
    ylab(&quot;Fragment Length&quot;) +
    xlab(&quot;&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/unnamed-chunk-5-8.png" width="1440" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-5-8">
Past versions of unnamed-chunk-5-8.png
</button>
</p>
<div id="fig-unnamed-chunk-5-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/unnamed-chunk-5-8.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="peak-count-across-files" class="section level4">
<h4>Peak count across files</h4>
<pre class="r"><code>peak_count_H3K27ac &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/macs2_output/H3K27ac_peak_count.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_peaks.narrowPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K27ac&quot;)

peak_count_H3K27me3 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/macs2_output/H3K27me3_peak_count.txt&quot;, 
    col_names = FALSE)%&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_peaks.broadPeak&quot;,&quot;&quot;,X2)) %&gt;% 
   dplyr::filter(X2 != &quot;H3K27me3_all&quot;) %&gt;% 
  mutate(histone=&quot;H3K27me3&quot;)


peak_count_H3K9me3 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/macs2_output/H3K9me3_peak_count.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_peaks.narrowPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K9me3&quot;)



peak_count_H3K36me3 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/macs2_output/H3K36me3_peak_count.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
  mutate(X2=gsub(&quot;_peaks.broadPeak&quot;,&quot;&quot;,X2)) %&gt;% 
   dplyr::filter(X2 != &quot;H3K36me3_all&quot;) %&gt;% 
  mutate(histone=&quot;H3K36me3&quot;)
 


All_peak_count &lt;- rbind(peak_count_H3K27ac,peak_count_H3K27me3,peak_count_H3K9me3,peak_count_H3K36me3)

All_peak_count %&gt;% 
   ggplot(.,aes(x=X2, y=X1,fill=histone))+
   geom_col()+
   ylab(&quot;Count&quot;)+
   theme_classic()+
  # facet_wrap(~histone)+
  ggtitle(&quot;Peak number for all samples&quot;)+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/peakcount-1.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-peakcount-1">
Past versions of peakcount-1.png
</button>
</p>
<div id="fig-peakcount-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/peakcount-1.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>All_peak_count %&gt;% 
  ggplot(., aes (x=histone, y = X1, fill = histone))+
  geom_boxplot()+
   ylab(&quot;Count&quot;)+
   theme_classic()+
  # facet_wrap(~histone)+
  ggtitle(&quot;Peak count across histones&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/peakcount-2.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-peakcount-2">
Past versions of peakcount-2.png
</button>
</p>
<div id="fig-peakcount-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/peakcount-2.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>All_peak_count %&gt;% 
  left_join(sampleinfo, by = c(&quot;X2&quot;=&quot;library_ID&quot;, &quot;histone&quot;=&quot;histone_mark&quot;)) %&gt;% 
  ggplot(., aes (x=trt, y = X1, fill = histone))+
  geom_boxplot()+
   ylab(&quot;Count&quot;)+
   theme_classic()+
  # facet_wrap(~histone)+
  ggtitle(&quot;Peak count across histones by trt&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/peakcount-3.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-peakcount-3">
Past versions of peakcount-3.png
</button>
</p>
<div id="fig-peakcount-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/peakcount-3.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>All_peak_count %&gt;% 
  left_join(sampleinfo, by = c(&quot;X2&quot;=&quot;library_ID&quot;, &quot;histone&quot;=&quot;histone_mark&quot;)) %&gt;% 
  ggplot(., aes (x=other_time, y = X1, fill = histone))+
  geom_boxplot()+
   ylab(&quot;Count&quot;)+
   theme_classic()+
  # facet_wrap(~histone)+
  ggtitle(&quot;Peak count across histones by time&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/peakcount-4.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-peakcount-4">
Past versions of peakcount-4.png
</button>
</p>
<div id="fig-peakcount-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/peakcount-4.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="h3k27ac-count-analysis" class="section level3">
<h3>H3K27ac count analysis</h3>
<pre class="r"><code>count_H3K27ac &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/count_H3K27ac.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)

noM_count_H3K27ac &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/noM_count_H3K27ac&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)

Fragment_H3K27ac_txt &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/count_H3K27ac.txt.summary&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)

Fragment_H3K27ac_txt_noM &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/noM_count_H3K27ac.summary&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)
rename_list &lt;- sampleinfo %&gt;% 
  mutate(stem= &quot;_final.bam&quot;) %&gt;% 
  mutate(oldname=paste0(library_ID,stem)) %&gt;% 
  mutate(newname=paste0(ind,&quot;_&quot;,trt,&quot;_&quot;,other_time,&quot;_&quot;,histone_mark)) %&gt;% 
  dplyr::select(oldname,newname)

noM_rename_list &lt;- sampleinfo %&gt;% 
  mutate(stem= &quot;_noM_final.bam&quot;) %&gt;% 
  mutate(oldname=paste0(library_ID,stem)) %&gt;% 
  mutate(newname=paste0(ind,&quot;_&quot;,trt,&quot;_&quot;,other_time,&quot;_&quot;,histone_mark)) %&gt;% 
  dplyr::select(oldname,newname)
  
rename_vec &lt;- setNames(rename_list$newname, rename_list$oldname)

rename_vec_noM &lt;- setNames(noM_rename_list$newname, noM_rename_list$oldname)

names(count_H3K27ac)[names(count_H3K27ac) %in% names(rename_vec)] &lt;- rename_vec[names(count_H3K27ac)[names(count_H3K27ac) %in% names(rename_vec)]]

names(noM_count_H3K27ac)[names(noM_count_H3K27ac) %in% names(rename_vec_noM)] &lt;- rename_vec_noM[names(noM_count_H3K27ac)[names(noM_count_H3K27ac) %in% names(rename_vec_noM)]]</code></pre>
<pre class="r"><code>H3K27ac_counts_raw &lt;- count_H3K27ac %&gt;% 
  dplyr::select(Geneid,`3_VEH_24r_H3K27ac`:`3_DOX_24r_H3K27ac`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()
H3K27ac_counts_raw_noM &lt;- noM_count_H3K27ac %&gt;% 
  dplyr::select(Geneid,`3_VEH_24r_H3K27ac`:`3_DOX_24r_H3K27ac`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()


first_H3K27ac_counts_cor &lt;- count_H3K27ac %&gt;% 
  dplyr::select(Geneid,`3_VEH_24r_H3K27ac`:`3_DOX_24r_H3K27ac`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  cpm(., log = TRUE) %&gt;% 
  cor()

first_H3K27ac_counts_cor_noM &lt;- noM_count_H3K27ac %&gt;% 
  dplyr::select(Geneid,`3_VEH_24r_H3K27ac`:`3_DOX_24r_H3K27ac`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  cpm(., log = TRUE) %&gt;% 
  cor()

annomat &lt;- data.frame(sample=colnames(first_H3K27ac_counts_cor)) %&gt;% 
  separate_wider_delim(sample,delim=&quot;_&quot;,names=c(&quot;ind&quot;,&quot;trt&quot;,&quot;time&quot;,NA),cols_remove = FALSE) %&gt;% 
  mutate(trt=factor(trt, levels = c(&quot;VEH&quot;,&quot;5FU&quot;,&quot;DOX&quot;)),
         time=factor(time, levels =c(&quot;24t&quot;,&quot;24r&quot;,&quot;144r&quot;))) %&gt;% 
  column_to_rownames(&quot;sample&quot;)
heatmap_first &lt;- ComplexHeatmap::HeatmapAnnotation(df = annomat)

Heatmap(first_H3K27ac_counts_cor, 
        top_annotation = heatmap_first,
        column_title=&quot;Unfiltered log2cpm H3K27ac&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/unnamed-chunk-6-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-6-1">
Past versions of unnamed-chunk-6-1.png
</button>
</p>
<div id="fig-unnamed-chunk-6-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/unnamed-chunk-6-1.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Heatmap(first_H3K27ac_counts_cor_noM, 
        top_annotation = heatmap_first,
        column_title=&quot;Unfiltered log2cpm H3K27ac_noM&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/unnamed-chunk-6-2.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-6-2">
Past versions of unnamed-chunk-6-2.png
</button>
</p>
<div id="fig-unnamed-chunk-6-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/unnamed-chunk-6-2.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="h3k27me3-count-analysis" class="section level3">
<h3>H3K27me3 count analysis</h3>
<pre class="r"><code>count_H3K27me3 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/H3K27me3_counts.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)

noM_count_H3K27me3 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/noM_count_H3K27me3&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)


Fragment_H3K27me3_txt &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/H3K27me3_counts.txt.summary&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)

Fragment_H3K27me3_txt_noM &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/noM_count_H3K27me3.summary&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)
rename_list &lt;- sampleinfo %&gt;% 
  mutate(stem= &quot;_final.bam&quot;) %&gt;% 
  mutate(oldname=paste0(library_ID,stem)) %&gt;% 
  mutate(newname=paste0(ind,&quot;_&quot;,trt,&quot;_&quot;,other_time,&quot;_&quot;,histone_mark)) %&gt;% 
  dplyr::select(oldname,newname)
  
rename_vec &lt;- setNames(rename_list$newname, rename_list$oldname)

names(count_H3K27me3)[names(count_H3K27me3) %in% names(rename_vec)] &lt;- rename_vec[names(count_H3K27me3)[names(count_H3K27me3) %in% names(rename_vec)]]

names(noM_count_H3K27me3)[names(noM_count_H3K27me3) %in% names(rename_vec_noM)] &lt;- rename_vec_noM[names(noM_count_H3K27me3)[names(noM_count_H3K27me3) %in% names(rename_vec_noM)]]</code></pre>
<pre class="r"><code>H3K27me3_counts_raw &lt;- count_H3K27me3 %&gt;% 
  dplyr::select(Geneid,`1_VEH_24t_H3K27me3`:`5_VEH_24r_H3K27me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()

H3K27me3_counts_raw_noM &lt;- noM_count_H3K27me3 %&gt;% 
  dplyr::select(Geneid,`1_VEH_24t_H3K27me3`:`5_VEH_24r_H3K27me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()


first_H3K27me3_counts_cor &lt;- count_H3K27me3 %&gt;% 
  dplyr::select(Geneid,`1_VEH_24t_H3K27me3`:`5_VEH_24r_H3K27me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  cpm(., log = TRUE) %&gt;% 
  cor()
first_H3K27me3_counts_cor_noM &lt;- noM_count_H3K27me3 %&gt;% 
  dplyr::select(Geneid,`1_VEH_24t_H3K27me3`:`5_VEH_24r_H3K27me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  cpm(., log = TRUE) %&gt;% 
  cor()

annomat_H3K27me3 &lt;- data.frame(sample=colnames(first_H3K27me3_counts_cor)) %&gt;% 
  separate_wider_delim(sample,delim=&quot;_&quot;,names=c(&quot;ind&quot;,&quot;trt&quot;,&quot;time&quot;,NA),cols_remove = FALSE) %&gt;% 
  mutate(trt=factor(trt, levels = c(&quot;VEH&quot;,&quot;5FU&quot;,&quot;DOX&quot;)),
         time=factor(time, levels =c(&quot;24t&quot;,&quot;24r&quot;,&quot;144r&quot;))) %&gt;% 
  column_to_rownames(&quot;sample&quot;)
heatmap_first_H3K27me3 &lt;- ComplexHeatmap::HeatmapAnnotation(df = annomat_H3K27me3)

Heatmap(first_H3K27me3_counts_cor, 
        top_annotation = heatmap_first_H3K27me3,
        column_title=&quot;Unfiltered log2cpm H3K27me3&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/H3K27me3%20heatmap-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-H3K27me3-heatmap-1">
Past versions of "H3K27me3 heatmap-1.png"
</button>
</p>
<div id="fig-no-spaces-H3K27me3-heatmap-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/H3K27me3 heatmap-1.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Heatmap(first_H3K27me3_counts_cor_noM, 
        top_annotation = heatmap_first_H3K27me3,
        column_title=&quot;Unfiltered log2cpm H3K27me3 no mitoreads&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/H3K27me3%20heatmap-2.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-H3K27me3-heatmap-2">
Past versions of "H3K27me3 heatmap-2.png"
</button>
</p>
<div id="fig-no-spaces-H3K27me3-heatmap-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/H3K27me3 heatmap-2.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="h3k9me3-count-analysis" class="section level3">
<h3>H3K9me3 count analysis</h3>
<pre class="r"><code>count_H3K9me3 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/H3K9me3_counts.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)

noM_count_H3K9me3 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/noM_count_H3K9me3&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)


Fragment_H3K9me3_txt &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/H3K9me3_counts.txt.summary&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)

Fragment_H3K9me3_txt_noM &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/noM_count_H3K9me3.summary&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)
rename_list &lt;- sampleinfo %&gt;% 
  mutate(stem= &quot;_final.bam&quot;) %&gt;% 
  mutate(oldname=paste0(library_ID,stem)) %&gt;% 
  mutate(newname=paste0(ind,&quot;_&quot;,trt,&quot;_&quot;,other_time,&quot;_&quot;,histone_mark)) %&gt;% 
  dplyr::select(oldname,newname)
  
rename_vec &lt;- setNames(rename_list$newname, rename_list$oldname)

names(count_H3K9me3)[names(count_H3K9me3) %in% names(rename_vec)] &lt;- rename_vec[names(count_H3K9me3)[names(count_H3K9me3) %in% names(rename_vec)]]
names(noM_count_H3K9me3)[names(noM_count_H3K9me3) %in% names(rename_vec_noM)] &lt;- rename_vec_noM[names(noM_count_H3K9me3)[names(noM_count_H3K9me3) %in% names(rename_vec_noM)]]</code></pre>
<pre class="r"><code>H3K9me3_counts_raw &lt;- count_H3K9me3 %&gt;% 
  dplyr::select(Geneid,`1_DOX_24t_H3K9me3`:`5_5FU_144r_H3K9me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()

H3K9me3_counts_raw &lt;- noM_count_H3K9me3 %&gt;% 
  dplyr::select(Geneid,`1_DOX_24t_H3K9me3`:`5_5FU_144r_H3K9me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()



first_H3K9me3_counts_cor &lt;- count_H3K9me3 %&gt;% 
  dplyr::select(Geneid,`1_DOX_24t_H3K9me3`:`5_5FU_144r_H3K9me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  cpm(., log = TRUE) %&gt;% 
  cor()
first_H3K9me3_counts_cor_noM &lt;- noM_count_H3K9me3 %&gt;% 
  dplyr::select(Geneid,`1_DOX_24t_H3K9me3`:`5_5FU_144r_H3K9me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  cpm(., log = TRUE) %&gt;% 
  cor()

annomat_H3K9me3 &lt;- data.frame(sample=colnames(first_H3K9me3_counts_cor)) %&gt;% 
  separate_wider_delim(sample,delim=&quot;_&quot;,names=c(&quot;ind&quot;,&quot;trt&quot;,&quot;time&quot;,NA),cols_remove = FALSE) %&gt;% 
  mutate(trt=factor(trt, levels = c(&quot;VEH&quot;,&quot;5FU&quot;,&quot;DOX&quot;)),
         time=factor(time, levels =c(&quot;24t&quot;,&quot;24r&quot;,&quot;144r&quot;))) %&gt;% 
  column_to_rownames(&quot;sample&quot;)
heatmap_first_H3K9me3 &lt;- ComplexHeatmap::HeatmapAnnotation(df = annomat_H3K9me3)

Heatmap(first_H3K9me3_counts_cor, 
        top_annotation = heatmap_first_H3K9me3,
        column_title=&quot;Unfiltered log2cpm H3K9me3&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/H3K9me3%20heatmap-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-H3K9me3-heatmap-1">
Past versions of "H3K9me3 heatmap-1.png"
</button>
</p>
<div id="fig-no-spaces-H3K9me3-heatmap-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/H3K9me3 heatmap-1.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Heatmap(first_H3K9me3_counts_cor_noM, 
        top_annotation = heatmap_first_H3K9me3,
        column_title=&quot;Unfiltered log2cpm H3K9me3 without mito reads&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/H3K9me3%20heatmap-2.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-H3K9me3-heatmap-2">
Past versions of "H3K9me3 heatmap-2.png"
</button>
</p>
<div id="fig-no-spaces-H3K9me3-heatmap-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/H3K9me3 heatmap-2.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="h3k36me3-count-analysis" class="section level3">
<h3>H3K36me3 count analysis</h3>
<pre class="r"><code>count_H3K36me3 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/H3K36me3_counts.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)
noM_count_H3K36me3 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/noM_count_H3K36me3&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)

Fragment_H3K36me3_txt &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/H3K36me3_counts.txt.summary&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)

Fragment_H3K36me3_txt_noM &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/noM_count_H3K36me3.summary&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)
rename_list &lt;- sampleinfo %&gt;% 
  mutate(stem= &quot;_final.bam&quot;) %&gt;% 
  mutate(oldname=paste0(library_ID,stem)) %&gt;% 
  mutate(newname=paste0(ind,&quot;_&quot;,trt,&quot;_&quot;,other_time,&quot;_&quot;,histone_mark)) %&gt;% 
  dplyr::select(oldname,newname)
  
rename_vec &lt;- setNames(rename_list$newname, rename_list$oldname)

names(count_H3K36me3)[names(count_H3K36me3) %in% names(rename_vec)] &lt;- rename_vec[names(count_H3K36me3)[names(count_H3K36me3) %in% names(rename_vec)]]
names(noM_count_H3K36me3)[names(noM_count_H3K36me3) %in% names(rename_vec_noM)] &lt;- rename_vec_noM[names(noM_count_H3K36me3)[names(noM_count_H3K36me3) %in% names(rename_vec_noM)]]</code></pre>
<pre class="r"><code>H3K36me3_counts_raw &lt;- count_H3K36me3 %&gt;% 
  dplyr::select(Geneid,`1_5FU_24t_H3K36me3`:`5_VEH_144r_H3K36me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()
H3K36me3_counts_raw_noM &lt;- noM_count_H3K36me3 %&gt;% 
  dplyr::select(Geneid,`1_5FU_24t_H3K36me3`:`5_VEH_144r_H3K36me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()


first_H3K36me3_counts_cor &lt;- count_H3K36me3 %&gt;% 
  dplyr::select(Geneid,`1_5FU_24t_H3K36me3`:`5_VEH_144r_H3K36me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  cpm(., log = TRUE) %&gt;% 
  cor()

first_H3K36me3_counts_cor_noM &lt;- noM_count_H3K36me3 %&gt;% 
  dplyr::select(Geneid,`1_5FU_24t_H3K36me3`:`5_VEH_144r_H3K36me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  cpm(., log = TRUE) %&gt;% 
  cor()

annomat_H3K36me3 &lt;- data.frame(sample=colnames(first_H3K36me3_counts_cor)) %&gt;% 
  separate_wider_delim(sample,delim=&quot;_&quot;,names=c(&quot;ind&quot;,&quot;trt&quot;,&quot;time&quot;,NA),cols_remove = FALSE) %&gt;% 
  mutate(trt=factor(trt, levels = c(&quot;VEH&quot;,&quot;5FU&quot;,&quot;DOX&quot;)),
         time=factor(time, levels =c(&quot;24t&quot;,&quot;24r&quot;,&quot;144r&quot;))) %&gt;% 
  column_to_rownames(&quot;sample&quot;)
heatmap_first_H3K36me3 &lt;- ComplexHeatmap::HeatmapAnnotation(df = annomat_H3K36me3)

Heatmap(first_H3K36me3_counts_cor, 
        top_annotation = heatmap_first_H3K36me3,
        column_title=&quot;Unfiltered log2cpm H3K36me3&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/H3K36me3%20heatmap-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-H3K36me3-heatmap-1">
Past versions of "H3K36me3 heatmap-1.png"
</button>
</p>
<div id="fig-no-spaces-H3K36me3-heatmap-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/H3K36me3 heatmap-1.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Heatmap(first_H3K36me3_counts_cor_noM, 
        top_annotation = heatmap_first_H3K36me3,
        column_title=&quot;Unfiltered log2cpm H3K36me3 without mito reads&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/H3K36me3%20heatmap-2.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-H3K36me3-heatmap-2">
Past versions of "H3K36me3 heatmap-2.png"
</button>
</p>
<div id="fig-no-spaces-H3K36me3-heatmap-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/H3K36me3 heatmap-2.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="fragment-count-per-sample" class="section level3">
<h3>Fragment count per sample</h3>
<pre class="r"><code>H3K9me3_Frag_count &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3_Frag_count.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %&gt;% 
  mutate(histone=&quot;H3K9me3&quot;) %&gt;% 
  dplyr::rename(&quot;sample&quot;=X1,&quot;count&quot;=X2)

H3K27me3_Frag_count &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3_Frag_count.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %&gt;% 
  mutate(histone=&quot;H3K27me3&quot;) %&gt;% 
  dplyr::rename(&quot;sample&quot;=X1,&quot;count&quot;=X2)

H3K27ac_Frag_count &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac_Frag_count.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %&gt;% 
  mutate(histone=&quot;H3K27ac&quot;) %&gt;% 
  dplyr::rename(&quot;sample&quot;=X1,&quot;count&quot;=X2)

H3K36me3_Frag_count &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3_Frag_count.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %&gt;% 
  mutate(histone=&quot;H3K36me3&quot;) %&gt;% 
  dplyr::rename(&quot;sample&quot;=X1,&quot;count&quot;=X2)
Frag_counts &lt;- H3K27ac_Frag_count %&gt;% 
  rbind(H3K27me3_Frag_count) %&gt;% 
  rbind(H3K36me3_Frag_count) %&gt;% 
  rbind(H3K9me3_Frag_count) %&gt;% 
  mutate(sample=gsub(&quot;_noM_final.bam&quot;,&quot;-noM&quot;,sample)) %&gt;% 
  mutate(sample=gsub(&quot;_final.bam&quot;,&quot;-wiM&quot;,sample)) %&gt;% 
  separate_wider_delim(.,cols=sample,delim= &quot;-&quot;,names = c(&quot;sample&quot;,&quot;MT_content&quot;)) %&gt;% 
  mutate(count=count/2)

Frag_counts %&gt;% 
  dplyr::filter(MT_content==&quot;noM&quot;) %&gt;% 
  mutate(count=count/1000000) %&gt;% 
  left_join(., sampleinfo,by=c(&quot;sample&quot;=&quot;library_ID&quot;,&quot;histone&quot;=&quot;histone_mark&quot;)) %&gt;% 
  ggplot(., aes(x=interaction(ind,trt,other_time), y=count, fill=trt, group = trt))+
  geom_col()+
  geom_hline(yintercept =5)+
  geom_text(aes(y = 0,label = sample), vjust = 0.2, size = 3, angle = 90)+

   theme_classic()+
  facet_wrap(~histone)+
  ggtitle(&quot;Fragment count by histone and sample&quot;)+
  ylab(&quot;Count of Fragments * 10^6&quot;)+
  xlab(&quot;samples&quot;)+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/fragment%20counts-1.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-fragment-counts-1">
Past versions of "fragment counts-1.png"
</button>
</p>
<div id="fig-no-spaces-fragment-counts-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/53be528d21140ebc2c5883f7a2ec5ba3151452ec/docs/figure/QC_seq_analysis.Rmd/fragment counts-1.png" target="_blank">53be528</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="fragment-width" class="section level4">
<h4>Fragment Width</h4>
<pre class="r"><code>histone_list &lt;- c(&quot;H3K27ac&quot;, &quot;H3K27me3&quot;, &quot;H3K9me3&quot;, &quot;H3K36me3&quot;)
base_path &lt;- &quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data&quot;

get_peak_widths_long_split &lt;- function(histone_name, base_path) {
  path &lt;- file.path(base_path, histone_name, &quot;macs2_output&quot;)
  
  file_list &lt;- list.files(path = path, pattern = &quot;\\.(narrowPeak|broadPeak)$&quot;, full.names = TRUE)
  
  # Exclude files with &#39;all_peaks&#39; in the name
  file_list &lt;- file_list[!grepl(&quot;all_peaks&quot;, basename(file_list), ignore.case = TRUE)]
  
  # Split into noM and not noM groups
  noM_files &lt;- file_list[grepl(&quot;noM&quot;, basename(file_list))]
  reg_files &lt;- file_list[!grepl(&quot;noM&quot;, basename(file_list))]

  # Define internal peak processing function
  process_peaks &lt;- function(file_path, group_label) {
    filename &lt;- basename(file_path)
    peak_df &lt;- fread(file_path, header = FALSE) %&gt;%
      transmute(
        row = row_number(),
        width = abs(V3 - V2),
        file = paste0(histone_name, &quot;-&quot;, filename),
        group = group_label
      )
    return(peak_df)
  }

  # Process each group
  peak_list &lt;- list(
    map(noM_files, process_peaks, group_label = &quot;noM&quot;),
    map(reg_files, process_peaks, group_label = &quot;regular&quot;)
  ) %&gt;% flatten()

  bind_rows(peak_list)
}</code></pre>
<pre class="r"><code># To apply to all histones:
all_peak_widths &lt;- purrr::map_df(histone_list, get_peak_widths_long_split, base_path = base_path)


anno_peak_width_long &lt;- all_peak_widths %&gt;% 
  dplyr::filter(group!=&quot;noM&quot;) %&gt;% 
    mutate(file= gsub(&quot;_noM_peaks.narrowPeak&quot;,&quot;&quot;,file)) %&gt;% 
    mutate(file= gsub(&quot;_noM_peaks.broadPeak&quot;,&quot;&quot;,file)) %&gt;% 
  separate_wider_delim(., cols=file, delim=&quot;-&quot;,names=c(&quot;histone&quot;,&quot;sample&quot;)) %&gt;% 
  left_join(sampleinfo, by =c(&quot;sample&quot;=&quot;library_ID&quot;))

anno_peak_width_long_noM &lt;-
  all_peak_widths %&gt;% 
  dplyr::filter(group==&quot;noM&quot;) %&gt;% 
    mutate(file= gsub(&quot;_noM_peaks.narrowPeak&quot;,&quot;&quot;,file)) %&gt;% 
    mutate(file= gsub(&quot;_noM_peaks.broadPeak&quot;,&quot;&quot;,file)) %&gt;% 
  separate_wider_delim(., cols=file, delim=&quot;-&quot;,names=c(&quot;histone&quot;,&quot;sample&quot;)) %&gt;% 
  left_join(sampleinfo, by =c(&quot;sample&quot;=&quot;library_ID&quot;))</code></pre>
<pre class="r"><code>anno_peak_width_long %&gt;%
  # distinct(histone)
  ggplot(.,aes(x=histone, y = width, fill = histone))+
  geom_violin()+
  # scale_fill_viridis_a(discrete = TRUE, begin = 0.1, end = 0.55, option = &quot;magma&quot;, alpha = 0.8) +
  #   scale_color_viridis_a(discrete = TRUE, begin = 0.1, end = 0.9) +
    scale_y_continuous(trans = &quot;log&quot;, breaks = c(400, 3000, 22000)) +
    theme_bw(base_size = 18) +
    ylab(&quot;Width of Peaks&quot;) +
    xlab(&quot;&quot;)+
  ggtitle(&quot;Width of all peaks with MT reads&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/unnamed-chunk-9-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-9-1">
Past versions of unnamed-chunk-9-1.png
</button>
</p>
<div id="fig-unnamed-chunk-9-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/unnamed-chunk-9-1.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>anno_peak_width_long_noM %&gt;%
  # distinct(histone)
  ggplot(.,aes(x=histone, y = width, fill = histone))+
  geom_violin()+
  # scale_fill_viridis_a(discrete = TRUE, begin = 0.1, end = 0.55, option = &quot;magma&quot;, alpha = 0.8) +
  #   scale_color_viridis_a(discrete = TRUE, begin = 0.1, end = 0.9) +
    scale_y_continuous(trans = &quot;log&quot;, breaks = c(400, 3000, 22000)) +
    theme_bw(base_size = 18) +
    ylab(&quot;Width of Peaks&quot;) +
    xlab(&quot;&quot;)+
  ggtitle(&quot;Width of all peaks without MT reads&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/unnamed-chunk-9-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-9-2">
Past versions of unnamed-chunk-9-2.png
</button>
</p>
<div id="fig-unnamed-chunk-9-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/unnamed-chunk-9-2.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>The smaller fragment bits, specifically in H3K26 and K3k9me3 made me
realize something was up. After visualizing the top ranked peaks in some
of the peak files by sample, I became aware of chrM contamination. Below
this mark is me trying to remap/and filter out the non-standard
chromosomes.</p>
<p>Code I used to check top peaks on files:</p>
<pre class="bash"><code>sort -k5,5rn MCW_CT_ROA_215_peaks.narrowPeak | head -n 30
</code></pre>
<pre class="r"><code>Fragment_H3K36me3_txt %&gt;% 
  pivot_longer(., cols=!Status, names_to = &quot;sample&quot;, values_to =&quot;count&quot;) %&gt;% 
  pivot_wider(., id_cols = sample, names_from = Status, values_from = count) %&gt;% 
  rowwise() %&gt;% 
  mutate(total = sum(c_across(Assigned:Unassigned_Ambiguity))) %&gt;% 
  dplyr::select(sample,Assigned, Unassigned_NoFeatures,total) %&gt;% 
  mutate(perc_assign= sprintf(&quot;%.2f&quot;,(Assigned/total)*100)) %&gt;% 
  mutate(sample= gsub(&quot;_final.bam&quot;,&quot;&quot;,sample)) %&gt;% 
  left_join(., sampleinfo, by = c(&quot;sample&quot;=&quot;library_ID&quot;)) %&gt;% 
  mutate(other_time=factor(other_time, levels = c (&quot;24t&quot;, &quot;24r&quot;, &quot;144r&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels = c(&quot;VEH&quot;,&quot;DOX&quot;,&quot;5FU&quot;))) %&gt;% 
  mutate(perc_assign=as.numeric(perc_assign)) %&gt;% 
  ggplot(., aes(x=interaction(ind,trt,other_time), y=perc_assign, fill=trt, group = trt))+
  geom_col()+
  geom_text(aes(y = 25,label = sample), vjust = 0.2, size = 3, angle = 90)+

   theme_classic()+
  ggtitle(&quot;FRIP for H3K36me3&quot;)+
  ylab(&quot;Fraction of reads in peaks * 100&quot;)+
  xlab(&quot;samples&quot;)+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/peakwidth%20full%20filtered%20peaks-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-peakwidth-full-filtered-peaks-1">
Past versions of "peakwidth full filtered peaks-1.png"
</button>
</p>
<div id="fig-no-spaces-peakwidth-full-filtered-peaks-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/peakwidth full filtered peaks-1.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>##############################noM
Fragment_H3K36me3_txt_noM %&gt;% 
  pivot_longer(., cols=!Status, names_to = &quot;sample&quot;, values_to =&quot;count&quot;) %&gt;% 
  pivot_wider(., id_cols = sample, names_from = Status, values_from = count) %&gt;% 
  rowwise() %&gt;% 
  mutate(total = sum(c_across(Assigned:Unassigned_Ambiguity))) %&gt;% 
  dplyr::select(sample,Assigned, Unassigned_NoFeatures,total) %&gt;% 
  mutate(perc_assign= sprintf(&quot;%.2f&quot;,(Assigned/total)*100)) %&gt;% 
  mutate(sample= gsub(&quot;_noM_final.bam&quot;,&quot;&quot;,sample)) %&gt;% 
  left_join(., sampleinfo, by = c(&quot;sample&quot;=&quot;library_ID&quot;)) %&gt;% 
  mutate(other_time=factor(other_time, levels = c (&quot;24t&quot;, &quot;24r&quot;, &quot;144r&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels = c(&quot;VEH&quot;,&quot;DOX&quot;,&quot;5FU&quot;))) %&gt;% 
  mutate(perc_assign=as.numeric(perc_assign)) %&gt;% 
  ggplot(., aes(x=interaction(ind,trt,other_time), y=perc_assign, fill=trt, group = trt))+
  geom_col()+
  geom_text(aes(y = 25,label = sample), vjust = 0.2, size = 3, angle = 90)+

   theme_classic()+
  ggtitle(&quot;FRIP for H3K36me3 without mt reads&quot;)+
  ylab(&quot;Fraction of reads in peaks * 100&quot;)+
  xlab(&quot;samples&quot;)+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/peakwidth%20full%20filtered%20peaks-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-peakwidth-full-filtered-peaks-2">
Past versions of "peakwidth full filtered peaks-2.png"
</button>
</p>
<div id="fig-no-spaces-peakwidth-full-filtered-peaks-2"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/peakwidth full filtered peaks-2.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="total-number-of-reads-by-sample" class="section level5">
<h5>Total number of reads by sample</h5>
<pre class="r"><code>Fragment_H3K36me3_txt_noM %&gt;% 
  pivot_longer(., cols=!Status, names_to = &quot;sample&quot;, values_to =&quot;count&quot;) %&gt;% 
  pivot_wider(., id_cols = sample, names_from = Status, values_from = count) %&gt;% 
  rowwise() %&gt;% 
  mutate(total = sum(c_across(Assigned:Unassigned_Ambiguity))) %&gt;% 
  dplyr::select(sample,Assigned, Unassigned_NoFeatures,total) %&gt;% 
  mutate(perc_assign= sprintf(&quot;%.2f&quot;,(Assigned/total)*100)) %&gt;% 
  mutate(graphed=total/1000000) %&gt;% 
  mutate(sample= gsub(&quot;_noM_final.bam&quot;,&quot;&quot;,sample)) %&gt;% 
  left_join(., sampleinfo, by = c(&quot;sample&quot;=&quot;library_ID&quot;)) %&gt;% 
  mutate(other_time=factor(other_time, levels = c (&quot;24t&quot;, &quot;24r&quot;, &quot;144r&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels = c(&quot;VEH&quot;,&quot;DOX&quot;,&quot;5FU&quot;))) %&gt;% 
  mutate(perc_assign=as.numeric(perc_assign)) %&gt;% 
  ggplot(., aes(x=interaction(ind,trt,other_time), y=graphed, fill=trt, group = trt))+
  geom_col()+
  geom_text(aes(y = 5,label = sample), vjust = 0.2, size = 3, angle = 90)+
   theme_classic()+
  ggtitle(&quot;Number of mapped reads by sample H3K36me3&quot;)+
  ylab(&quot;Count of total mapped reads X 10^6&quot;)+
  xlab(&quot;samples&quot;)+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/reads%20in%20peaks%20across%20samples%20H3K36me3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-reads-in-peaks-across-samples-H3K36me3-1">
Past versions of "reads in peaks across samples H3K36me3-1.png"
</button>
</p>
<div id="fig-no-spaces-reads-in-peaks-across-samples-H3K36me3-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/53be528d21140ebc2c5883f7a2ec5ba3151452ec/docs/figure/QC_seq_analysis.Rmd/reads in peaks across samples H3K36me3-1.png" target="_blank">53be528</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Fragment_H3K27me3_txt %&gt;% 
  pivot_longer(., cols=!Status, names_to = &quot;sample&quot;, values_to =&quot;count&quot;) %&gt;% 
  pivot_wider(., id_cols = sample, names_from = Status, values_from = count) %&gt;% 
  rowwise() %&gt;% 
  mutate(total = sum(c_across(Assigned:Unassigned_Ambiguity))) %&gt;% 
  dplyr::select(sample,Assigned, Unassigned_NoFeatures,total) %&gt;% 
  mutate(perc_assign= sprintf(&quot;%.2f&quot;,(Assigned/total)*100)) %&gt;% 
  mutate(sample= gsub(&quot;_final.bam&quot;,&quot;&quot;,sample)) %&gt;% 
  left_join(., sampleinfo, by = c(&quot;sample&quot;=&quot;library_ID&quot;)) %&gt;% 
  mutate(other_time=factor(other_time, levels = c (&quot;24t&quot;, &quot;24r&quot;, &quot;144r&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels = c(&quot;VEH&quot;,&quot;DOX&quot;,&quot;5FU&quot;))) %&gt;% 
  mutate(perc_assign=as.numeric(perc_assign)) %&gt;% 
  ggplot(., aes(x=interaction(ind,trt,other_time), y=perc_assign, fill=trt, group = trt))+
  geom_col()+
  geom_text(aes(y = 25,label = sample), vjust = 0.2, size = 3, angle = 90)+
   theme_classic()+
  ylab(&quot;Fraction of reads in peaks * 100&quot;)+
  xlab(&quot;samples&quot;)+
  ggtitle(&quot;FRIP for H3K27me3&quot;)+ theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/frip%20H3K27me3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-frip-H3K27me3-1">
Past versions of "frip H3K27me3-1.png"
</button>
</p>
<div id="fig-no-spaces-frip-H3K27me3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/frip H3K27me3-1.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>######################noM################################3
Fragment_H3K27me3_txt_noM %&gt;% 
  pivot_longer(., cols=!Status, names_to = &quot;sample&quot;, values_to =&quot;count&quot;) %&gt;% 
  pivot_wider(., id_cols = sample, names_from = Status, values_from = count) %&gt;% 
  rowwise() %&gt;% 
  mutate(total = sum(c_across(Assigned:Unassigned_Ambiguity))) %&gt;% 
  dplyr::select(sample,Assigned, Unassigned_NoFeatures,total) %&gt;% 
  mutate(perc_assign= sprintf(&quot;%.2f&quot;,(Assigned/total)*100)) %&gt;% 
  mutate(sample= gsub(&quot;_noM_final.bam&quot;,&quot;&quot;,sample)) %&gt;% 
  left_join(., sampleinfo, by = c(&quot;sample&quot;=&quot;library_ID&quot;)) %&gt;% 
  mutate(other_time=factor(other_time, levels = c (&quot;24t&quot;, &quot;24r&quot;, &quot;144r&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels = c(&quot;VEH&quot;,&quot;DOX&quot;,&quot;5FU&quot;))) %&gt;% 
  mutate(perc_assign=as.numeric(perc_assign)) %&gt;% 
  ggplot(., aes(x=interaction(ind,trt,other_time), y=perc_assign, fill=trt, group = trt))+
  geom_col()+
  geom_text(aes(y = 25,label = sample), vjust = 0.2, size = 3, angle = 90)+
   theme_classic()+
  ylab(&quot;Fraction of reads in peaks * 100&quot;)+
  xlab(&quot;samples&quot;)+
  ggtitle(&quot;FRIP for H3K27me3 without mt reads&quot;)+ theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/frip%20H3K27me3-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-frip-H3K27me3-2">
Past versions of "frip H3K27me3-2.png"
</button>
</p>
<div id="fig-no-spaces-frip-H3K27me3-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/frip H3K27me3-2.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Fragment_H3K27me3_txt_noM %&gt;% 
 pivot_longer(., cols=!Status, names_to = &quot;sample&quot;, values_to =&quot;count&quot;) %&gt;% 
  pivot_wider(., id_cols = sample, names_from = Status, values_from = count) %&gt;% 
  rowwise() %&gt;% 
  mutate(total = sum(c_across(Assigned:Unassigned_Ambiguity))) %&gt;% 
  dplyr::select(sample,Assigned, Unassigned_NoFeatures,total) %&gt;% 
  mutate(perc_assign= sprintf(&quot;%.2f&quot;,(Assigned/total)*100)) %&gt;% 
  mutate(graphed=total/1000000) %&gt;% 
  mutate(sample= gsub(&quot;_noM_final.bam&quot;,&quot;&quot;,sample)) %&gt;% 
  left_join(., sampleinfo, by = c(&quot;sample&quot;=&quot;library_ID&quot;)) %&gt;% 
  mutate(other_time=factor(other_time, levels = c (&quot;24t&quot;, &quot;24r&quot;, &quot;144r&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels = c(&quot;VEH&quot;,&quot;DOX&quot;,&quot;5FU&quot;))) %&gt;% 
  mutate(perc_assign=as.numeric(perc_assign)) %&gt;% 
  ggplot(., aes(x=interaction(ind,trt,other_time), y=graphed, fill=trt, group = trt))+
  geom_col()+
  geom_text(aes(y = 10,label = sample), vjust = 0.2, size = 3, angle = 90)+
   theme_classic()+
  ggtitle(&quot;Number of mapped fragments by sample H3K27me3&quot;)+
  ylab(&quot;Count of total mapped Paired reads X 10^6&quot;)+
  xlab(&quot;samples&quot;)+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/total%20number%20of%20reads%20per%20sample%20H3K27me3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-total-number-of-reads-per-sample-H3K27me3-1">
Past versions of "total number of reads per sample H3K27me3-1.png"
</button>
</p>
<div id="fig-no-spaces-total-number-of-reads-per-sample-H3K27me3-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/53be528d21140ebc2c5883f7a2ec5ba3151452ec/docs/figure/QC_seq_analysis.Rmd/total number of reads per sample H3K27me3-1.png" target="_blank">53be528</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Fragment_H3K27ac_txt %&gt;% 
  pivot_longer(., cols=!Status, names_to = &quot;sample&quot;, values_to =&quot;count&quot;) %&gt;% 
  pivot_wider(., id_cols = sample, names_from = Status, values_from = count) %&gt;% 
  rowwise() %&gt;% 
  mutate(total = sum(c_across(Assigned:Unassigned_Ambiguity))) %&gt;% 
  dplyr::select(sample,Assigned, Unassigned_NoFeatures,total) %&gt;% 
  mutate(perc_assign= sprintf(&quot;%.2f&quot;,(Assigned/total)*100)) %&gt;% 
  mutate(sample= gsub(&quot;_final.bam&quot;,&quot;&quot;,sample)) %&gt;% 
  left_join(., sampleinfo, by = c(&quot;sample&quot;=&quot;library_ID&quot;)) %&gt;% 
  mutate(other_time=factor(other_time, levels = c (&quot;24t&quot;, &quot;24r&quot;, &quot;144r&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels = c(&quot;VEH&quot;,&quot;DOX&quot;,&quot;5FU&quot;))) %&gt;% 
  mutate(perc_assign=as.numeric(perc_assign)) %&gt;% 
  ggplot(., aes(x=interaction(ind,trt,other_time), y=perc_assign, fill=trt, group = trt))+
  geom_col()+
  geom_text(aes(y = 15,label = sample), vjust = 0.2, size = 3, angle = 90)+
   theme_classic()+
  ggtitle(&quot;FRIP for H3K27ac&quot;)+ theme(axis.text.x=element_text(vjust = .2,angle=90))+
  ylab(&quot;Fraction of reads in peaks * 100&quot;)+
  xlab(&quot;samples&quot;)+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/frip%20H3K27ac-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-frip-H3K27ac-1">
Past versions of "frip H3K27ac-1.png"
</button>
</p>
<div id="fig-no-spaces-frip-H3K27ac-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/frip H3K27ac-1.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#######################noM################################
Fragment_H3K27ac_txt_noM %&gt;% 
  pivot_longer(., cols=!Status, names_to = &quot;sample&quot;, values_to =&quot;count&quot;) %&gt;% 
  pivot_wider(., id_cols = sample, names_from = Status, values_from = count) %&gt;% 
  rowwise() %&gt;% 
  mutate(total = sum(c_across(Assigned:Unassigned_Ambiguity))) %&gt;% 
  dplyr::select(sample,Assigned, Unassigned_NoFeatures,total) %&gt;% 
  mutate(perc_assign= sprintf(&quot;%.2f&quot;,(Assigned/total)*100)) %&gt;% 
  mutate(sample= gsub(&quot;_noM_final.bam&quot;,&quot;&quot;,sample)) %&gt;% 
  left_join(., sampleinfo, by = c(&quot;sample&quot;=&quot;library_ID&quot;)) %&gt;% 
  mutate(other_time=factor(other_time, levels = c (&quot;24t&quot;, &quot;24r&quot;, &quot;144r&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels = c(&quot;VEH&quot;,&quot;DOX&quot;,&quot;5FU&quot;))) %&gt;% 
  mutate(perc_assign=as.numeric(perc_assign)) %&gt;% 
  ggplot(., aes(x=interaction(ind,trt,other_time), y=perc_assign, fill=trt, group = trt))+
  geom_col()+
  geom_text(aes(y = 15,label = sample), vjust = 0.2, size = 3, angle = 90)+
   theme_classic()+
  ggtitle(&quot;FRIP for H3K27ac without mt reads&quot;)+ theme(axis.text.x=element_text(vjust = .2,angle=90))+
  ylab(&quot;Fraction of reads in peaks * 100&quot;)+
  xlab(&quot;samples&quot;)+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/frip%20H3K27ac-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-frip-H3K27ac-2">
Past versions of "frip H3K27ac-2.png"
</button>
</p>
<div id="fig-no-spaces-frip-H3K27ac-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/frip H3K27ac-2.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>read_count_H3K27ac &lt;- Fragment_H3K27ac_txt_noM %&gt;% 
 pivot_longer(., cols=!Status, names_to = &quot;sample&quot;, values_to =&quot;count&quot;) %&gt;% 
  pivot_wider(., id_cols = sample, names_from = Status, values_from = count) %&gt;% 
  rowwise() %&gt;% 
  mutate(total = sum(c_across(Assigned:Unassigned_Ambiguity))) %&gt;% 
  dplyr::select(sample,Assigned, Unassigned_NoFeatures,total) %&gt;% 
  mutate(perc_assign= sprintf(&quot;%.2f&quot;,(Assigned/total)*100)) %&gt;% 
  mutate(graphed=total/1000000) %&gt;% 
  mutate(sample= gsub(&quot;_noM_final.bam&quot;,&quot;&quot;,sample)) %&gt;% 
  left_join(., sampleinfo, by = c(&quot;sample&quot;=&quot;library_ID&quot;)) %&gt;% 
  mutate(other_time=factor(other_time, levels = c (&quot;24t&quot;, &quot;24r&quot;, &quot;144r&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels = c(&quot;VEH&quot;,&quot;DOX&quot;,&quot;5FU&quot;))) %&gt;% 
  mutate(perc_assign=as.numeric(perc_assign))

read_count_H3K27ac %&gt;% 
  ggplot(., aes(x=interaction(ind,trt,other_time), y=graphed, fill=trt, group = trt))+
  geom_col()+
  geom_text(aes(y = 10,label = sample), vjust = 0.2, size = 3, angle = 90)+
   theme_classic()+
  ggtitle(&quot;Number of mapped reads by sample H3K27ac&quot;)+
  ylab(&quot;Count of total mapped reads X 10^6&quot;)+
  xlab(&quot;samples&quot;)+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/reads%20per%20sample%20H3K27ac-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-reads-per-sample-H3K27ac-1">
Past versions of "reads per sample H3K27ac-1.png"
</button>
</p>
<div id="fig-no-spaces-reads-per-sample-H3K27ac-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/53be528d21140ebc2c5883f7a2ec5ba3151452ec/docs/figure/QC_seq_analysis.Rmd/reads per sample H3K27ac-1.png" target="_blank">53be528</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># read_count_H3K27ac %&gt;% dplyr::select(sample,perc_assign:trt, other_time)</code></pre>
</div>
</div>
</div>
<div id="peak-width-of-master-peaks" class="section level3">
<h3>Peak width of master peaks</h3>
<pre class="r"><code>H3K27ac_masterpeak &lt;- readBed(file=&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/macs2_output/filt_H3k27ac_all_merged.bed&quot;, remove.unusual = FALSE)

# H3K27ac_masterpeak %&gt;% 
#          as.data.frame() %&gt;% mutate(standard= if_else(seqnames %in% standard_chroms, &quot;yes&quot;,&quot;no&quot;)) %&gt;% group_by(standard) %&gt;% tally


H3K27me3_masterpeak &lt;- readBed(file=&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/macs2_output/filt_H3k27me3_all_merged.bed&quot;, remove.unusual = FALSE)
       # H3K27me3_masterpeak %&gt;% 
       #   as.data.frame() %&gt;% mutate(standard= if_else(seqnames %in% standard_chroms, &quot;yes&quot;,&quot;no&quot;)) %&gt;% group_by(standard) %&gt;% tally
       
H3K9me3_masterpeak &lt;- readBed(file=&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/macs2_output/filt_H3k9me3_all_merged.bed&quot;, remove.unusual = FALSE)
       # H3K9me3_masterpeak %&gt;% 
       #   as.data.frame() %&gt;% mutate(standard= if_else(seqnames %in% standard_chroms, &quot;yes&quot;,&quot;no&quot;)) %&gt;% group_by(standard) %&gt;% tally  
       
H3K36me3_masterpeak &lt;- readBed(file=&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/macs2_output/filt_H3k36me3_all_merged.bed&quot;, remove.unusual = FALSE)
       # H3K36me3_masterpeak %&gt;% 
       #   as.data.frame() %&gt;% mutate(standard= if_else(seqnames %in% standard_chroms, &quot;yes&quot;,&quot;no&quot;)) %&gt;% group_by(standard) %&gt;% tally         
       


master_peak_width &lt;- data_frame(histone_width=width(H3K27me3_masterpeak),histone=&quot;H3K27me3&quot;) %&gt;% 
  rbind(data_frame(histone_width=width(H3K27ac_masterpeak),histone=&quot;H3K27ac&quot;)) %&gt;% 
  rbind(data_frame(histone_width=width(H3K9me3_masterpeak),histone=&quot;H3K9me3&quot;)) %&gt;% 
  rbind(data_frame(histone_width=width(H3K36me3_masterpeak), histone = &quot;H3K36me3&quot;))

master_peak_width %&gt;% 
  ggplot(.,aes(x=histone, y = histone_width, fill = histone))+
  geom_violin()+
    scale_y_continuous(trans = &quot;log&quot;, breaks = c(400, 3000, 22000)) +
    theme_bw(base_size = 18) +
    ylab(&quot;Width of Peaks&quot;) +
    xlab(&quot;&quot;)+
  ggtitle(&quot;Master peak list widths with mt reads&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/Width%20of%20master%20peaks-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Width-of-master-peaks-1">
Past versions of "Width of master peaks-1.png"
</button>
</p>
<div id="fig-no-spaces-Width-of-master-peaks-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/Width of master peaks-1.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>H3K27ac_masterpeak_noM &lt;- readBed(file=&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/macs2_output/filt_H3k27ac_noM_all_merged.bed&quot;, remove.unusual = FALSE)

H3K27me3_masterpeak_noM &lt;- readBed(file=&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/macs2_output/filt_H3k27me3_noM_all_merged.bed&quot;, remove.unusual = FALSE)
       
H3K9me3_masterpeak_noM &lt;- readBed(file=&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/macs2_output/filt_H3k9me3_noM_all_merged.bed&quot;, remove.unusual = FALSE)
       
       
H3K36me3_masterpeak_noM &lt;- readBed(file=&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/macs2_output/filt_H3k36me3_noM_all_merged.bed&quot;, remove.unusual = FALSE)
      


master_peak_width_noM &lt;- data_frame(histone_width=width(H3K27me3_masterpeak_noM),histone=&quot;H3K27me3&quot;) %&gt;% 
  rbind(data_frame(histone_width=width(H3K27ac_masterpeak_noM),histone=&quot;H3K27ac&quot;)) %&gt;% 
  rbind(data_frame(histone_width=width(H3K9me3_masterpeak_noM),histone=&quot;H3K9me3&quot;)) %&gt;% 
  rbind(data_frame(histone_width=width(H3K36me3_masterpeak_noM), histone = &quot;H3K36me3&quot;))

master_peak_width_noM %&gt;% 
  ggplot(.,aes(x=histone, y = histone_width, fill = histone))+
  geom_violin()+
    scale_y_continuous(trans = &quot;log&quot;, breaks = c(400, 3000, 22000)) +
    theme_bw(base_size = 18) +
    ylab(&quot;Width of Peaks&quot;) +
    xlab(&quot;&quot;)+
  ggtitle(&quot;Master Peak list widths without MT reads&quot;)</code></pre>
<p><img src="figure/QC_seq_analysis.Rmd/unnamed-chunk-11-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-11-1">
Past versions of unnamed-chunk-11-1.png
</button>
</p>
<div id="fig-unnamed-chunk-11-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/53be528d21140ebc2c5883f7a2ec5ba3151452ec/docs/figure/QC_seq_analysis.Rmd/unnamed-chunk-11-1.png" target="_blank">53be528</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-16
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/24b82ffde1514e71a5eba5a0c444a4a678a414ad/docs/figure/QC_seq_analysis.Rmd/unnamed-chunk-11-1.png" target="_blank">24b82ff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Frag_counts %&gt;% 
  # dplyr::filter(MT_content==&quot;noM&quot;) %&gt;% 
  mutate(count=count/1000000) %&gt;% 
  pivot_wider(., id_cols = c(sample,histone),names_from = MT_content, values_from = count) %&gt;% 
  dplyr::rename(&quot;frag_with_mito&quot;=wiM, &quot;frag_no_mito&quot;=noM) %&gt;% 
  left_join(., sampleinfo,by=c(&quot;sample&quot;=&quot;library_ID&quot;,&quot;histone&quot;=&quot;histone_mark&quot;)) %&gt;% 
  left_join(., (combo_trim_df %&gt;% 
                   dplyr::select(prefix,read, `Total Sequences`) %&gt;%
                  pivot_wider(., id_cols = c(prefix), 
                              names_from = read,
                              values_from = &#39;Total Sequences&#39; ) %&gt;%
                  rowwise() %&gt;% 
                  mutate(&quot;comboR1_R2&quot;= sum(R1,R2))),
            by=c(&quot;sample&quot;=&quot;prefix&quot;)) %&gt;% 
  left_join(., for_plots, by=c(&quot;sample&quot;=&quot;sample_ID&quot;)) %&gt;%
  dplyr::rename(&quot;Total_R1R2&quot;=comboR1_R2, 
         &quot;frag_map_unique&quot; = aligned_concordant_1,
         &quot;frag_no_map&quot;=aligned_concordant_0,
         &quot;frag_more_than1map&quot;=aligned_concordant_g1) %&gt;% 
  mutate(renee_name=paste0(ind.y,&quot;_&quot;,trt.y,&quot;_&quot;,other_time.y)) %&gt;% 
  dplyr::select(sample,sample_ID,histone,Total_R1R2,Paired_Reads,percent_alignment,frag_map_unique,frag_more_than1map,frag_no_map,frag_with_mito, frag_no_mito,renee_name) %&gt;% 
  
  datatable(., options = list(scrollX = TRUE, 
                              scrollY = &quot;400px&quot;,
                              scrollCollapse = TRUE,
                              fixedColumns = list(leftColumns =2),
                              fixedHeader= TRUE),
            extensions = c(&quot;FixedColumns&quot;,&quot;Scroller&quot;),
            class = &quot;display&quot;)</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-daa9f702b0b8ec7e7d53" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-daa9f702b0b8ec7e7d53">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","Scroller"],"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94"],["MCW_CT_ROA_100","MCW_CT_ROA_101","MCW_CT_ROA_102","MCW_CT_ROA_107","MCW_CT_ROA_109","MCW_CT_ROA_112","MCW_CT_ROA_113","MCW_CT_ROA_114","MCW_CT_ROA_115","MCW_CT_ROA_116","MCW_CT_ROA_117","MCW_CT_ROA_119","MCW_CT_ROA_121","MCW_CT_ROA_78","MCW_CT_ROA_79","MCW_CT_ROA_80","MCW_CT_ROA_81","MCW_CT_ROA_83","MCW_CT_ROA_84","MCW_CT_ROA_85","MCW_CT_ROA_86","MCW_CT_ROA_87","MCW_CT_ROA_88","MCW_CT_ROA_89","MCW_CT_ROA_94","MCW_CT_ROA_95","MCW_CT_ROA_96","MCW_CT_ROA_97","MCW_CT_ROA_99","MCW_CT_ROA_124","MCW_CT_ROA_125","MCW_CT_ROA_127","MCW_CT_ROA_130","MCW_CT_ROA_138","MCW_CT_ROA_139","MCW_CT_ROA_140","MCW_CT_ROA_141","MCW_CT_ROA_142","MCW_CT_ROA_145","MCW_CT_ROA_146","MCW_CT_ROA_147","MCW_CT_ROA_148","MCW_CT_ROA_149","MCW_CT_ROA_156","MCW_CT_ROA_157","MCW_CT_ROA_158","MCW_CT_ROA_159","MCW_CT_ROA_163","MCW_CT_ROA_167","MCW_CT_ROA_171","MCW_CT_ROA_172","MCW_CT_ROA_173","MCW_CT_ROA_174","MCW_CT_ROA_175","MCW_CT_ROA_177","MCW_CT_ROA_180","MCW_CT_ROA_181","MCW_CT_ROA_183","MCW_CT_ROA_185","MCW_CT_ROA_189","MCW_CT_ROA_192","MCW_CT_ROA_194","MCW_CT_ROA_195","MCW_CT_ROA_202","MCW_CT_ROA_206","MCW_CT_ROA_207","MCW_CT_ROA_208","MCW_CT_ROA_209","MCW_CT_ROA_210","MCW_CT_ROA_211","MCW_CT_ROA_213","MCW_CT_ROA_215","MCW_CT_ROA_217","MCW_CT_ROA_218","MCW_CT_ROA_221","MCW_CT_ROA_223","MCW_CT_ROA_224","MCW_CT_ROA_226","MCW_CT_ROA_227","MCW_CT_ROA_228","MCW_CT_ROA_229","MCW_CT_ROA_230","MCW_CT_ROA_231","MCW_CT_ROA_232","MCW_CT_ROA_233","MCW_CT_ROA_236","MCW_CT_ROA_238","MCW_CT_ROA_239","MCW_CT_ROA_243","MCW_CT_ROA_245","MCW_CT_ROA_246","MCW_CT_ROA_247","MCW_CT_ROA_249","MCW_CT_ROA_254"],["Ind3_17-3_VEH_24+24","Ind3_17-3_5FU_24+144","Ind3_17-3_DOX_24+144","Ind4_90-1_5FU_24+24","Ind4_90-1_VEH_24+24","Ind4_90-1_VEH_24+144","Ind5_84-1_5FU_24","Ind5_84-1_DOX_24","Ind5_84-1_VEH_24","Ind5_84-1_5FU_24+24","Ind5_84-1_DOX_24+24","Ind5_84-1_5FU_24+144","Ind5_84-1_VEH_24+144","Ind1_78-1_DOX_24","Ind1_78-1_VEH_24","Ind1_78-1_5FU_24+24","Ind1_78-1_DOX_24+24","Ind1_78-1_5FU_24+144","Ind1_78-1_DOX_24+144","Ind1_78-1_VEH_24+144","Ind2_75-1_5FU_24","Ind2_75-1_DOX_24","Ind2_75-1_VEH_24","Ind2_75-1_5FU_24+24","Ind2_75-1_VEH_24+144","Ind3_17-3_5FU_24","Ind3_17-3_DOX_24","Ind3_17-3_VEH_24","Ind3_17-3_DOX_24+24","Ind1_78-1_VEH_24","Ind1_78-1_5FU_24+24","Ind1_78-1_VEH_24+24","Ind1_78-1_VEH_24+144","Ind2_75-1_DOX_24+144","Ind2_75-1_VEH_24+144","Ind3_17-3_5FU_24","Ind3_17-3_DOX_24","Ind3_17-3_VEH_24","Ind3_17-3_VEH_24+24","Ind3_17-3_5FU_24+144","Ind3_17-3_DOX_24+144","Ind3_17-3_VEH_24+144","Ind4_90-1_5FU_24","Ind4_90-1_DOX_24+144","Ind4_90-1_VEH_24+144","Ind5_84-1_5FU_24","Ind5_84-1_DOX_24","Ind5_84-1_VEH_24+24","Ind1_78-1_5FU_24","Ind1_78-1_DOX_24+24","Ind1_78-1_VEH_24+24","Ind1_78-1_5FU_24+144","Ind1_78-1_DOX_24+144","Ind1_78-1_VEH_24+144","Ind2_75-1_DOX_24","Ind2_75-1_DOX_24+24","Ind2_75-1_VEH_24+24","Ind2_75-1_DOX_24+144","Ind3_17-3_5FU_24","Ind3_17-3_DOX_24+24","Ind3_17-3_DOX_24+144","Ind4_90-1_5FU_24","Ind4_90-1_DOX_24","Ind4_90-1_VEH_24+144","Ind5_84-1_5FU_24+24","Ind5_84-1_DOX_24+24","Ind5_84-1_VEH_24+24","Ind5_84-1_5FU_24+144","Ind5_84-1_DOX_24+144","Ind5_84-1_VEH_24+144","Ind1_78-1_DOX_24","Ind1_78-1_5FU_24+24","Ind1_78-1_VEH_24+24","Ind1_78-1_5FU_24+144","Ind2_75-1_5FU_24","Ind2_75-1_VEH_24","Ind2_75-1_5FU_24+24","Ind2_75-1_VEH_24+24","Ind2_75-1_5FU_24+144","Ind2_75-1_DOX_24+144","Ind2_75-1_VEH_24+144","Ind3_17-3_5FU_24","Ind3_17-3_DOX_24","Ind3_17-3_VEH_24","Ind3_17-3_5FU_24+24","Ind3_17-3_5FU_24+144","Ind3_17-3_VEH_24+144","Ind4_90-1_5FU_24","Ind4_90-1_DOX_24+24","Ind4_90-1_5FU_24+144","Ind4_90-1_DOX_24+144","Ind4_90-1_VEH_24+144","Ind5_84-1_DOX_24","Ind5_84-1_5FU_24+144"],["H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27ac","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K27me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K36me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3","H3K9me3"],[19292808,11942502,17989056,28188928,27378536,25606824,24422154,18244118,23405742,22606010,17679418,21356052,29694330,17029074,34460820,22146510,20608322,27746944,18921658,35742620,14259444,21984030,12273802,20953908,14796968,18489208,16395150,13477494,16195052,11725906,27218492,18139524,20073458,20892212,21524350,17440938,24387042,14767338,20763680,24901930,15255262,16484266,33398742,23901902,27972240,17232180,16269700,13909732,20508874,30098054,22248276,25640734,11276696,1469674,20533500,28875790,25964076,22143914,20743000,18037890,19234630,26677654,28914724,15503886,16910920,22012608,11713698,22595648,14801486,17876034,26999162,24475702,27042496,31555544,23349554,24930050,15645834,22240750,23623712,11407888,21222068,17256604,15166030,17331972,22860356,19522258,20696768,22825948,21841612,21648076,25348896,23263756,19561916,14241594],[9646404,5971251,8994528,14094464,13689268,12803412,12211077,9122059,11702871,11303005,8839709,10678026,14847165,8514537,17230410,11073255,10304161,13873472,9460829,17871310,7129722,10992015,6136901,10476954,7398484,9244604,8197575,6738747,8097526,5862953,13609246,9069762,10036729,10446106,10762175,8720469,12193521,7383669,10381840,12450965,7627631,8242133,16699371,11950951,13986120,8616090,8134850,6954866,10254437,15049027,11124138,12820367,5638348,734837,10266750,14437895,12982038,11071957,10371500,9018945,9617315,13338827,14457362,7751943,8455460,11006304,5856849,11297824,7400743,8938017,13499581,12237851,13521248,15777772,11674777,12465025,7822917,11120375,11811856,5703944,10611034,8628302,7583015,8665986,11430178,9761129,10348384,11412974,10920806,10824038,12674448,11631878,9780958,7120797],["94.95%","97.11%","95.58%","96.13%","94.93%","95.43%","95.37%","94.64%","97.15%","95.42%","94.36%","92.76%","89.02%","95.37%","95.65%","95.36%","96.68%","96.42%","96.61%","96.59%","96.28%","96.37%","96.24%","96.64%","97.21%","94.23%","93.85%","95.68%","95.66%","98.07%","97.74%","97.67%","98.56%","98.53%","97.78%","97.96%","98.71%","98.19%","98.37%","97.88%","98.14%","98.44%","97.48%","97.63%","96.13%","98.44%","98.60%","97.34%","94.78%","97.60%","97.07%","97.51%","97.68%","96.13%","96.19%","97.47%","97.61%","97.06%","96.82%","96.88%","96.88%","97.40%","96.31%","96.95%","96.62%","95.41%","95.54%","93.99%","93.87%","94.42%","98.09%","97.72%","97.53%","97.81%","97.13%","97.45%","97.49%","96.87%","97.81%","98.08%","97.70%","98.10%","93.25%","97.38%","97.95%","98.25%","96.73%","97.59%","97.46%","96.70%","97.58%","97.58%","96.99%","95.01%"],[5746445,4279384,5895789,9669822,8788342,9061482,7290370,5678494,7501717,6793739,5496942,5451719,6092487,4454677,10921835,6094802,6909479,9607312,6281523,12887951,4998439,7180888,4367664,7372185,5202764,5462429,4828923,4109592,5337230,4569976,10291964,7035945,7735497,7973512,8064720,6784325,9284343,5801304,7791123,9158338,5873262,6363806,12588934,8870887,10211147,6537637,6121793,5222610,3892332,7562957,3611930,6889739,3468729,371197,4669078,6234032,5865365,4366138,4739964,4431648,4183944,5058731,5876489,3106251,4130128,5110713,2913687,4742105,3172790,3921761,7311755,5844416,6700522,8119847,5875284,6133660,3787480,5648597,5996517,3137321,5424718,4562488,3953510,4543469,5907899,5457706,5139928,5757179,5557589,5000814,5903745,5176881,5647927,3843884],[3413223,1519083,2701402,3878510,4206765,3157277,4354788,2954323,3867152,3991271,2844619,4452844,7124896,3665980,5558254,4464549,3052077,3769451,2858374,4373607,1866383,3412380,1538234,2753144,1989539,3248930,2864677,2338206,2408977,1179679,3009776,1822827,2157170,2318882,2458693,1758362,2751749,1448916,2421493,3028553,1612390,1749526,3690176,2796675,3233647,1944360,1899075,1547572,5827197,7124277,7186360,5611225,2038550,335206,5206356,7838641,6806673,6379813,5302043,4305605,5133772,7933514,8047379,4409135,4039621,5390099,2681847,5876858,3774613,4517362,5930133,6114717,6486501,7312326,5464650,6013452,3839306,5123472,5556105,2456931,4941746,3901991,3117425,3895199,5288033,4132460,4870533,5380755,5085599,5465753,6463449,6173774,3838198,2921907],[486736,172784,397337,546132,694161,584653,565919,489242,334002,517995,498148,773463,1629782,393880,750321,513904,342605,496709,320932,609752,264900,398747,231003,351625,206181,533245,503975,290949,351319,113298,307506,210990,144062,153712,238762,177782,157429,133449,169224,264074,141979,128801,420261,283389,541326,134093,113982,184684,534908,361793,325848,319403,131069,28434,391316,365222,310000,326006,329493,281692,299599,346582,533494,236557,285711,505492,261315,678861,453340,498894,257693,278718,334225,345599,334843,317913,196131,348306,259234,109692,244570,163823,512080,227318,234246,170963,337923,275040,277618,357471,307254,281223,294833,355006],[7.585611,5.114084,7.392188,11.623581,10.903407,10.669888,9.614352,7.171073,9.550421,8.853230999999999,6.931403,7.690792,9.714510000000001,6.234551,13.774172,8.38303,8.505501000000001,11.639544,7.854059,15.159263,5.968223,8.978376000000001,5.156794,8.812471,6.274504,7.200578,6.329208,5.402415,6.628307,5.131472,11.69309,7.862915,8.809119000000001,9.121584,9.224961,7.598297,10.668368,6.479865,9.008963,10.604817,6.619474,7.204345,14.2613,10.152228,11.63731,7.52577,7.114144,5.966159,6.920972,11.200265,7.329385,9.825475000000001,4.612758,0.550125,7.232579,10.388072,9.387342,7.655502,7.639592,6.817638,6.911457,9.015853,9.964430999999999,5.370801,6.151552,7.847745,4.333384,7.742554,5.061675,6.199063,10.127623,8.728175999999999,9.832698000000001,11.63466,8.53459,9.026515,5.610871,8.168642999999999,8.644461,4.37154,7.836103,6.477565,5.553532,6.497582,8.399236,7.440379,7.383286,8.282883,8.013000999999999,7.507981,8.769004000000001,7.761698,7.470093,5.290288],[5.98452,4.590166,6.26845,10.538283,9.419226999999999,9.886699999999999,7.866944,6.168308,8.527241,7.109534,5.719349,5.70626,5.987978,5.022635,11.796935,6.660882,7.716502,10.76382,7.015928,14.485685,5.43431,7.740852,4.699364,7.904141,5.746107,5.743313,5.194217,4.327721,5.673374,5.122628,11.652319,7.825767,8.792935,9.100077000000001,9.188527000000001,7.579713,10.65124,6.464286,8.988778999999999,10.57285,6.575777,7.186612,14.185407,10.110356,11.572242,7.507821,7.100565,5.943152,4.393605,8.75006,3.637251,8.259458,4.173002,0.476587,4.995655,6.701359,6.745445,4.60718,5.23014,5.052337,4.501846,5.195083,6.16066,3.396973,4.472214,5.588649,3.308931,5.022856,3.280323,4.251249,9.997521000000001,8.444775999999999,9.561945,11.408998,8.302038,8.807836999999999,5.34663,7.859336,8.333565999999999,4.199841,7.631265,6.296073,4.967105,6.211535,8.13578,7.295417,7.15338,8.047459999999999,7.634229,7.239798,8.567128,7.599675,7.346138,5.110576],["3_VEH_24r","3_5FU_144r","3_DOX_144r","4_5FU_24r","4_VEH_24r","4_VEH_144r","5_5FU_24t","5_DOX_24t","5_VEH_24t","5_5FU_24r","5_DOX_24r","5_5FU_144r","5_VEH_144r","1_DOX_24t","1_VEH_24t","1_5FU_24r","1_DOX_24r","1_5FU_144r","1_DOX_144r","1_VEH_144r","2_5FU_24t","2_DOX_24t","2_VEH_24t","2_5FU_24r","2_VEH_144r","3_5FU_24t","3_DOX_24t","3_VEH_24t","3_DOX_24r","1_VEH_24t","1_5FU_24r","1_VEH_24r","1_VEH_144r","2_DOX_144r","2_VEH_144r","3_5FU_24t","3_DOX_24t","3_VEH_24t","3_VEH_24r","3_5FU_144r","3_DOX_144r","3_VEH_144r","4_5FU_24t","4_DOX_144r","4_VEH_144r","5_5FU_24t","5_DOX_24t","5_VEH_24r","1_5FU_24t","1_DOX_24r","1_VEH_24r","1_5FU_144r","1_DOX_144r","1_VEH_144r","2_DOX_24t","2_DOX_24r","2_VEH_24r","2_DOX_144r","3_5FU_24t","3_DOX_24r","3_DOX_144r","4_5FU_24t","4_DOX_24t","4_VEH_144r","5_5FU_24r","5_DOX_24r","5_VEH_24r","5_5FU_144r","5_DOX_144r","5_VEH_144r","1_DOX_24t","1_5FU_24r","1_VEH_24r","1_5FU_144r","2_5FU_24t","2_VEH_24t","2_5FU_24r","2_VEH_24r","2_5FU_144r","2_DOX_144r","2_VEH_144r","3_5FU_24t","3_DOX_24t","3_VEH_24t","3_5FU_24r","3_5FU_144r","3_VEH_144r","4_5FU_24t","4_DOX_24r","4_5FU_144r","4_DOX_144r","4_VEH_144r","5_DOX_24t","5_5FU_144r"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>sample<\/th>\n      <th>sample_ID<\/th>\n      <th>histone<\/th>\n      <th>Total_R1R2<\/th>\n      <th>Paired_Reads<\/th>\n      <th>percent_alignment<\/th>\n      <th>frag_map_unique<\/th>\n      <th>frag_more_than1map<\/th>\n      <th>frag_no_map<\/th>\n      <th>frag_with_mito<\/th>\n      <th>frag_no_mito<\/th>\n      <th>renee_name<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"scrollY":"400px","scrollCollapse":true,"fixedColumns":{"leftColumns":2},"fixedHeader":true,"columnDefs":[{"className":"dt-right","targets":[4,5,7,8,9,10,11]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"sample","targets":1},{"name":"sample_ID","targets":2},{"name":"histone","targets":3},{"name":"Total_R1R2","targets":4},{"name":"Paired_Reads","targets":5},{"name":"percent_alignment","targets":6},{"name":"frag_map_unique","targets":7},{"name":"frag_more_than1map","targets":8},{"name":"frag_no_map","targets":9},{"name":"frag_with_mito","targets":10},{"name":"frag_no_mito","targets":11},{"name":"renee_name","targets":12}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # combo_trim_df %&gt;% 
  #   dplyr::select(prefix,read, `Total Sequences`) %&gt;% 
  #   pivot_wider(., id_cols = c(prefix), names_from = read,values_from = &#39;Total Sequences&#39; ) %&gt;% 
  #   rowwise() %&gt;% 
  #   mutate(&quot;comboR1_R2&quot;=sum(R1,R2))
  #   </code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.2 (2024-10-31 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 11 x64 (build 26100)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] stats4    grid      stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] GenomicRanges_1.58.0  GenomeInfoDb_1.42.3   IRanges_2.40.1       
 [4] S4Vectors_0.44.0      BiocGenerics_0.52.0   genomation_1.38.0    
 [7] data.table_1.17.0     ComplexHeatmap_2.22.0 edgeR_4.4.2          
[10] limma_3.62.2          DT_0.33               kableExtra_1.4.0     
[13] lubridate_1.9.4       forcats_1.0.0         stringr_1.5.1        
[16] dplyr_1.1.4           purrr_1.0.4           readr_2.1.5          
[19] tidyr_1.3.1           tibble_3.2.1          ggplot2_3.5.1        
[22] tidyverse_2.0.0       workflowr_1.7.1      

loaded via a namespace (and not attached):
  [1] RColorBrewer_1.1-3          rstudioapi_0.17.1          
  [3] jsonlite_1.9.1              shape_1.4.6.1              
  [5] magrittr_2.0.3              magick_2.8.5               
  [7] farver_2.1.2                rmarkdown_2.29             
  [9] GlobalOptions_0.1.2         fs_1.6.5                   
 [11] BiocIO_1.16.0               zlibbioc_1.52.0            
 [13] vctrs_0.6.5                 Rsamtools_2.22.0           
 [15] RCurl_1.98-1.16             rstatix_0.7.2              
 [17] htmltools_0.5.8.1           S4Arrays_1.6.0             
 [19] plotrix_3.8-4               curl_6.2.1                 
 [21] broom_1.0.7                 Formula_1.2-5              
 [23] SparseArray_1.6.2           sass_0.4.9                 
 [25] KernSmooth_2.23-26          bslib_0.9.0                
 [27] htmlwidgets_1.6.4           plyr_1.8.9                 
 [29] impute_1.80.0               cachem_1.1.0               
 [31] GenomicAlignments_1.42.0    whisker_0.4.1              
 [33] lifecycle_1.0.4             iterators_1.0.14           
 [35] pkgconfig_2.0.3             Matrix_1.7-3               
 [37] R6_2.6.1                    fastmap_1.2.0              
 [39] GenomeInfoDbData_1.2.13     MatrixGenerics_1.18.1      
 [41] clue_0.3-66                 digest_0.6.37              
 [43] colorspace_2.1-1            ps_1.9.0                   
 [45] rprojroot_2.0.4             crosstalk_1.2.1            
 [47] ggpubr_0.6.0                labeling_0.4.3             
 [49] timechange_0.3.0            httr_1.4.7                 
 [51] abind_1.4-8                 compiler_4.4.2             
 [53] bit64_4.6.0-1               withr_3.0.2                
 [55] doParallel_1.0.17           backports_1.5.0            
 [57] BiocParallel_1.40.0         carData_3.0-5              
 [59] ggsignif_0.6.4              DelayedArray_0.32.0        
 [61] rjson_0.2.23                tools_4.4.2                
 [63] httpuv_1.6.15               glue_1.8.0                 
 [65] restfulr_0.0.15             callr_3.7.6                
 [67] promises_1.3.2              getPass_0.2-4              
 [69] gridBase_0.4-7              cluster_2.1.8.1            
 [71] reshape2_1.4.4              generics_0.1.3             
 [73] gtable_0.3.6                BSgenome_1.74.0            
 [75] tzdb_0.4.0                  seqPattern_1.38.0          
 [77] hms_1.1.3                   car_3.1-3                  
 [79] xml2_1.3.7                  XVector_0.46.0             
 [81] foreach_1.5.2               pillar_1.10.1              
 [83] vroom_1.6.5                 later_1.4.1                
 [85] circlize_0.4.16             lattice_0.22-6             
 [87] rtracklayer_1.66.0          bit_4.6.0                  
 [89] tidyselect_1.2.1            locfit_1.5-9.12            
 [91] Biostrings_2.74.1           knitr_1.49                 
 [93] git2r_0.35.0                SummarizedExperiment_1.36.0
 [95] svglite_2.1.3               xfun_0.51                  
 [97] Biobase_2.66.0              statmod_1.5.0              
 [99] matrixStats_1.5.0           stringi_1.8.4              
[101] UCSC.utils_1.2.0            yaml_2.3.10                
[103] evaluate_1.0.3              codetools_0.2-20           
[105] cli_3.6.4                   systemfonts_1.2.1          
[107] munsell_0.5.1               processx_3.8.6             
[109] jquerylib_0.1.4             Rcpp_1.0.14                
[111] png_0.1-8                   XML_3.99-0.18              
[113] parallel_4.4.2              bitops_1.0-9               
[115] viridisLite_0.4.2           scales_1.3.0               
[117] crayon_1.5.3                GetoptLong_1.0.5           
[119] rlang_1.1.5                </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
