<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Renee Matthews" />

<meta name="date" content="2025-04-21" />

<title>Q value thresholding</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">DXR_project</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/reneeisnowhere/DXR_project">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Q value thresholding</h1>
<h4 class="author">Renee Matthews</h4>
<h4 class="date">2025-04-21</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-04-28
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>DXR_project/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20250313code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20250313)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20250313code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20250313)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereDXRprojecttreea08ec5ba3247fe9e14a6bc2d349e5df6c9c08064targetblanka08ec5ba">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/reneeisnowhere/DXR_project/tree/a08ec5ba3247fe9e14a6bc2d349e5df6c9c08064" target="_blank">a08ec5b</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereDXRprojecttreea08ec5ba3247fe9e14a6bc2d349e5df6c9c08064targetblanka08ec5ba"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/reneeisnowhere/DXR_project/tree/a08ec5ba3247fe9e14a6bc2d349e5df6c9c08064" target="_blank">a08ec5b</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/first_run_alignment_229andover.RDS
    Ignored:    data/first_run_alignment_specs.RDS
    Ignored:    data/multiqc_fastqc_notrim.txt
    Ignored:    data/multiqc_fastqc_trim.txt
    Ignored:    data/multiqc_genral_stats_notrim.txt
    Ignored:    data/multiqc_genral_stats_trim.txt
    Ignored:    data/prelim_DEG/
    Ignored:    data/sampleinfo.txt

Untracked files:
    Untracked:  DXR_project.Rproj
    Untracked:  analysis/prelim_DEG.Rmd
    Untracked:  multiqc_report_no_cut.html
    Untracked:  multiqc_report_trim.html

Unstaged changes:
    Deleted:    5FU_project.Rproj

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Thresholding.Rmd</code>) and HTML
(<code>docs/Thresholding.html</code>) files. If you’ve configured a
remote Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/a08ec5ba3247fe9e14a6bc2d349e5df6c9c08064/analysis/Thresholding.Rmd" target="_blank">a08ec5b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-28
</td>
<td>
adding more thresholds
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/DXR_project/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/Thresholding.html" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/10df05509dd6e6e274e75b01612aaedb6a9ed677/analysis/Thresholding.Rmd" target="_blank">10df055</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
<td>
updates on q05,q01,q005
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/DXR_project/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/Thresholding.html" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/f2d46a582d926b8eaa409b9d81f7b6b8d0c3b352/analysis/Thresholding.Rmd" target="_blank">f2d46a5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
<td>
first commit
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<div id="loading-packages" class="section level5">
<h5>Loading Packages</h5>
<pre class="r"><code>library(tidyverse)
library(readr)
library(kableExtra)
library(DT)
library(readr)
library(edgeR)
library(ComplexHeatmap)
library(genomation)
library(data.table)
library(limma)
library(GenomicRanges)</code></pre>
<pre class="r"><code>Plot_cpm_counts &lt;- function(df, histone){
  lcpm_df_rm &lt;- cpm(df, log = TRUE)
  df_rm &lt;- rowMeans(lcpm_df_rm)
  df_filt_raw &lt;- df[df_rm&gt;0,]


  df_anno_mat &lt;- data.frame(timeset=colnames(df)) %&gt;% 
    mutate(sample=timeset) %&gt;% 
    separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;,&quot;histone&quot;)) %&gt;% 
    mutate(trt=factor(trt, levels = c(&quot;VEH&quot;,&quot;5FU&quot;,&quot;DOX&quot;)),
           time=factor(time, levels =c(&quot;24t&quot;,&quot;24r&quot;,&quot;144r&quot;))) 

  df_filt_lcpm &lt;- cpm(df_filt_raw, log = TRUE)
  ###PCa on unfiltered data
  
  pca_df_unfilt &lt;- prcomp(t(lcpm_df_rm), scale. = TRUE)
  percent_var &lt;- round((pca_df_unfilt$sdev)^2 / sum(pca_df_unfilt$sdev^2) * 100, 2)
  anno_pca_df_unfilt &lt;- as.data.frame(pca_df_unfilt$x) %&gt;% 
    cbind(df_anno_mat)
  
  
  pca_df_filt &lt;- prcomp(t(df_filt_lcpm), scale. = TRUE)
  percent_var_filt &lt;- round((pca_df_filt$sdev)^2 / sum(pca_df_filt$sdev^2) * 100, 2)
  anno_pca_df_filt &lt;- as.data.frame(pca_df_filt$x) %&gt;% 
    cbind(df_anno_mat)
  
  ##histogram before filtering
  plot1 &lt;- lcpm_df_rm %&gt;% 
    as.data.frame() %&gt;% 
    rownames_to_column(&quot;region&quot;) %&gt;%
    pivot_longer(., -region, names_to=&quot;sample&quot;,values_to = &quot;log2_cpm&quot;) %&gt;%
    ggplot(.,aes(x=log2_cpm))+
    geom_histogram()+
    ggtitle(&quot;Log2 cpm before filtering&quot;)+
    theme_bw()
  
  #### PCA before filtering
  plot2 &lt;- anno_pca_df_unfilt %&gt;% 
    ggplot(aes(x = PC1, y = PC2, color = trt, shape = time)) +
    geom_point(size = 5) +
    ggrepel::geom_text_repel(aes(label = indv)) +
    theme_bw() +
    ggtitle(paste0(histone,&quot; unfiltered log2cpm&quot;)) +
    labs(
      x = sprintf(&quot;PC1: %.2f%%&quot;, percent_var[1]),
      y = sprintf(&quot;PC2: %.2f%%&quot;, percent_var[2])
    )
 
  ## hist after filtering
  plot3 &lt;- df_filt_lcpm %&gt;%
    as.data.frame() %&gt;% 
    rownames_to_column(&quot;region&quot;) %&gt;%
    pivot_longer(., -region, names_to=&quot;sample&quot;,values_to = &quot;log2_cpm&quot;) %&gt;%
    ggplot(.,aes(x=log2_cpm))+
    geom_histogram()+
    ggtitle(paste0(&quot;Log2 cpm after filtering&quot;, histone))+
    theme_bw()
  
  
  plot4 &lt;- anno_pca_df_filt %&gt;% 
    ggplot(aes(x = PC1, y = PC2, color = trt, shape = time)) +
    geom_point(size = 5) +
    ggrepel::geom_text_repel(aes(label = indv)) +
    theme_bw() +
    ggtitle(paste0(histone,&quot; filtered log2cpm&quot;)) +
    labs(
      x = sprintf(&quot;PC1: %.2f%%&quot;, percent_var[1]),
      y = sprintf(&quot;PC2: %.2f%%&quot;, percent_var[2])
    )

 print(plot1)
  print(plot2)
  print(plot3)
  print(plot4)

  return(invisible(list(
    hist_before = plot1,
    hist_after = plot3,
    pca_unfiltered = plot2,
    pca_filtered = plot4
  )))
}</code></pre>
</div>
<div id="q-value-threshholding" class="section level2">
<h2>Q value threshholding</h2>
<div id="h3k27ac-info" class="section level4">
<h4>H3K27ac info</h4>
<pre class="r"><code>sampleinfo &lt;- read_delim(&quot;data/sampleinfo.txt&quot;, delim = &quot;\t&quot;)

peak_count_H3K27ac_05 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/macs2_peak_q05/peak_count_q05.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.narrowPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K27ac&quot;) %&gt;% 
  mutate(qval=&quot;0.05&quot;)
peak_count_H3K27ac_01 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/macs2_peak_q01/peak_count_q01.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.narrowPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K27ac&quot;) %&gt;% 
  mutate(qval=&quot;0.01&quot;)
peak_count_H3K27ac_005 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/macs2_peak_q005/peak_count_q005.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.narrowPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K27ac&quot;) %&gt;% 
  mutate(qval=&quot;0.005&quot;)
### Count matrices for each
count_H3K27ac_05 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/noM_count_H3K27ac_macs2_peak_q05.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)

count_H3K27ac_01 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/noM_count_H3K27ac_macs2_peak_q01.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)

count_H3K27ac_005 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/noM_count_H3K27ac_macs2_peak_q005.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)
##contains the summary of reads that mapped to the peak file and those that did not
summary_H3K27ac_05 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/noM_count_H3K27ac_macs2_peak_q05.txt.summary&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)
summary_H3K27ac_01 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/noM_count_H3K27ac_macs2_peak_q01.txt.summary&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)
summary_H3K27ac_005 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/noM_count_H3K27ac_macs2_peak_q005.txt.summary&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)

rename_list &lt;- sampleinfo %&gt;% 
  mutate(stem= &quot;_noM_final.bam&quot;) %&gt;%
  mutate(front=&quot;H3K27ac/&quot;) %&gt;% 
  mutate(oldname=paste0(front,library_ID,stem)) %&gt;% 
  mutate(newname=paste0(ind,&quot;_&quot;,trt,&quot;_&quot;,other_time,&quot;_&quot;,histone_mark)) %&gt;% 
  dplyr::select(oldname,newname)
  
rename_vec &lt;- setNames(rename_list$newname, rename_list$oldname)

names(count_H3K27ac_05)[names(count_H3K27ac_05) %in% names(rename_vec)] &lt;- rename_vec[names(count_H3K27ac_05)[names(count_H3K27ac_05) %in% names(rename_vec)]]

names(count_H3K27ac_01)[names(count_H3K27ac_01) %in% names(rename_vec)] &lt;- rename_vec[names(count_H3K27ac_01)[names(count_H3K27ac_01) %in% names(rename_vec)]]

names(count_H3K27ac_005)[names(count_H3K27ac_005) %in% names(rename_vec)] &lt;- rename_vec[names(count_H3K27ac_005)[names(count_H3K27ac_005) %in% names(rename_vec)]]</code></pre>
<pre class="r"><code>peak_count_H3K27ac_05 %&gt;% 
  rbind(peak_count_H3K27ac_01) %&gt;% 
  rbind(peak_count_H3K27ac_005) %&gt;% 
  mutate(qval=factor(qval, levels =c(&quot;0.05&quot;,&quot;0.01&quot;,&quot;0.005&quot;))) %&gt;% 
   group_by(qval) %&gt;% 
  mutate(median_X1 = median(X1)) %&gt;%  # Calculate the median for each qval group
  ungroup() %&gt;% 
  ggplot(., aes(x=X2,y=X1, group()))+
  geom_point(aes(color=qval))+
  geom_path(aes(group = qval, color = qval), size = 1, linetype = &quot;solid&quot;) +
  geom_hline(aes(yintercept = median_X1, color = qval), linetype = &quot;dashed&quot;, size = 1) +
  ylab(&quot;Count of total peaks&quot;)+
  xlab(&quot;samples&quot;)+
  theme_bw()+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  ggtitle(&quot;Number of called peaks by file H3K27ac&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/H3K27ac%20peak%20counts-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-H3K27ac-peak-counts-1">
Past versions of "H3K27ac peak counts-1.png"
</button>
</p>
<div id="fig-no-spaces-H3K27ac-peak-counts-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/H3K27ac peak counts-1.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="peak-width-by-q-value" class="section level3">
<h3>Peak width by q value</h3>
<pre class="r"><code>H3K27ac_masterpeak_05 &lt;- readBed(file=&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/macs2_peak_q05/filt_macs2_peak_q05_H3k27ac_all_merged.bed&quot;, remove.unusual = FALSE)

H3K27ac_masterpeak_01 &lt;- readBed(file=&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/macs2_peak_q01/filt_macs2_peak_q01_H3k27ac_all_merged.bed&quot;, remove.unusual = FALSE)

H3K27ac_masterpeak_005 &lt;- readBed(file=&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/macs2_peak_q005/filt_macs2_peak_q005_H3k27ac_all_merged.bed&quot;, remove.unusual = FALSE)

master_peak_width_H3K27ac &lt;- data_frame(peak_width=width(H3K27ac_masterpeak_01),histone=&quot;H3K27ac&quot;,qval=&quot;0.01&quot;) %&gt;%
  rbind(data_frame(peak_width=width(H3K27ac_masterpeak_05),histone=&quot;H3K27ac&quot;,qval=&quot;0.05&quot;)) %&gt;%
  rbind(data_frame(peak_width=width(H3K27ac_masterpeak_005),histone=&quot;H3K27ac&quot;,qval=&quot;0.005&quot;))

master_peak_width_H3K27ac %&gt;% 
   mutate(qval=factor(qval, levels =c(&quot;0.05&quot;,&quot;0.01&quot;,&quot;0.005&quot;))) %&gt;% 
   ggplot(.,aes(x=qval, y = peak_width, fill = qval))+
  geom_violin()+
    # scale_y_continuous(trans = &quot;log&quot;, breaks = c(400, 3000, 22000)) +
    theme_bw(base_size = 18) +
    ylab(&quot;Width of Peaks&quot;) +
    xlab(&quot;&quot;)+
  ggtitle(&quot;Master Peak list widths for H3K27ac&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/H3K27ac%20peak%20width-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-H3K27ac-peak-width-1">
Past versions of "H3K27ac peak width-1.png"
</button>
</p>
<div id="fig-no-spaces-H3K27ac-peak-width-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/H3K27ac peak width-1.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>H3K27ac_counts_raw_05 &lt;- count_H3K27ac_05 %&gt;% 
  dplyr::select(Geneid,`3_VEH_24r_H3K27ac`:`3_DOX_24r_H3K27ac`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()
H3K27ac_counts_raw_01 &lt;- count_H3K27ac_01 %&gt;% 
  dplyr::select(Geneid,`3_VEH_24r_H3K27ac`:`3_DOX_24r_H3K27ac`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()
H3K27ac_counts_raw_005 &lt;- count_H3K27ac_005 %&gt;% 
  dplyr::select(Geneid,`3_VEH_24r_H3K27ac`:`3_DOX_24r_H3K27ac`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()</code></pre>
<pre class="r"><code>Plot_cpm_counts(H3K27ac_counts_raw_05,&quot;H3K27ac_05&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-1">
Past versions of "log2cpm transform-1.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform-1.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-2">
Past versions of "log2cpm transform-2.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform-2.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-3">
Past versions of "log2cpm transform-3.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform-3.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-4">
Past versions of "log2cpm transform-4.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform-4.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Plot_cpm_counts(H3K27ac_counts_raw_01,&quot;H3K27ac_01&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-5">
Past versions of "log2cpm transform-5.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform-5.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-6">
Past versions of "log2cpm transform-6.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform-6.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-7">
Past versions of "log2cpm transform-7.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform-7.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-8">
Past versions of "log2cpm transform-8.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform-8.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Plot_cpm_counts(H3K27ac_counts_raw_005,&quot;H3K27ac_005&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-9">
Past versions of "log2cpm transform-9.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform-9.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform-10.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-10">
Past versions of "log2cpm transform-10.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-10" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform-10.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform-11.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-11">
Past versions of "log2cpm transform-11.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-11" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform-11.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform-12.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-12">
Past versions of "log2cpm transform-12.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-12" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform-12.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="h3k27me3-info" class="section level4">
<h4>H3K27me3 info</h4>
<pre class="r"><code>peak_count_H3K27me3_05 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/macs2_peak_q05/peak_count_q05.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.broadPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K27me3&quot;) %&gt;% 
  mutate(qval=&quot;0.05&quot;)
peak_count_H3K27me3_01 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/macs2_peak_q01/peak_count_q01.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.broadPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K27me3&quot;) %&gt;% 
  mutate(qval=&quot;0.01&quot;)
peak_count_H3K27me3_005 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/macs2_peak_q005/peak_count_q005.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.broadPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K27me3&quot;) %&gt;% 
  mutate(qval=&quot;0.005&quot;)
### Count matrices for each
count_H3K27me3_05 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/noM_count_H3K27me3_macs2_peak_q05.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)

count_H3K27me3_01 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/noM_count_H3K27me3_macs2_peak_q01.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)

count_H3K27me3_005 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/noM_count_H3K27me3_macs2_peak_q005.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)
##contains the summary of reads that mapped to the peak file and those that did not
summary_H3K27me3_05 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/noM_count_H3K27me3_macs2_peak_q05.txt.summary&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)
summary_H3K27me3_01 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/noM_count_H3K27me3_macs2_peak_q01.txt.summary&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)
summary_H3K27me3_005 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/noM_count_H3K27me3_macs2_peak_q005.txt.summary&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)

rename_list_k27me3 &lt;- sampleinfo %&gt;% 
  mutate(stem= &quot;_noM_final.bam&quot;) %&gt;%
  mutate(front=&quot;H3K27me3/&quot;) %&gt;% 
  mutate(oldname=paste0(front,library_ID,stem)) %&gt;% 
  mutate(newname=paste0(ind,&quot;_&quot;,trt,&quot;_&quot;,other_time,&quot;_&quot;,histone_mark)) %&gt;% 
  dplyr::select(oldname,newname)
  
rename_vec_k27me3 &lt;- setNames(rename_list_k27me3$newname, rename_list_k27me3$oldname)

names(count_H3K27me3_05)[names(count_H3K27me3_05) %in% names(rename_vec_k27me3)] &lt;- 
  rename_vec_k27me3[names(count_H3K27me3_05)[names(count_H3K27me3_05) %in% names(rename_vec_k27me3)]]

names(count_H3K27me3_01)[names(count_H3K27me3_01) %in% names(rename_vec_k27me3)] &lt;- 
  rename_vec_k27me3[names(count_H3K27me3_01)[names(count_H3K27me3_01) %in% names(rename_vec_k27me3)]]

names(count_H3K27me3_005)[names(count_H3K27me3_005) %in% names(rename_vec_k27me3)] &lt;- 
  rename_vec_k27me3[names(count_H3K27me3_005)[names(count_H3K27me3_005) %in% names(rename_vec_k27me3)]]</code></pre>
<pre class="r"><code>peak_count_H3K27me3_05 %&gt;% 
  rbind(peak_count_H3K27me3_01) %&gt;% 
  rbind(peak_count_H3K27me3_005) %&gt;% 
  mutate(X2=gsub(&quot;macs2_peak_q05/&quot;,&quot;&quot;, X2)) %&gt;% 
  mutate(X2=gsub(&quot;macs2_peak_q01/&quot;,&quot;&quot;, X2)) %&gt;% 
    mutate(X2=gsub(&quot;macs2_peak_q005/&quot;,&quot;&quot;, X2)) %&gt;% 
  mutate(qval=factor(qval, levels =c(&quot;0.05&quot;,&quot;0.01&quot;,&quot;0.005&quot;))) %&gt;% 
   group_by(qval) %&gt;% 
  mutate(median_X1 = median(X1)) %&gt;%  # Calculate the median for each qval group
  ungroup() %&gt;% 
  ggplot(., aes(x=X2,y=X1, group()))+
  geom_point(aes(color=qval))+
  geom_path(aes(group = qval, color = qval), size = 1, linetype = &quot;solid&quot;) +
  geom_hline(aes(yintercept = median_X1, color = qval), linetype = &quot;dashed&quot;, size = 1) +
  ylab(&quot;Count of total peaks&quot;)+
  xlab(&quot;samples&quot;)+
  theme_bw()+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  ggtitle(&quot;Number of called peaks by file H3K27me3&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/H3K27me3%20peak%20counts-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-H3K27me3-peak-counts-1">
Past versions of "H3K27me3 peak counts-1.png"
</button>
</p>
<div id="fig-no-spaces-H3K27me3-peak-counts-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/H3K27me3 peak counts-1.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/H3K27me3 peak counts-1.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>H3K27me3_masterpeak_05 &lt;- readBed(file=&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/macs2_peak_q05/filt_macs2_peak_q05_H3k27me3_all_merged.bed&quot;, remove.unusual = FALSE)

H3K27me3_masterpeak_01 &lt;- readBed(file=&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/macs2_peak_q01/filt_macs2_peak_q01_H3k27me3_all_merged.bed&quot;, remove.unusual = FALSE)

H3K27ac_masterpeak_005 &lt;- readBed(file=&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/macs2_peak_q005/filt_macs2_peak_q005_H3k27me3_all_merged.bed&quot;, remove.unusual = FALSE)

master_peak_width_H3K27me3 &lt;- data_frame(peak_width=width(H3K27me3_masterpeak_01),histone=&quot;H3K27me3&quot;,qval=&quot;0.01&quot;) %&gt;%
  rbind(data_frame(peak_width=width(H3K27me3_masterpeak_05),histone=&quot;H3K27me3&quot;,qval=&quot;0.05&quot;)) %&gt;%
  rbind(data_frame(peak_width=width(H3K27ac_masterpeak_005),histone=&quot;H3K27me3&quot;,qval=&quot;0.005&quot;))

master_peak_width_H3K27me3 %&gt;% 
   mutate(qval=factor(qval, levels =c(&quot;0.05&quot;,&quot;0.01&quot;,&quot;0.005&quot;))) %&gt;% 
   ggplot(.,aes(x=qval, y = peak_width, fill = qval))+
  geom_violin()+
    # scale_y_continuous(trans = &quot;log&quot;, breaks = c(400, 3000, 22000)) +
    theme_bw(base_size = 18) +
    ylab(&quot;Width of Peaks&quot;) +
    xlab(&quot;&quot;)+
  ggtitle(&quot;Master Peak list widths for H3K27me3&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/more%20H3k27me3%20things-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-more-H3k27me3-things-1">
Past versions of "more H3k27me3 things-1.png"
</button>
</p>
<div id="fig-no-spaces-more-H3k27me3-things-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/more H3k27me3 things-1.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/more H3k27me3 things-1.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>H3K27me3_counts_raw_05 &lt;- count_H3K27me3_05 %&gt;% 
  dplyr::select(Geneid,`1_VEH_24t_H3K27me3`:`5_VEH_24r_H3K27me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()

H3K27me3_counts_raw_01 &lt;- count_H3K27me3_01 %&gt;% 
  dplyr::select(Geneid,`1_VEH_24t_H3K27me3`:`5_VEH_24r_H3K27me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()

H3K27me3_counts_raw_005 &lt;- count_H3K27me3_005 %&gt;% 
  dplyr::select(Geneid,`1_VEH_24t_H3K27me3`:`5_VEH_24r_H3K27me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()</code></pre>
<pre class="r"><code>Plot_cpm_counts(H3K27me3_counts_raw_05,&quot;H3K27me3_05&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k27me3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k27me3-1">
Past versions of "log2cpm transform h3k27me3-1.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k27me3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-1.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-1.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k27me3-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k27me3-2">
Past versions of "log2cpm transform h3k27me3-2.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k27me3-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-2.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-2.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k27me3-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k27me3-3">
Past versions of "log2cpm transform h3k27me3-3.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k27me3-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-3.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-3.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k27me3-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k27me3-4">
Past versions of "log2cpm transform h3k27me3-4.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k27me3-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-4.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-4.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Plot_cpm_counts(H3K27me3_counts_raw_01,&quot;H3K27me3_01&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k27me3-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k27me3-5">
Past versions of "log2cpm transform h3k27me3-5.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k27me3-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-5.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-5.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k27me3-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k27me3-6">
Past versions of "log2cpm transform h3k27me3-6.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k27me3-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-6.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-6.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k27me3-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k27me3-7">
Past versions of "log2cpm transform h3k27me3-7.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k27me3-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-7.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-7.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k27me3-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k27me3-8">
Past versions of "log2cpm transform h3k27me3-8.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k27me3-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-8.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-8.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Plot_cpm_counts(H3K27me3_counts_raw_005,&quot;H3K27me3_005&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k27me3-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k27me3-9">
Past versions of "log2cpm transform h3k27me3-9.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k27me3-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-9.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-9.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k27me3-10.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k27me3-10">
Past versions of "log2cpm transform h3k27me3-10.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k27me3-10" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-10.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-10.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k27me3-11.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k27me3-11">
Past versions of "log2cpm transform h3k27me3-11.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k27me3-11" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-11.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-11.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k27me3-12.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k27me3-12">
Past versions of "log2cpm transform h3k27me3-12.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k27me3-12" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-12.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/75809ebc1b22d2f84d6a37fa845bd35ed72763d4/docs/figure/Thresholding.Rmd/log2cpm transform h3k27me3-12.png" target="_blank">75809eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="h3k36me3-info" class="section level4">
<h4>H3K36me3 info</h4>
<pre class="r"><code>peak_count_H3K36me3_05 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/macs2_peak_q05/peak_count_q05.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.broadPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K36me3&quot;) %&gt;% 
  mutate(qval=&quot;0.05&quot;)
peak_count_H3K36me3_01 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/macs2_peak_q01/peak_count_q01.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.broadPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K36me3&quot;) %&gt;% 
  mutate(qval=&quot;0.01&quot;)
peak_count_H3K36me3_005 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/macs2_peak_q005/peak_count_q005.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.broadPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K36me3&quot;) %&gt;% 
  mutate(qval=&quot;0.005&quot;)
### Count matrices for each
count_H3K36me3_05 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/noM_count_H3K36me3_macs2_peak_q05.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)

count_H3K36me3_01 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/noM_count_H3K36me3_macs2_peak_q01.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)

count_H3K36me3_005 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/noM_count_H3K36me3_macs2_peak_q005.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)
##contains the summary of reads that mapped to the peak file and those that did not
summary_H3K36me3_05 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/noM_count_H3K36me3_macs2_peak_q05.txt.summary&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)
summary_H3K36me3_01 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/noM_count_H3K36me3_macs2_peak_q01.txt.summary&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)
summary_H3K36me3_005 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/noM_count_H3K36me3_macs2_peak_q005.txt.summary&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)

rename_list_k36me3 &lt;- sampleinfo %&gt;% 
  mutate(stem= &quot;_noM_final.bam&quot;) %&gt;%
  mutate(front=&quot;H3K36me3/&quot;) %&gt;% 
  mutate(oldname=paste0(front,library_ID,stem)) %&gt;% 
  mutate(newname=paste0(ind,&quot;_&quot;,trt,&quot;_&quot;,other_time,&quot;_&quot;,histone_mark)) %&gt;% 
  dplyr::select(oldname,newname)
  
rename_vec_k36me3 &lt;- setNames(rename_list_k36me3$newname, rename_list_k36me3$oldname)

names(count_H3K36me3_05)[names(count_H3K36me3_05) %in% names(rename_vec_k36me3)] &lt;- 
  rename_vec_k36me3[names(count_H3K36me3_05)[names(count_H3K36me3_05) %in% names(rename_vec_k36me3)]]

names(count_H3K36me3_01)[names(count_H3K36me3_01) %in% names(rename_vec_k36me3)] &lt;- 
  rename_vec_k36me3[names(count_H3K36me3_01)[names(count_H3K36me3_01) %in% names(rename_vec_k36me3)]]

names(count_H3K36me3_005)[names(count_H3K36me3_005) %in% names(rename_vec_k36me3)] &lt;- 
  rename_vec_k36me3[names(count_H3K36me3_005)[names(count_H3K36me3_005) %in% names(rename_vec_k36me3)]]</code></pre>
<pre class="r"><code>peak_count_H3K36me3_05 %&gt;% 
  rbind(peak_count_H3K36me3_01) %&gt;% 
  rbind(peak_count_H3K36me3_005) %&gt;% 
  mutate(X2=gsub(&quot;macs2_peak_q05/&quot;,&quot;&quot;, X2)) %&gt;% 
  mutate(X2=gsub(&quot;macs2_peak_q01/&quot;,&quot;&quot;, X2)) %&gt;% 
    mutate(X2=gsub(&quot;macs2_peak_q005/&quot;,&quot;&quot;, X2)) %&gt;% 
  mutate(qval=factor(qval, levels =c(&quot;0.05&quot;,&quot;0.01&quot;,&quot;0.005&quot;))) %&gt;% 
   group_by(qval) %&gt;% 
  mutate(median_X1 = median(X1)) %&gt;%  # Calculate the median for each qval group
  ungroup() %&gt;% 
  ggplot(., aes(x=X2,y=X1, group()))+
  geom_point(aes(color=qval))+
  geom_path(aes(group = qval, color = qval), size = 1, linetype = &quot;solid&quot;) +
  geom_hline(aes(yintercept = median_X1, color = qval), linetype = &quot;dashed&quot;, size = 1) +
  ylab(&quot;Count of total peaks&quot;)+
  xlab(&quot;samples&quot;)+
  theme_bw()+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  ggtitle(&quot;Number of called peaks by file H3K36me&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/H3K36me3%20peak%20counts-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-H3K36me3-peak-counts-1">
Past versions of "H3K36me3 peak counts-1.png"
</button>
</p>
<div id="fig-no-spaces-H3K36me3-peak-counts-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/H3K36me3 peak counts-1.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>H3K36me3_masterpeak_05 &lt;- readBed(file=&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/macs2_peak_q05/filt_macs2_peak_q05_H3k36me3_all_merged.bed&quot;, remove.unusual = FALSE)

H3K36me3_masterpeak_01 &lt;- readBed(file=&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/macs2_peak_q01/filt_macs2_peak_q01_H3k36me3_all_merged.bed&quot;, remove.unusual = FALSE)

H3K36ac_masterpeak_005 &lt;- readBed(file=&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/macs2_peak_q005/filt_macs2_peak_q005_H3k36me3_all_merged.bed&quot;, remove.unusual = FALSE)

master_peak_width_H3K36me3 &lt;- data_frame(peak_width=width(H3K36me3_masterpeak_01),histone=&quot;H3K36me3&quot;,qval=&quot;0.01&quot;) %&gt;%
  rbind(data_frame(peak_width=width(H3K36me3_masterpeak_05),histone=&quot;H3K36me3&quot;,qval=&quot;0.05&quot;)) %&gt;%
  rbind(data_frame(peak_width=width(H3K36ac_masterpeak_005),histone=&quot;H3K36me3&quot;,qval=&quot;0.005&quot;))

master_peak_width_H3K36me3 %&gt;% 
   mutate(qval=factor(qval, levels =c(&quot;0.05&quot;,&quot;0.01&quot;,&quot;0.005&quot;))) %&gt;% 
   ggplot(.,aes(x=qval, y = peak_width, fill = qval))+
  geom_violin()+
    # scale_y_continuous(trans = &quot;log&quot;, breaks = c(400, 3000, 22000)) +
    theme_bw(base_size = 18) +
    ylab(&quot;Width of Peaks&quot;) +
    xlab(&quot;&quot;)+
  ggtitle(&quot;Master Peak list widths for H3K36me3&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/more%20H3k36me3%20things-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-more-H3k36me3-things-1">
Past versions of "more H3k36me3 things-1.png"
</button>
</p>
<div id="fig-no-spaces-more-H3k36me3-things-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/more H3k36me3 things-1.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>H3K36me3_counts_raw_05 &lt;- count_H3K36me3_05 %&gt;% 
  dplyr::select(Geneid,`1_5FU_24t_H3K36me3`:`5_VEH_144r_H3K36me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()

H3K36me3_counts_raw_01 &lt;- count_H3K36me3_01 %&gt;% 
  dplyr::select(Geneid,`1_5FU_24t_H3K36me3`:`5_VEH_144r_H3K36me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()

H3K36me3_counts_raw_005 &lt;- count_H3K36me3_005 %&gt;% 
  dplyr::select(Geneid,`1_5FU_24t_H3K36me3`:`5_VEH_144r_H3K36me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()</code></pre>
<pre class="r"><code>Plot_cpm_counts(H3K36me3_counts_raw_05,&quot;H3K36me3_05&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k36me3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k36me3-1">
Past versions of "log2cpm transform h3k36me3-1.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k36me3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k36me3-1.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k36me3-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k36me3-2">
Past versions of "log2cpm transform h3k36me3-2.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k36me3-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k36me3-2.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k36me3-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k36me3-3">
Past versions of "log2cpm transform h3k36me3-3.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k36me3-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k36me3-3.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k36me3-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k36me3-4">
Past versions of "log2cpm transform h3k36me3-4.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k36me3-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k36me3-4.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Plot_cpm_counts(H3K36me3_counts_raw_01,&quot;H3K36me3_01&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k36me3-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k36me3-5">
Past versions of "log2cpm transform h3k36me3-5.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k36me3-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k36me3-5.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k36me3-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k36me3-6">
Past versions of "log2cpm transform h3k36me3-6.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k36me3-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k36me3-6.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k36me3-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k36me3-7">
Past versions of "log2cpm transform h3k36me3-7.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k36me3-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k36me3-7.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k36me3-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k36me3-8">
Past versions of "log2cpm transform h3k36me3-8.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k36me3-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k36me3-8.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Plot_cpm_counts(H3K36me3_counts_raw_005,&quot;H3K36me3_005&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k36me3-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k36me3-9">
Past versions of "log2cpm transform h3k36me3-9.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k36me3-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k36me3-9.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k36me3-10.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k36me3-10">
Past versions of "log2cpm transform h3k36me3-10.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k36me3-10" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k36me3-10.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k36me3-11.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k36me3-11">
Past versions of "log2cpm transform h3k36me3-11.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k36me3-11" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k36me3-11.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k36me3-12.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k36me3-12">
Past versions of "log2cpm transform h3k36me3-12.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k36me3-12" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k36me3-12.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="h3k9me3-info" class="section level4">
<h4>H3K9me3 info</h4>
<pre class="r"><code>peak_count_H3K9me3_05 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/macs2_peak_q05/peak_count_q05.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.narrowPeak&quot;,&quot;&quot;,X2)) %&gt;% 
   mutate(histone=&quot;H3K9me3&quot;) %&gt;% 
  mutate(qval=&quot;0.05&quot;)

peak_count_H3K9me3_01 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/macs2_peak_q01/peak_count_q01.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.narrowPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K9me3&quot;) %&gt;% 
  mutate(qval=&quot;0.01&quot;)
peak_count_H3K9me3_005 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/macs2_peak_q005/peak_count_q005.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.narrowPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K9me3&quot;) %&gt;% 
  mutate(qval=&quot;0.005&quot;)
### Count matrices for each
count_H3K9me3_05 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/noM_count_H3K9me3_macs2_peak_q05.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)

count_H3K9me3_01 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/noM_count_H3K9me3_macs2_peak_q01.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)

count_H3K9me3_005 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/noM_count_H3K9me3_macs2_peak_q005.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)
##contains the summary of reads that mapped to the peak file and those that did not
summary_H3K9me3_05 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/noM_count_H3K9me3_macs2_peak_q05.txt.summary&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)
summary_H3K9me3_01 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/noM_count_H3K9me3_macs2_peak_q01.txt.summary&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)
summary_H3K9me3_005 &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/noM_count_H3K9me3_macs2_peak_q005.txt.summary&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)

rename_list_k9me3 &lt;- sampleinfo %&gt;% 
  mutate(stem= &quot;_noM_final.bam&quot;) %&gt;%
  mutate(front=&quot;H3K9me3/&quot;) %&gt;% 
  mutate(oldname=paste0(front,library_ID,stem)) %&gt;% 
  mutate(newname=paste0(ind,&quot;_&quot;,trt,&quot;_&quot;,other_time,&quot;_&quot;,histone_mark)) %&gt;% 
  dplyr::select(oldname,newname)
  
rename_vec_k9me3 &lt;- setNames(rename_list_k9me3$newname, rename_list_k9me3$oldname)

names(count_H3K9me3_05)[names(count_H3K9me3_05) %in% names(rename_vec_k9me3)] &lt;- 
  rename_vec_k9me3[names(count_H3K9me3_05)[names(count_H3K9me3_05) %in% names(rename_vec_k9me3)]]

names(count_H3K9me3_01)[names(count_H3K9me3_01) %in% names(rename_vec_k9me3)] &lt;- 
  rename_vec_k9me3[names(count_H3K9me3_01)[names(count_H3K9me3_01) %in% names(rename_vec_k9me3)]]

names(count_H3K9me3_005)[names(count_H3K9me3_005) %in% names(rename_vec_k9me3)] &lt;- 
  rename_vec_k9me3[names(count_H3K9me3_005)[names(count_H3K9me3_005) %in% names(rename_vec_k9me3)]]</code></pre>
<pre class="r"><code>peak_count_H3K9me3_05 %&gt;% 
  rbind(peak_count_H3K9me3_01) %&gt;% 
  rbind(peak_count_H3K9me3_005) %&gt;% 
  mutate(X2=gsub(&quot;macs2_peak_q05/&quot;,&quot;&quot;, X2)) %&gt;% 
  mutate(X2=gsub(&quot;macs2_peak_q01/&quot;,&quot;&quot;, X2)) %&gt;% 
    mutate(X2=gsub(&quot;macs2_peak_q005/&quot;,&quot;&quot;, X2)) %&gt;% 
  mutate(qval=factor(qval, levels =c(&quot;0.05&quot;,&quot;0.01&quot;,&quot;0.005&quot;))) %&gt;% 
   group_by(qval) %&gt;% 
  mutate(median_X1 = median(X1)) %&gt;%  # Calculate the median for each qval group
  ungroup() %&gt;% 
  ggplot(., aes(x=X2,y=X1, group()))+
  geom_point(aes(color=qval))+
  geom_path(aes(group = qval, color = qval), size = 1, linetype = &quot;solid&quot;) +
  geom_hline(aes(yintercept = median_X1, color = qval), linetype = &quot;dashed&quot;, size = 1) +
  ylab(&quot;Count of total peaks&quot;)+
  xlab(&quot;samples&quot;)+
  theme_bw()+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  ggtitle(&quot;Number of called peaks by file H3K9me3&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/H3K9me3%20peak%20counts-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-H3K9me3-peak-counts-1">
Past versions of "H3K9me3 peak counts-1.png"
</button>
</p>
<div id="fig-no-spaces-H3K9me3-peak-counts-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/H3K9me3 peak counts-1.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>H3K9me3_masterpeak_05 &lt;- readBed(file=&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/macs2_peak_q05/filt_macs2_peak_q05_H3k9me3_all_merged.bed&quot;, remove.unusual = FALSE)

H3K9me3_masterpeak_01 &lt;- readBed(file=&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/macs2_peak_q01/filt_macs2_peak_q01_H3k9me3_all_merged.bed&quot;, remove.unusual = FALSE)

H3K9ac_masterpeak_005 &lt;- readBed(file=&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/macs2_peak_q005/filt_macs2_peak_q005_H3k9me3_all_merged.bed&quot;, remove.unusual = FALSE)

master_peak_width_H3K9me3 &lt;- data_frame(peak_width=width(H3K9me3_masterpeak_01),histone=&quot;H3K9me3&quot;,qval=&quot;0.01&quot;) %&gt;%
  rbind(data_frame(peak_width=width(H3K9me3_masterpeak_05),histone=&quot;H3K9me3&quot;,qval=&quot;0.05&quot;)) %&gt;%
  rbind(data_frame(peak_width=width(H3K9ac_masterpeak_005),histone=&quot;H3K9me3&quot;,qval=&quot;0.005&quot;))

master_peak_width_H3K9me3 %&gt;% 
   mutate(qval=factor(qval, levels =c(&quot;0.05&quot;,&quot;0.01&quot;,&quot;0.005&quot;))) %&gt;% 
   ggplot(.,aes(x=qval, y = peak_width, fill = qval))+
  geom_violin()+
    # scale_y_continuous(trans = &quot;log&quot;, breaks = c(400, 3000, 22000)) +
    theme_bw(base_size = 18) +
    ylab(&quot;Width of Peaks&quot;) +
    xlab(&quot;&quot;)+
  ggtitle(&quot;Master Peak list widths for H3K9me3&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/more%20H3k9me3%20things-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-more-H3k9me3-things-1">
Past versions of "more H3k9me3 things-1.png"
</button>
</p>
<div id="fig-no-spaces-more-H3k9me3-things-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/more H3k9me3 things-1.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>H3K9me3_counts_raw_05 &lt;- count_H3K9me3_05 %&gt;% 
  dplyr::select(Geneid,`1_DOX_24t_H3K9me3`:`5_5FU_144r_H3K9me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()
H3K9me3_counts_raw_01 &lt;- count_H3K9me3_01 %&gt;% 
  dplyr::select(Geneid,`1_DOX_24t_H3K9me3`:`5_5FU_144r_H3K9me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()

H3K9me3_counts_raw_005 &lt;- count_H3K9me3_005 %&gt;% 
  dplyr::select(Geneid,`1_DOX_24t_H3K9me3`:`5_5FU_144r_H3K9me3`) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()</code></pre>
<pre class="r"><code>Plot_cpm_counts(H3K9me3_counts_raw_05,&quot;H3K9me3_05&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k9me3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k9me3-1">
Past versions of "log2cpm transform h3k9me3-1.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k9me3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k9me3-1.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k9me3-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k9me3-2">
Past versions of "log2cpm transform h3k9me3-2.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k9me3-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k9me3-2.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k9me3-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k9me3-3">
Past versions of "log2cpm transform h3k9me3-3.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k9me3-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k9me3-3.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k9me3-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k9me3-4">
Past versions of "log2cpm transform h3k9me3-4.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k9me3-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k9me3-4.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Plot_cpm_counts(H3K9me3_counts_raw_01,&quot;H3K9me3_01&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k9me3-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k9me3-5">
Past versions of "log2cpm transform h3k9me3-5.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k9me3-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k9me3-5.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k9me3-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k9me3-6">
Past versions of "log2cpm transform h3k9me3-6.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k9me3-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k9me3-6.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k9me3-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k9me3-7">
Past versions of "log2cpm transform h3k9me3-7.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k9me3-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k9me3-7.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k9me3-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k9me3-8">
Past versions of "log2cpm transform h3k9me3-8.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k9me3-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k9me3-8.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Plot_cpm_counts(H3K9me3_counts_raw_005,&quot;H3K9me3_005&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k9me3-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k9me3-9">
Past versions of "log2cpm transform h3k9me3-9.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k9me3-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k9me3-9.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k9me3-10.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k9me3-10">
Past versions of "log2cpm transform h3k9me3-10.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k9me3-10" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k9me3-10.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k9me3-11.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k9me3-11">
Past versions of "log2cpm transform h3k9me3-11.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k9me3-11" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k9me3-11.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Thresholding.Rmd/log2cpm%20transform%20h3k9me3-12.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log2cpm-transform-h3k9me3-12">
Past versions of "log2cpm transform h3k9me3-12.png"
</button>
</p>
<div id="fig-no-spaces-log2cpm-transform-h3k9me3-12" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/DXR_project/blob/8d76c4eee519b7ae9945f63314bbfeac125b10f4/docs/figure/Thresholding.Rmd/log2cpm transform h3k9me3-12.png" target="_blank">8d76c4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="thresholding-and-frip" class="section level3">
<h3>Thresholding and FRIP</h3>
<pre class="r"><code>q05_first_frip_indv_call &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/q05_first_frip_indv_call.tsv&quot;,delim = &quot;\t&quot;, escape_double = FALSE,trim_ws = TRUE)


q05_first_frip_indv_call %&gt;% 
  dplyr::filter(Histone==&quot;H3K27ac&quot;) %&gt;% 
  ggplot(., aes(x=Sample, y=FRiP)) +
  geom_col()+
  ggtitle(&quot;Individual file Frip by Histone H3K27ac, q=0.05&quot;)+
  # xlab(&quot;samples&quot;)+
  theme_bw()+
  theme(axis.text.x=element_text(vjust = .2,angle=90))</code></pre>
<p><img src="figure/Thresholding.Rmd/unnamed-chunk-6-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>q05_first_frip_indv_call %&gt;% 
  dplyr::filter(Histone==&quot;H3K27me3&quot;) %&gt;% 
  ggplot(., aes(x=Sample, y=FRiP)) +
  geom_col()+
  ggtitle(&quot;Individual file Frip by Histone H3K27me3, q=0.05&quot;)+
  # xlab(&quot;samples&quot;)+
  theme_bw()+
  theme(axis.text.x=element_text(vjust = .2,angle=90))</code></pre>
<p><img src="figure/Thresholding.Rmd/unnamed-chunk-6-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>q05_first_frip_indv_call %&gt;% 
  dplyr::filter(Histone==&quot;H3K36me3&quot;) %&gt;% 
  ggplot(., aes(x=Sample, y=FRiP)) +
  geom_col()+
  ggtitle(&quot;Individual file Frip by Histone H3K36me3, q=0.05&quot;)+
  # xlab(&quot;samples&quot;)+
  theme_bw()+
  theme(axis.text.x=element_text(vjust = .2,angle=90))</code></pre>
<p><img src="figure/Thresholding.Rmd/unnamed-chunk-6-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>q05_first_frip_indv_call %&gt;% 
  dplyr::filter(Histone==&quot;H3K9me3&quot;) %&gt;% 
  ggplot(., aes(x=Sample, y=FRiP)) +
  geom_col()+
  ggtitle(&quot;Individual file Frip by Histone H3K9me3, q=0.05&quot;)+
  # xlab(&quot;samples&quot;)+
  theme_bw()+
  theme(axis.text.x=element_text(vjust = .2,angle=90))</code></pre>
<p><img src="figure/Thresholding.Rmd/unnamed-chunk-6-4.png" width="672" style="display: block; margin: auto;" /></p>
<div id="h3k27ac-llocal-and-nolambda" class="section level4">
<h4>H3K27ac llocal and nolambda</h4>
<pre class="r"><code>frip_llocal20k_nolambda &lt;- read_delim(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/Frip_files/frip_llocal20k_nolambda_detailed_results.tsv&quot;, 
     delim = &quot;\t&quot;, escape_double = FALSE, 
     trim_ws = TRUE)

frip_llocal20k_nolambda %&gt;% 
  dplyr::rename(&quot;call_type&quot;=&quot;Total_Reads&quot;,
                &quot;Tot_Read&quot;= &quot;Reads_in_Peaks&quot;) %&gt;%
  separate_wider_delim(.,FRiP, names = c(&quot;Frag_in_peaks&quot;,&quot;Frip&quot;),delim=&quot;\t&quot;) %&gt;% 
  mutate(Frag_in_peaks=as.numeric(Frag_in_peaks),Frip=as.numeric(Frip)) %&gt;% 
  mutate(call_type=factor(call_type, levels = c(&quot;macs2_nolambda_q05&quot;, &quot;macs2_nolambda_q01&quot;,&quot;macs2_llocal20k_q05&quot;, &quot;macs2_llocal20k_q01&quot;))) %&gt;% 
  dplyr::filter(Histone == &quot;H3K27ac&quot;) %&gt;% 
   group_by(call_type) %&gt;% 
  mutate(median_Frip = median(Frip)) %&gt;%  # Calculate the median for each qval group
  ungroup() %&gt;% 
  ggplot(., aes(x=Sample,y=Frip))+
  geom_point(aes(color=call_type))+
  geom_path(aes(group = call_type, color = call_type), size = 1, linetype = &quot;solid&quot;) +
  geom_hline(aes(yintercept = median_Frip, color = call_type), linetype = &quot;dashed&quot;, size = 1)+
  ggtitle(&quot;Individual file Frip by Histone H3K27ac&quot;)+
  # xlab(&quot;samples&quot;)+
  theme_bw()+
  theme(axis.text.x=element_text(vjust = .2,angle=90))</code></pre>
<p><img src="figure/Thresholding.Rmd/unnamed-chunk-7-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>peak_count_nolambda_H3K27ac_05 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/macs2_nolambda_q05/peak_count_nolambda_q05.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.narrowPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K27ac&quot;,type=&quot;nolambda&quot;) %&gt;% 
  mutate(qval=&quot;0.05&quot;)
peak_count_nolambda_H3K27ac_01 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/macs2_nolambda_q01/peak_count_nolambda_q01.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.narrowPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K27ac&quot;,type=&quot;nolambda&quot;) %&gt;% 
  mutate(qval=&quot;0.01&quot;)

peak_count_llocal20k_H3K27ac_05 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/macs2_llocal20k_q05/peak_count_llocal120k_q05.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.narrowPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K27ac&quot;,type=&quot;llocal20k&quot;) %&gt;% 
  mutate(qval=&quot;0.05&quot;)
peak_count_llocal20k_H3K27ac_01 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27ac/macs2_llocal20k_q01/peak_count_llocal120k_q01.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.narrowPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K27ac&quot;,type=&quot;llocal20k&quot;) %&gt;% 
  mutate(qval=&quot;0.01&quot;)



####################older data for integration 

firstPeak_H3K27ac &lt;- peak_count_H3K27ac_05 %&gt;% 
  rbind(peak_count_H3K27ac_01) %&gt;% 
  rbind(peak_count_H3K27ac_005) %&gt;% 
  mutate(qval=factor(qval, levels =c(&quot;0.05&quot;,&quot;0.01&quot;,&quot;0.005&quot;))) %&gt;% 
   group_by(qval) %&gt;% 
  mutate(median_X1 = median(X1)) %&gt;%  # Calculate the median for each qval group
  ungroup() %&gt;% 
  mutate(type=&quot;first_attempt&quot;)

peak_count_nolambda_H3K27ac_05 %&gt;% 
  rbind(peak_count_nolambda_H3K27ac_01) %&gt;% 
  rbind(peak_count_llocal20k_H3K27ac_05) %&gt;% 
  rbind(peak_count_llocal20k_H3K27ac_01) %&gt;% 
  mutate(X2=gsub(&quot;macs2_nolambda_q05/&quot;,&quot;&quot;, X2)) %&gt;% 
  mutate(X2=gsub(&quot;macs2_nolambda_q01/&quot;,&quot;&quot;, X2))%&gt;% 
  mutate(X2=gsub(&quot;macs2_llocal20k_q01/&quot;,&quot;&quot;, X2)) %&gt;% 
  mutate(X2=gsub(&quot;macs2_llocal20k_q05/&quot;,&quot;&quot;, X2)) %&gt;% 
  mutate(qval=factor(qval, levels =c(&quot;0.05&quot;,&quot;0.01&quot;))) %&gt;% 
   group_by(qval, type) %&gt;% 
  mutate(median_X1 = median(X1)) %&gt;%  # Calculate the median for each qval group
  ungroup() %&gt;% 

  ggplot(., aes(x=X2,y=X1, group = interaction(qval, type) ))+
  geom_point(aes(color= interaction(qval, type) ))+
  geom_path(aes(group = interaction(qval, type), color = interaction(qval, type)), size = 1, linetype = &quot;solid&quot;) +
  geom_point(data = firstPeak_H3K27ac, aes(x=X2,y=X1, group = interaction(qval, type)), color = &quot;black&quot;) +      # manual black color for new data
  geom_path(data = firstPeak_H3K27ac, aes(x=X2,y=X1, group = interaction(qval, type)), color = &quot;black&quot;)  +       # manual black color for new data
  geom_hline(aes(yintercept = median_X1, color = interaction(qval, type)), linetype = &quot;dashed&quot;, size = 1) +
  ylab(&quot;Count of total peaks&quot;)+
  xlab(&quot;samples&quot;)+
  theme_bw()+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  ggtitle(&quot;Number of called peaks by file H3K27ac&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/unnamed-chunk-7-2.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="h3k9me3-llocal-and-nolambda" class="section level4">
<h4>H3K9me3 llocal and nolambda</h4>
<pre class="r"><code>frip_llocal20k_nolambda %&gt;% 
  dplyr::rename(&quot;call_type&quot;=&quot;Total_Reads&quot;,
                &quot;Tot_Read&quot;= &quot;Reads_in_Peaks&quot;) %&gt;%
  separate_wider_delim(.,FRiP, names = c(&quot;Frag_in_peaks&quot;,&quot;Frip&quot;),delim=&quot;\t&quot;) %&gt;% 
  mutate(Frag_in_peaks=as.numeric(Frag_in_peaks),Frip=as.numeric(Frip)) %&gt;% 
  mutate(call_type=factor(call_type, levels = c(&quot;macs2_nolambda_q05&quot;, &quot;macs2_nolambda_q01&quot;,&quot;macs2_llocal20k_q05&quot;, &quot;macs2_llocal20k_q01&quot;))) %&gt;% 
  dplyr::filter(Histone == &quot;H3K9me3&quot;) %&gt;% 
   group_by(call_type) %&gt;% 
  mutate(median_Frip = median(Frip)) %&gt;%  # Calculate the median for each qval group
  ungroup() %&gt;% 
  ggplot(., aes(x=Sample,y=Frip))+
  geom_point(aes(color=call_type))+
  geom_path(aes(group = call_type, color = call_type), size = 1, linetype = &quot;solid&quot;) +
  geom_hline(aes(yintercept = median_Frip, color = call_type), linetype = &quot;dashed&quot;, size = 1)+
  ggtitle(&quot;Individual file Frip by Histone H3K9me3&quot;)+
  # xlab(&quot;samples&quot;)+
  theme_bw()+
  theme(axis.text.x=element_text(vjust = .2,angle=90))</code></pre>
<p><img src="figure/Thresholding.Rmd/unnamed-chunk-8-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>peak_count_nolambda_H3K9me3_05 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/macs2_nolambda_q05/peak_count_nolambda_q05.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.narrowPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K9me3&quot;, type=&quot;nolambda&quot;) %&gt;% 
  mutate(qval=&quot;0.05&quot;)
peak_count_nolambda_H3K9me3_01 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/macs2_nolambda_q01/peak_count_nolambda_q01.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.narrowPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K9me3&quot;,type = &quot;nolambda&quot;) %&gt;% 
  mutate(qval=&quot;0.01&quot;)

peak_count_llocal20k_H3K9me3_05 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/macs2_llocal20k_q05/peak_count_llocal120k_q05.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.narrowPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K9me3&quot;,type = &quot;llocal20k&quot;) %&gt;% 
  mutate(qval=&quot;0.05&quot;)
peak_count_llocal20k_H3K9me3_01 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K9me3/macs2_llocal20k_q01/peak_count_llocal120k_q01.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.narrowPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K9me3&quot;, type = &quot;llocal20k&quot;) %&gt;% 
  
  mutate(qval=&quot;0.01&quot;)



peak_count_nolambda_H3K9me3_05 %&gt;% 
  rbind(peak_count_nolambda_H3K9me3_01) %&gt;% 
  rbind(peak_count_llocal20k_H3K9me3_05) %&gt;% 
  rbind(peak_count_llocal20k_H3K9me3_01) %&gt;% 
  mutate(X2=gsub(&quot;macs2_nolambda_q05/&quot;,&quot;&quot;, X2)) %&gt;% 
  mutate(X2=gsub(&quot;macs2_nolambda_q01/&quot;,&quot;&quot;, X2))%&gt;% 
  mutate(X2=gsub(&quot;macs2_llocal20k_q01/&quot;,&quot;&quot;, X2)) %&gt;% 
  mutate(X2=gsub(&quot;macs2_llocal20k_q05/&quot;,&quot;&quot;, X2)) %&gt;% 
  mutate(qval=factor(qval, levels =c(&quot;0.05&quot;,&quot;0.01&quot;))) %&gt;% 
   group_by(qval, type) %&gt;% 
  mutate(median_X1 = median(X1)) %&gt;%  # Calculate the median for each qval group
  ungroup() %&gt;% 
  ggplot(., aes(x=X2,y=X1, group = interaction(qval, type) ))+
  geom_point(aes(color= interaction(qval, type) ))+
  geom_path(aes(group = interaction(qval, type), color = interaction(qval, type)), size = 1, linetype = &quot;solid&quot;) +
  geom_hline(aes(yintercept = median_X1, color = interaction(qval, type)), linetype = &quot;dashed&quot;, size = 1) +
  ylab(&quot;Count of total peaks&quot;)+
  xlab(&quot;samples&quot;)+
  theme_bw()+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  ggtitle(&quot;Number of called peaks by file H3K9me3&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/unnamed-chunk-8-2.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="h3k27me3-llocal-and-nolambda" class="section level4">
<h4>H3K27me3 llocal and nolambda</h4>
<pre class="r"><code>frip_llocal20k_nolambda %&gt;% 
  dplyr::rename(&quot;call_type&quot;=&quot;Total_Reads&quot;,
                &quot;Tot_Read&quot;= &quot;Reads_in_Peaks&quot;) %&gt;%
  separate_wider_delim(.,FRiP, names = c(&quot;Frag_in_peaks&quot;,&quot;Frip&quot;),delim=&quot;\t&quot;) %&gt;% 
  mutate(Frag_in_peaks=as.numeric(Frag_in_peaks),Frip=as.numeric(Frip)) %&gt;% 
  mutate(call_type=factor(call_type, levels = c(&quot;macs2_nolambda_q05&quot;, &quot;macs2_nolambda_q01&quot;,&quot;macs2_llocal20k_q05&quot;, &quot;macs2_llocal20k_q01&quot;))) %&gt;% 
  dplyr::filter(Histone == &quot;H3K27me3&quot;) %&gt;% 
   group_by(call_type) %&gt;% 
  mutate(median_Frip = median(Frip)) %&gt;%  # Calculate the median for each qval group
  ungroup() %&gt;% 
  ggplot(., aes(x=Sample,y=Frip))+
  geom_point(aes(color=call_type))+
  geom_path(aes(group = call_type, color = call_type), size = 1, linetype = &quot;solid&quot;) +
  geom_hline(aes(yintercept = median_Frip, color = call_type), linetype = &quot;dashed&quot;, size = 1)+
  ggtitle(&quot;Individual file Frip by Histone H3K27me3&quot;)+
  # xlab(&quot;samples&quot;)+
  theme_bw()+
  theme(axis.text.x=element_text(vjust = .2,angle=90))</code></pre>
<p><img src="figure/Thresholding.Rmd/unnamed-chunk-9-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>peak_count_nolambda_H3K27me3_05 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/macs2_nolambda_q05/peak_count_nolambda_q05.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.broadPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K27me3&quot;) %&gt;% 
  mutate(qval=&quot;0.05&quot;, type = &quot;nolambda&quot;)


peak_count_nolambda_H3K27me3_01 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/macs2_nolambda_q01/peak_count_nolambda_q01.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.broadPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K27me3&quot;) %&gt;% 
  mutate(qval=&quot;0.01&quot;,type = &quot;nolambda&quot;)

peak_count_nolambda_bcq_H3K27me3_05 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/macs2_nolambda_bcq05/peak_count_nolambda_bcq05.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.broadPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K27me3&quot;, type = &quot;bcq&quot;) %&gt;% 
  mutate(qval=&quot;0.05&quot;)
peak_count_nolambda_bcq_H3K27me3_01 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/macs2_nolambda_bcq01/peak_count_nolambda_bcq01.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.broadPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K27me3&quot;,type = &quot;bcq&quot;) %&gt;% 
  mutate(qval=&quot;0.01&quot;)





peak_count_llocal20k_H3K27me3_05 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/macs2_llocal20k_q05/peak_count_llocal20k_q05.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.broadPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K27me3&quot;) %&gt;% 
  mutate(qval=&quot;0.05&quot;,type = &quot;llocal20k&quot;)

peak_count_llocal20k_H3K27me3_01 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K27me3/macs2_llocal20k_q01/peak_count_llocal20k_q01.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.broadPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K27me3&quot;) %&gt;% 
  mutate(qval=&quot;0.01&quot;,type = &quot;llocal20k&quot;)

################looking at previous values#########################
firstPeak_H3K27me3 &lt;- peak_count_H3K27me3_05 %&gt;% 
  rbind(peak_count_H3K27me3_01) %&gt;% 
  # rbind(peak_count_H3K27me3_005) %&gt;% 
  mutate(X2=gsub(&quot;macs2_nolambda_q05/&quot;,&quot;&quot;, X2)) %&gt;% 
  mutate(X2=gsub(&quot;macs2_peak_q01/&quot;,&quot;&quot;, X2)) %&gt;% 
    # mutate(X2=gsub(&quot;macs2_peak_q005/&quot;,&quot;&quot;, X2)) %&gt;% 
  mutate(qval=factor(qval, levels =c(&quot;0.05&quot;,&quot;0.01&quot;,&quot;0.005&quot;))) %&gt;% 
   group_by(qval) %&gt;% 
  mutate(median_X1 = median(X1), type = &quot;first_run&quot;) %&gt;%  # Calculate the median for each qval group
  ungroup() 


peak_count_nolambda_H3K27me3_05 %&gt;% 
  rbind(peak_count_nolambda_H3K27me3_01) %&gt;% 
  rbind(peak_count_llocal20k_H3K27me3_05) %&gt;% 
  rbind(peak_count_llocal20k_H3K27me3_01) %&gt;% 
  rbind(peak_count_nolambda_bcq_H3K27me3_05) %&gt;% 
   rbind(peak_count_nolambda_bcq_H3K27me3_01) %&gt;%
  mutate(X2=gsub(&quot;macs2_nolambda_q05/&quot;,&quot;&quot;, X2)) %&gt;% 
  mutate(X2=gsub(&quot;macs2_nolambda_q01/&quot;,&quot;&quot;, X2))%&gt;% 
  mutate(X2=gsub(&quot;macs2_llocal20k_q01/&quot;,&quot;&quot;, X2)) %&gt;% 
  mutate(X2=gsub(&quot;macs2_llocal20k_q05/&quot;,&quot;&quot;, X2)) %&gt;% 
    mutate(X2=gsub(&quot;macs2_nolambda_bcq05/&quot;,&quot;&quot;, X2)) %&gt;% 
  mutate(X2=gsub(&quot;macs2_nolambda_bcq01/&quot;,&quot;&quot;, X2))%&gt;%
  mutate(qval=factor(qval, levels =c(&quot;0.05&quot;,&quot;0.01&quot;))) %&gt;% 
   group_by(qval, type) %&gt;% 
  mutate(median_X1 = median(X1)) %&gt;%  # Calculate the median for each qval group
  ungroup() %&gt;% 
  ggplot(., aes(x=X2,y=X1, group = interaction(qval, type) ))+
  geom_point(aes(color= interaction(qval, type) ))+
  geom_path(aes(group = interaction(qval, type), color = interaction(qval, type)), size = 1, linetype = &quot;solid&quot;) +
  # geom_point(data = firstPeak_H3K27me3, aes(x=X2,y=X1, group = interaction(qval, type)), color = &quot;black&quot;) +      # manual black color for new data
  # geom_path(data = firstPeak_H3K27me3, aes(x=X2,y=X1, group = interaction(qval, type)), color = &quot;black&quot;, linetype=&quot;dashed&quot;)  +       # manual black color for new data
  
  geom_hline(aes(yintercept = median_X1, color = interaction(qval, type)), linetype = &quot;dashed&quot;, size = 1) +
  ylab(&quot;Count of total peaks&quot;)+
  xlab(&quot;samples&quot;)+
  theme_bw()+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  ggtitle(&quot;Number of called peaks by file H3K27me3&quot;)</code></pre>
<p><img src="figure/Thresholding.Rmd/unnamed-chunk-9-2.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="h3k36me3-llocal-and-nolambda" class="section level4">
<h4>H3K36me3 llocal and nolambda</h4>
<pre class="r"><code>frip_llocal20k_nolambda %&gt;% 
  dplyr::rename(&quot;call_type&quot;=&quot;Total_Reads&quot;,
                &quot;Tot_Read&quot;= &quot;Reads_in_Peaks&quot;) %&gt;%
  separate_wider_delim(.,FRiP, names = c(&quot;Frag_in_peaks&quot;,&quot;Frip&quot;),delim=&quot;\t&quot;) %&gt;% 
  mutate(Frag_in_peaks=as.numeric(Frag_in_peaks),Frip=as.numeric(Frip)) %&gt;% 
  mutate(call_type=factor(call_type, levels = c(&quot;macs2_nolambda_q05&quot;, &quot;macs2_nolambda_q01&quot;,&quot;macs2_llocal20k_q05&quot;, &quot;macs2_llocal20k_q01&quot;))) %&gt;% 
  dplyr::filter(Histone == &quot;H3K36me3&quot;) %&gt;% 
   group_by(call_type) %&gt;% 
  mutate(median_Frip = median(Frip)) %&gt;%  # Calculate the median for each qval group
  ungroup() %&gt;% 
  ggplot(., aes(x=Sample,y=Frip))+
  geom_point(aes(color=call_type))+
  geom_path(aes(group = call_type, color = call_type), size = 1, linetype = &quot;solid&quot;) +
  geom_hline(aes(yintercept = median_Frip, color = call_type), linetype = &quot;dashed&quot;, size = 1)+
  ggtitle(&quot;Individual file Frip by Histone H3K36me3&quot;)+
  # xlab(&quot;samples&quot;)+
  theme_bw()+
  theme(axis.text.x=element_text(vjust = .2,angle=90))</code></pre>
<p><img src="figure/Thresholding.Rmd/unnamed-chunk-10-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>peak_count_nolambda_H3K36me3_05 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/macs2_nolambda_q05/peak_count_nolambda_q05.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.broadPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K36me3&quot;) %&gt;% 
  mutate(qval=&quot;0.05&quot;)
peak_count_nolambda_H3K36me3_01 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/macs2_nolambda_q01/peak_count_nolambda_q01.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.broadPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K36me3&quot;) %&gt;% 
  mutate(qval=&quot;0.01&quot;)


peak_count_nolambda_bcq_H3K36me3_05 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/macs2_nolambda_bcq05/peak_count_nolambda_bcq05.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.broadPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K36me3&quot;, type = &quot;bcq&quot;) %&gt;% 
  mutate(qval=&quot;0.05&quot;)
peak_count_nolambda_bcq_H3K36me3_01 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/macs2_nolambda_bcq01/peak_count_nolambda_bcq01.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.broadPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K36me3&quot;,type = &quot;bcq&quot;) %&gt;% 
  mutate(qval=&quot;0.01&quot;)

peak_count_llocal20k_H3K36me3_05 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/macs2_llocal20k_q05/peak_count_llocal20k_q05.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.broadPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K36me3&quot;) %&gt;% 
  mutate(qval=&quot;0.05&quot;)
peak_count_llocal20k_H3K36me3_01 &lt;- read_table(&quot;C:/Users/renee/Other_projects_data/DXR_data/prelim-data/H3K36me3/macs2_llocal20k_q01/peak_count_llocal20k_q01.txt&quot;, 
    col_names = FALSE) %&gt;% 
  dplyr::filter(X2 != &quot;total&quot;) %&gt;% 
   mutate(X2=gsub(&quot;_noM_peaks.broadPeak&quot;,&quot;&quot;,X2)) %&gt;% 
    mutate(histone=&quot;H3K36me3&quot;) %&gt;% 
  mutate(qval=&quot;0.01&quot;)
peak_count_nolambda_H3K36me3_05</code></pre>
<pre><code># A tibble: 16 × 4
       X1 X2                                histone  qval 
    &lt;dbl&gt; &lt;chr&gt;                             &lt;chr&gt;    &lt;chr&gt;
 1 360234 macs2_nolambda_q05/MCW_CT_ROA_171 H3K36me3 0.05 
 2 376201 macs2_nolambda_q05/MCW_CT_ROA_172 H3K36me3 0.05 
 3 115840 macs2_nolambda_q05/MCW_CT_ROA_174 H3K36me3 0.05 
 4 151336 macs2_nolambda_q05/MCW_CT_ROA_175 H3K36me3 0.05 
 5 348299 macs2_nolambda_q05/MCW_CT_ROA_177 H3K36me3 0.05 
 6 259283 macs2_nolambda_q05/MCW_CT_ROA_180 H3K36me3 0.05 
 7 265939 macs2_nolambda_q05/MCW_CT_ROA_181 H3K36me3 0.05 
 8 362141 macs2_nolambda_q05/MCW_CT_ROA_183 H3K36me3 0.05 
 9 384188 macs2_nolambda_q05/MCW_CT_ROA_189 H3K36me3 0.05 
10 338593 macs2_nolambda_q05/MCW_CT_ROA_192 H3K36me3 0.05 
11 399252 macs2_nolambda_q05/MCW_CT_ROA_195 H3K36me3 0.05 
12 290749 macs2_nolambda_q05/MCW_CT_ROA_202 H3K36me3 0.05 
13 293162 macs2_nolambda_q05/MCW_CT_ROA_207 H3K36me3 0.05 
14 287015 macs2_nolambda_q05/MCW_CT_ROA_208 H3K36me3 0.05 
15 243064 macs2_nolambda_q05/MCW_CT_ROA_210 H3K36me3 0.05 
16 314605 macs2_nolambda_q05/MCW_CT_ROA_211 H3K36me3 0.05 </code></pre>
</div>
<div id="duplication-rates-and-frippeak-calls" class="section level4">
<h4>Duplication rates and FRiP/peak calls</h4>
<pre class="r"><code>sampleinfo &lt;- read_delim(&quot;data/sampleinfo.txt&quot;, delim = &quot;\t&quot;)
multiqc_gene_stats_trim &lt;- read_delim(&quot;data/multiqc_genral_stats_trim.txt&quot;,delim = &quot;\t&quot;)
multiqc_fastqc_trim &lt;- read_delim(&quot;data/multiqc_fastqc_trim.txt&quot;,delim = &quot;\t&quot;)
    
combo_trim_df &lt;- multiqc_fastqc_trim %&gt;% 
 extract(., Sample, into = c(&quot;prefix&quot;,&quot;read&quot;), regex= &quot;(.+)_R(\\d+)&quot;, remove=FALSE) %&gt;% 
  mutate(read = paste0(&quot;R&quot;, read)) %&gt;% 
  left_join(., sampleinfo, by =c(&quot;prefix&quot;=&quot;library_ID&quot;)) %&gt;% 
  left_join(., multiqc_gene_stats_trim, by = c(&quot;Sample&quot; = &quot;Sample&quot;)) %&gt;% 
  mutate(ind=factor(ind, levels = c(&quot;1&quot;,&quot;2&quot;,&quot;3&quot;,&quot;4&quot;,&quot;5&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels = c(&quot;VEH&quot;,&quot;5FU&quot;,&quot;DOX&quot;))) %&gt;% 
  mutate(other_time=factor(other_time, levels=c(&quot;24t&quot;,&quot;24r&quot;,&quot;144r&quot;)))


combo_trim_df %&gt;% 
  dplyr::filter(trt!=&quot;5FU&quot;&amp;histone_mark==&quot;H3K27ac&quot;) %&gt;% 
  ggplot(., aes(x=interaction(read,prefix),y=`FastQC_mqc-generalstats-fastqc-percent_duplicates`))+
  geom_col(aes(fill=interaction(trt,other_time)))+
  ggtitle(&quot;Duplication rates by sample for H3K27ac data&quot;)+
  xlab(&quot;sample&quot;)+
  ylab(&quot;Percent duplication&quot;)+
  theme_bw()+
  theme(axis.text.x=element_text(vjust = .2,angle=90))</code></pre>
<p><img src="figure/Thresholding.Rmd/unnamed-chunk-11-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>combo_trim_df %&gt;% 
  dplyr::filter(trt!=&quot;5FU&quot;&amp;histone_mark==&quot;H3K27me3&quot;) %&gt;% 
  ggplot(., aes(x=interaction(read,prefix),y=`FastQC_mqc-generalstats-fastqc-percent_duplicates`))+
  geom_col(aes(fill=interaction(trt,other_time)))+
  ggtitle(&quot;Duplication rates by sample for H3K27me3 data&quot;)+
  xlab(&quot;sample&quot;)+
  ylab(&quot;Percent duplication&quot;)+
  theme_bw()+
  theme(axis.text.x=element_text(vjust = .2,angle=90))</code></pre>
<p><img src="figure/Thresholding.Rmd/unnamed-chunk-11-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>combo_trim_df %&gt;% 
  dplyr::filter(trt!=&quot;5FU&quot;&amp;histone_mark==&quot;H3K36me3&quot;) %&gt;% 
  ggplot(., aes(x=interaction(read,prefix),y=`FastQC_mqc-generalstats-fastqc-percent_duplicates`))+
  geom_col(aes(fill=interaction(trt,other_time)))+
  ggtitle(&quot;Duplication rates by sample for H3K36me3 data&quot;)+
  xlab(&quot;sample&quot;)+
  ylab(&quot;Percent duplication&quot;)+
  theme_bw()+
  theme(axis.text.x=element_text(vjust = .2,angle=90))</code></pre>
<p><img src="figure/Thresholding.Rmd/unnamed-chunk-11-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>combo_trim_df %&gt;% 
  dplyr::filter(trt!=&quot;5FU&quot;&amp;histone_mark==&quot;H3K9me3&quot;) %&gt;% 
  ggplot(., aes(x=interaction(read,prefix),y=`FastQC_mqc-generalstats-fastqc-percent_duplicates`))+
  geom_col(aes(fill=interaction(trt,other_time)))+
  ggtitle(&quot;Duplication rates by sample for H3K9me3 data&quot;)+
  xlab(&quot;sample&quot;)+
  ylab(&quot;Percent duplication&quot;)+
  theme_bw()+
  theme(axis.text.x=element_text(vjust = .2,angle=90))</code></pre>
<p><img src="figure/Thresholding.Rmd/unnamed-chunk-11-4.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.2 (2024-10-31 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 11 x64 (build 26100)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] stats4    grid      stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] GenomicRanges_1.58.0  GenomeInfoDb_1.42.3   IRanges_2.40.1       
 [4] S4Vectors_0.44.0      BiocGenerics_0.52.0   data.table_1.17.0    
 [7] genomation_1.38.0     ComplexHeatmap_2.22.0 edgeR_4.4.2          
[10] limma_3.62.2          DT_0.33               kableExtra_1.4.0     
[13] lubridate_1.9.4       forcats_1.0.0         stringr_1.5.1        
[16] dplyr_1.1.4           purrr_1.0.4           readr_2.1.5          
[19] tidyr_1.3.1           tibble_3.2.1          ggplot2_3.5.1        
[22] tidyverse_2.0.0       workflowr_1.7.1      

loaded via a namespace (and not attached):
  [1] RColorBrewer_1.1-3          rstudioapi_0.17.1          
  [3] jsonlite_1.9.1              shape_1.4.6.1              
  [5] magrittr_2.0.3              farver_2.1.2               
  [7] rmarkdown_2.29              GlobalOptions_0.1.2        
  [9] fs_1.6.5                    BiocIO_1.16.0              
 [11] zlibbioc_1.52.0             vctrs_0.6.5                
 [13] Rsamtools_2.22.0            RCurl_1.98-1.16            
 [15] htmltools_0.5.8.1           S4Arrays_1.6.0             
 [17] plotrix_3.8-4               curl_6.2.1                 
 [19] SparseArray_1.6.2           sass_0.4.9                 
 [21] KernSmooth_2.23-26          bslib_0.9.0                
 [23] htmlwidgets_1.6.4           plyr_1.8.9                 
 [25] impute_1.80.0               cachem_1.1.0               
 [27] GenomicAlignments_1.42.0    whisker_0.4.1              
 [29] lifecycle_1.0.4             iterators_1.0.14           
 [31] pkgconfig_2.0.3             Matrix_1.7-3               
 [33] R6_2.6.1                    fastmap_1.2.0              
 [35] GenomeInfoDbData_1.2.13     MatrixGenerics_1.18.1      
 [37] clue_0.3-66                 digest_0.6.37              
 [39] colorspace_2.1-1            ps_1.9.0                   
 [41] rprojroot_2.0.4             labeling_0.4.3             
 [43] timechange_0.3.0            httr_1.4.7                 
 [45] abind_1.4-8                 compiler_4.4.2             
 [47] bit64_4.6.0-1               withr_3.0.2                
 [49] doParallel_1.0.17           BiocParallel_1.40.0        
 [51] DelayedArray_0.32.0         rjson_0.2.23               
 [53] tools_4.4.2                 httpuv_1.6.15              
 [55] glue_1.8.0                  restfulr_0.0.15            
 [57] callr_3.7.6                 promises_1.3.2             
 [59] getPass_0.2-4               gridBase_0.4-7             
 [61] cluster_2.1.8.1             reshape2_1.4.4             
 [63] generics_0.1.3              gtable_0.3.6               
 [65] BSgenome_1.74.0             tzdb_0.4.0                 
 [67] seqPattern_1.38.0           hms_1.1.3                  
 [69] utf8_1.2.4                  xml2_1.3.7                 
 [71] XVector_0.46.0              ggrepel_0.9.6              
 [73] foreach_1.5.2               pillar_1.10.1              
 [75] vroom_1.6.5                 later_1.4.1                
 [77] circlize_0.4.16             lattice_0.22-6             
 [79] rtracklayer_1.66.0          bit_4.6.0                  
 [81] tidyselect_1.2.1            locfit_1.5-9.12            
 [83] Biostrings_2.74.1           knitr_1.49                 
 [85] git2r_0.35.0                SummarizedExperiment_1.36.0
 [87] svglite_2.1.3               xfun_0.51                  
 [89] Biobase_2.66.0              statmod_1.5.0              
 [91] matrixStats_1.5.0           stringi_1.8.4              
 [93] UCSC.utils_1.2.0            yaml_2.3.10                
 [95] evaluate_1.0.3              codetools_0.2-20           
 [97] cli_3.6.4                   systemfonts_1.2.1          
 [99] munsell_0.5.1               processx_3.8.6             
[101] jquerylib_0.1.4             Rcpp_1.0.14                
[103] png_0.1-8                   XML_3.99-0.18              
[105] parallel_4.4.2              bitops_1.0-9               
[107] viridisLite_0.4.2           scales_1.3.0               
[109] crayon_1.5.3                GetoptLong_1.0.5           
[111] rlang_1.1.5                </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
