---
title: "sequencing QC"
author: "Renee Matthews"
date: "2025-03-13"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE,
			dev = c("png","pdf")
	
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```

##### Loading Packages

```{r package loading}
library(tidyverse)
library(readr)
library(kableExtra)
library(DT)
library(readr)
library(edgeR)
library(ComplexHeatmap)
library(data.table)
library(genomation)
library(GenomicRanges)
```

### data loading

```{r data loading}

sampleinfo <- read_delim("data/sampleinfo.txt", delim = "\t")

multiqc_gene_stats_notrim <- read_delim("data/multiqc_genral_stats_notrim.txt",delim = "\t")
multiqc_gene_stats_trim <- read_delim("data/multiqc_genral_stats_trim.txt",delim = "\t")

multiqc_fastqc_notrim <- read_delim("data/multiqc_fastqc_notrim.txt",delim = "\t")
multiqc_fastqc_trim <- read_delim("data/multiqc_fastqc_trim.txt",delim = "\t")
    
```
###### functions

```{r}
drug_pal <- c("#8B006D","#DF707E","#F1B72B", "#3386DD","#707031","#41B333")
pca_plot <-
  function(df,
           col_var = NULL,
           shape_var = NULL,
           title = "") {
    ggplot(df) + geom_point(aes_string(
      x = "PC1",
      y = "PC2",
      color = col_var,
      shape = shape_var
    ),
    size = 5) +
      labs(title = title, x = "PC 1", y = "PC 2") +
      scale_color_manual(values = c(
        "#8B006D",
        "#DF707E",
        "#F1B72B",
        "#3386DD",
        "#707031",
        "#41B333"
      ))
  }
pca_var_plot <- function(pca) {
  # x: class == prcomp
  pca.var <- pca$sdev ^ 2
  pca.prop <- pca.var / sum(pca.var)
  var.plot <-
    qplot(PC, prop, data = data.frame(PC = 1:length(pca.prop),
                                      prop = pca.prop)) +
    labs(title = 'Variance contributed by each PC',
         x = 'PC', y = 'Proportion of variance')
  plot(var.plot)
}

calc_pca <- function(x) {
  # Performs principal components analysis with prcomp
  # x: a sample-by-gene numeric matrix
  prcomp(x, scale. = TRUE, retx = TRUE)
}

get_regr_pval <- function(mod) {
  # Returns the p-value for the Fstatistic of a linear model
  # mod: class lm
  stopifnot(class(mod) == "lm")
  fstat <- summary(mod)$fstatistic
  pval <- 1 - pf(fstat[1], fstat[2], fstat[3])
  return(pval)
}

plot_versus_pc <- function(df, pc_num, fac) {
  # df: data.frame
  # pc_num: numeric, specific PC for plotting
  # fac: column name of df for plotting against PC
  pc_char <- paste0("PC", pc_num)
  # Calculate F-statistic p-value for linear model
  pval <- get_regr_pval(lm(df[, pc_char] ~ df[, fac]))
  if (is.numeric(df[, f])) {
    ggplot(df, aes_string(x = f, y = pc_char)) + geom_point() +
      geom_smooth(method = "lm") + labs(title = sprintf("p-val: %.2f", pval))
  } else {
    ggplot(df, aes_string(x = f, y = pc_char)) + geom_boxplot() +
      labs(title = sprintf("p-val: %.2f", pval))
  }
}
x_axis_labels = function(labels, every_nth = 1, ...) {
  axis(side = 1,
       at = seq_along(labels),
       labels = F)
  text(
    x = (seq_along(labels))[seq_len(every_nth) == 1],
    y = par("usr")[3] - 0.075 * (par("usr")[4] - par("usr")[3]),
    labels = labels[seq_len(every_nth) == 1],
    xpd = TRUE,
    ...
  )
}

```

######## Combining the dataframes for visualization

```{r datacombine}
combo_notrim_df <- multiqc_fastqc_notrim %>% 
 extract(., Sample, into = c("prefix","read"), regex= "(.+)_R(\\d+)", remove=FALSE) %>% 
  mutate(read = paste0("R", read)) %>% 
  left_join(., sampleinfo, by =c("prefix"="library_ID")) %>% 
  left_join(., multiqc_gene_stats_notrim, by = c("Sample" = "Sample")) %>% 
  mutate(ind=factor(ind, levels = c("1","2","3","4","5"))) %>% 
  mutate(trt=factor(trt, levels = c("VEH","5FU","DOX"))) %>% 
  mutate(other_time=factor(other_time, levels=c("24t","24r","144r")))

combo_trim_df <- multiqc_fastqc_trim %>% 
 extract(., Sample, into = c("prefix","read"), regex= "(.+)_R(\\d+)", remove=FALSE) %>% 
  mutate(read = paste0("R", read)) %>% 
  left_join(., sampleinfo, by =c("prefix"="library_ID")) %>% 
  left_join(., multiqc_gene_stats_trim, by = c("Sample" = "Sample")) %>% 
  mutate(ind=factor(ind, levels = c("1","2","3","4","5"))) %>% 
  mutate(trt=factor(trt, levels = c("VEH","5FU","DOX"))) %>% 
  mutate(other_time=factor(other_time, levels=c("24t","24r","144r")))
```

### Basic Stats

```{r Basic stats}
combo_notrim_df %>% 
  dplyr::filter(read=="R1") %>% 
  group_by(trt,other_time,histone_mark) %>% 
  tally() %>% 
  ggplot(., aes(x = other_time, y= n))+
  geom_col(position="dodge",aes(fill=trt)) + 
  facet_wrap(~histone_mark)+
  theme(axis.text.x=element_text(angle=90))+
  ylab("number of samples")+
  ggtitle("Breakdown of samples by mark and trt-time")

combo_notrim_df %>% 
  dplyr::filter(read=="R1") %>% 
  group_by(trt,other_time,histone_mark) %>% 
  tally() %>% 
  pivot_wider(., id_cols=c(trt,other_time), names_from = histone_mark, values_from = n) %>% 
  kable(.,caption = ("Sample counts")) %>% 
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(full_width = FALSE,font_size = 16) %>%
  scroll_box(width = "100%", height = "500px")

```


### Visualization of counts

```{r counts viz, fig.height=8, fig.width=12}

combo_notrim_df %>% 
  dplyr::filter(read=="R1") %>% 
  ggplot(., aes(x = Sample, y= `Total Sequences`))+
  geom_col(aes(fill=histone_mark)) + 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  ylab("sequene count")+
  ggtitle("Read counts by sample and histone mark")+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))

combo_trim_df %>% 
  dplyr::filter(read=="R1") %>% 
  ggplot(., aes(x = Sample, y= `Total Sequences`))+
  geom_col(aes(fill=histone_mark)) + 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  ylab("sequene count")+
  ggtitle("Read counts by sample and histone mark trimmed adapters")+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))



combo_notrim_df %>% 
  dplyr::filter(read=="R1") %>% 
   ggplot(., aes(x = histone_mark, y= `Total Sequences`))+
geom_boxplot(aes(fill=histone_mark)) + 
    geom_point(aes(color=ind))+
  facet_wrap(trt~other_time)+
  ylab("count")+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  ggtitle("Sequence count by trt and time no trim")

combo_trim_df %>% 
  dplyr::filter(read=="R1") %>% 
   ggplot(., aes(x = histone_mark, y= `Total Sequences`))+
geom_boxplot(aes(fill=histone_mark)) + 
    geom_point(aes(color=ind))+
  facet_wrap(trt~other_time)+
  ylab("count")+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  ggtitle("Sequence count by trt and time trimmed adapter")


```

### Trim information

```{r trim info1}
combo_trim_df %>% 
   ggplot(., aes(x = read, y= avg_sequence_length))+
geom_boxplot(aes(fill=read))

combo_trim_df %>% 
  ggplot(., aes(x = read, y= avg_sequence_length))+
geom_boxplot(aes(fill=histone_mark)) +
  ggtitle("Boxplot of trim read length across histone marks")

combo_trim_df %>% 
  datatable(., options = list(scrollX = TRUE, 
                              scrollY = "400px",
                              scrollCollapse = TRUE,
                              fixedColumns = list(leftColumns =2),
                              fixedHeader= TRUE),
            extensions = c("FixedColumns","Scroller"),
            class = "display")

# combo_trim_df %>% 
#   dplyr::select(prefix,histone_mark,sample_ID:other_time,`Total Sequences`) %>% 
#   distinct() %>% 
#   mutate(File_name=paste0(prefix,".bed"))


```


```{r trim info2,fig.height=8, fig.width=12}
combo_trim_df %>% 
  dplyr::filter(read=="R1") %>% 
  ggplot(., aes(x = Sample, y= avg_sequence_length))+
geom_col(aes(fill=histone_mark)) +
  geom_hline( yintercept = 75)+
  theme_classic()+
  ggtitle("Graph of average read length across R1 samples")+ theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))

combo_trim_df %>% 
  dplyr::filter(read=="R2") %>% 
  ggplot(., aes(x = Sample, y= avg_sequence_length))+
geom_col(aes(fill=histone_mark)) +
  geom_hline( yintercept = 75)+
  theme_classic()+
  ggtitle("Graph of average read length across R2 samples")+ theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))
  


```

percent gc

```{r GCper,fig.height=6, fig.width=12}
combo_notrim_df %>% 
  dplyr::filter(read=="R1") %>% 
  ggplot(., aes(x = Sample, y= `%GC`))+
  geom_col(aes(fill=histone_mark)) +
  theme_classic()+
  ggtitle("Graph of %GC for R1 not-trimmed")+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))


combo_trim_df %>% 
  dplyr::filter(read=="R1") %>% 
  ggplot(., aes(x = Sample, y= `%GC`))+
  geom_col(aes(fill=histone_mark)) +
  theme_classic()+
  ggtitle("Graph of %GC for R1 trimmed")+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))

combo_notrim_df %>% 
  dplyr::filter(read=="R2") %>% 
  ggplot(., aes(x = Sample, y= `%GC`))+
  geom_col(aes(fill=histone_mark)) +
  theme_classic()+
  ggtitle("Graph of %GC for R2 not-trimmed")+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))


combo_trim_df %>% 
  dplyr::filter(read=="R2") %>% 
  ggplot(., aes(x = Sample, y= `%GC`))+
  geom_col(aes(fill=histone_mark)) +
  theme_classic()+
  ggtitle("Graph of %GC for R2 trimmed")+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))


```


Duplication information
```{r dup viz, fig.height=8, fig.width=8}
combo_notrim_df %>% 
  dplyr::filter(read=="R1") %>% 
  ggplot(., aes(x = histone_mark, y= `FastQC_mqc-generalstats-fastqc-percent_duplicates`))+
  geom_boxplot(aes(fill=histone_mark)) + 
    geom_point(aes(color=ind))+
  facet_wrap(trt~other_time)+
  ylab("percent duplication")+
  theme(axis.text.x=element_text(angle=90))+
   ggtitle("Duplication percentage (R1 no trim)")

combo_trim_df %>% 
  dplyr::filter(read=="R1") %>% 
  ggplot(., aes(x = histone_mark, y= `FastQC_mqc-generalstats-fastqc-percent_duplicates`))+
  geom_boxplot(aes(fill=histone_mark)) + 
    geom_point(aes(color=ind))+
  facet_wrap(trt~other_time)+
  ylab("percent duplication")+
  theme(axis.text.x=element_text(angle=90))+
  ggtitle("Duplication percentage (R1 trimmed)")
  


combo_notrim_df %>% 
  dplyr::filter(read=="R1") %>% 
  ggplot(., aes(x = interaction(other_time,trt), y= `FastQC_mqc-generalstats-fastqc-percent_duplicates`))+
  geom_boxplot(aes(fill=histone_mark)) + 
  geom_point(aes(color=ind))+
facet_wrap(~histone_mark)+
  ylab("percent duplication")+
  theme(axis.text.x=element_text(angle=90))+
  ggtitle("Duplication percentage (R1 no trim)")

combo_trim_df %>% 
  dplyr::filter(read=="R1") %>% 
  ggplot(., aes(x = interaction(other_time,trt), y= `FastQC_mqc-generalstats-fastqc-percent_duplicates`))+
  geom_boxplot(aes(fill=histone_mark)) + 
  geom_point(aes(color=ind))+
facet_wrap(~histone_mark)+
  ylab("percent duplication")+
  theme(axis.text.x=element_text(angle=90))+
  ggtitle("Duplication percentage (R1 trimmed)")


  
```





```{r,Dup all samp,fig.height=6, fig.width=12}
combo_notrim_df %>% 
  dplyr::filter(read=="R1") %>% 
  ggplot(., aes(x = Sample, y= `FastQC_mqc-generalstats-fastqc-percent_duplicates`))+
  geom_col(aes(fill=histone_mark)) +
  theme_classic()+
  ggtitle("Graph of percent duplicates for R1 not-trimmed")+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous(limits = c(0,100), expand = expansion(mult = c(0, .1)))


combo_trim_df %>% 
  dplyr::filter(read=="R1") %>% 
  ggplot(., aes(x = Sample, y= `FastQC_mqc-generalstats-fastqc-percent_duplicates`))+
  geom_col(aes(fill=histone_mark)) +
  theme_classic()+
  ggtitle("Graph of percent duplicates for R1 trimmed")+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( limits = c(0,100),expand = expansion(mult = c(0, .1)))

combo_notrim_df %>% 
  dplyr::filter(read=="R2") %>% 
  ggplot(., aes(x = Sample, y= `FastQC_mqc-generalstats-fastqc-percent_duplicates`))+
  geom_col(aes(fill=histone_mark)) +
  theme_classic()+
  ggtitle("Graph of percent duplicates for R2 not-trimmed")+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous(limits = c(0,100), expand = expansion(mult = c(0, .1)))


combo_trim_df %>% 
  dplyr::filter(read=="R2") %>% 
  ggplot(., aes(x = Sample, y= `FastQC_mqc-generalstats-fastqc-percent_duplicates`))+
  geom_col(aes(fill=histone_mark)) +
  theme_classic()+
  ggtitle("Graph of percent duplicates for R2 trimmed")+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous(limits = c(0,100), expand = expansion(mult = c(0, .1)))


```

```{r code chunk, echo=TRUE, eval=FALSE}
### creating the sample info file from bowtie2 metrics from the first run:

### import files

metric_1 <-  read_delim("C:/Users/reneels/Documents/Ward Lab/5FU_project/Sample_info/first_run_metrics/metric_1.txt", 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE)
metric_2 <- read_delim("C:/Users/renee/Documents/Ward Lab/5FU_project/Sample_info/first_run_metrics/Second_set_info.txt", delim = "\t", escape_double = FALSE, col_names = FALSE, trim_ws = TRUE)
pull_name_file <- read_csv("C:/Users/renee/Documents/Ward Lab/5FU_project/Sample_info/first_run_metrics/samplesheet.txt", col_names = FALSE)
pull <- pull_name_file[65:94,]

 fun <- cbind(pull, metric_2[,1:8])
 colnames(fun) <- c("X1","X2","X3","X4","X5","X6","X7","X8","X9")
 
 # almost_fun <- cbind(pull_name_file[1:65,],metric_1)
 
 ##remove 229 from almost_fun
# subset_almost_fun <- almost_fun[1:64,]

alignment_specs <- rbind(metric_1,fun)

colnames(alignment_specs) <- c("sample_ID",
                "Paired_Reads",
                "aligned_pairs",
                "aligned_concordant_0",
                "zero",
                "aligned_concordant_1",
                "one",
                "aligned_concordant_g1",
                "percent_alignment",
                "N_A")
colnames(fun) <- c("sample_ID",
                "Paired_Reads",
                "aligned_pairs",
                "aligned_concordant_0",
                "zero",
                "aligned_concordant_1",
                "one",
                "aligned_concordant_g1",
                "percent_alignment",
                "N_A")
saveRDS(fun, "data/first_run_alignment_229andover.RDS")
saveRDS(alignment_specs, "data/first_run_alignment_specs.RDS")

```
```{r alignmentspec}

alignment_temp<-readRDS("data/first_run_alignment_specs.RDS")
for_plots <- alignment_temp %>% 
  left_join(.,sampleinfo, by=c("sample_ID"="library_ID"))%>% 
  dplyr::select(sample_ID:aligned_concordant_0,aligned_concordant_1, aligned_concordant_g1,percent_alignment, histone_mark,ind, trt,other_time) %>% 
  distinct()
```

alignment pair numbers
```{r,fig.height=6, fig.width=12}
for_plots %>% 
  group_by(histone_mark) %>% 
  ggplot(., aes(x=sample_ID, y=Paired_Reads))+
    geom_col(aes(fill = histone_mark)) +
  theme_classic()+
  ggtitle("Graph of total paired reads")+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))

for_plots %>% 
  mutate(percent_alignment=parse_number(percent_alignment)) %>% 
  group_by(histone_mark) %>% 
  ggplot(., aes(x=sample_ID, y=percent_alignment))+
    geom_col(aes(fill = histone_mark)) +
  theme_classic()+
  ggtitle("Percent aligned of paired reads") + 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous(expand = expansion(mult = c(0, .1)))+
  coord_cartesian(ylim=c(80,100))


for_plots %>% 
  group_by(histone_mark) %>% 
   mutate(percent_aligned_unique = round(aligned_concordant_1 / Paired_Reads * 100, 2)) %>% 
  ggplot(., aes(x=sample_ID, y=percent_aligned_unique))+
    geom_col(aes(fill = histone_mark)) +
  theme_classic()+
  ggtitle("Percentage of paired reads aligning once")+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))
```
### Fragment analysis
```{r message=FALSE, warning=FALSE}
file_list <- list.files(path="C:/Users/renee/Documents/5FU_files/metric_files",
           pattern ="frag_len_count\\.txt$",full.names = TRUE)
read_and_label <- function(file) {
  df <- read_delim(file, delim = "\t", col_names = c("Col1", "Col2"))  # Adjust delimiter if needed
  df <- df %>%
    mutate(File = basename(file),  # Add filename column
  weight = Col2/sum(Col2))
  return(df)
}
combined_df <- map_df(file_list, read_and_label)

annotated_combo_df <- combined_df %>% 
  mutate(sample = gsub("_frag_len_count.txt","",File)) %>% 
  left_join(., sampleinfo, by = c("sample"="library_ID"))
```

```{r, fig.width=15}
annotated_combo_df %>% 
  dplyr::filter(histone_mark=="H3K9me3") %>% 
  ggplot(., aes(x=Col1, y=Col2, color = sample))+
  geom_line(size=1)+
  # geom_point(size=2,alpha =0.7)+
  scale_x_continuous(breaks = seq(0, max(annotated_combo_df$Col1), by = 50))+
   facet_wrap(~sample)+
  labs(title = "Fragment length, H3K9me3",
       x = "Fragment Length (bp)",
       y = "Count",
       color= "Sample")+
  theme_minimal()
annotated_combo_df %>% 
  dplyr::filter(histone_mark=="H3K9me3") %>%
  ggplot(., aes(x=sample, y=Col1, weight = weight,fill = histone_mark))+
geom_violin(bw = 5) +
    scale_y_continuous(breaks = seq(0, 800, 50)) +
   theme_bw(base_size = 20) +
    ggpubr::rotate_x_text(angle = 90) +
  ggtitle("Fragment lengths for H3K9me3")+
    ylab("Fragment Length") +
    xlab("")

annotated_combo_df %>% 
  dplyr::filter(histone_mark=="H3K27me3") %>% 
  ggplot(., aes(x=Col1, y=Col2, color = sample))+
  geom_line(size=1)+
  # geom_point(size=2,alpha =0.7)+
  scale_x_continuous(breaks = seq(0, max(annotated_combo_df$Col1), by = 50))+
   facet_wrap(~sample)+
  labs(title = "Fragment length, H3K27me3",
       x = "Fragment Length (bp)",
       y = "Count",
       color= "Sample")+
  theme_minimal()
annotated_combo_df %>% 
  dplyr::filter(histone_mark=="H3K27me3") %>%
  ggplot(., aes(x=sample, y=Col1, weight = weight,fill = histone_mark))+
geom_violin(bw = 5) +
    scale_y_continuous(breaks = seq(0, 800, 50)) +
   theme_bw(base_size = 20) +
    ggpubr::rotate_x_text(angle = 90) +
  ggtitle("Fragment lengths for H3K27me3")+
    ylab("Fragment Length") +
    xlab("")

annotated_combo_df %>% 
  dplyr::filter(histone_mark=="H3K27ac") %>% 
  ggplot(., aes(x=Col1, y=Col2, color = sample))+
  geom_line(size=1)+
  # geom_point(size=2,alpha =0.7)+
  scale_x_continuous(breaks = seq(0, max(annotated_combo_df$Col1), by = 50))+
   facet_wrap(~sample)+
  labs(title = "Fragment length, H3K27ac",
       x = "Fragment Length (bp)",
       y = "Count",
       color= "Sample")+
  theme_minimal()
annotated_combo_df %>% 
  dplyr::filter(histone_mark=="H3K27ac") %>%
  ggplot(., aes(x=sample, y=Col1, weight = weight,fill = histone_mark))+
geom_violin(bw = 5) +
    scale_y_continuous(breaks = seq(0, 800, 50)) +
   theme_bw(base_size = 20) +
    ggpubr::rotate_x_text(angle = 90) +
  ggtitle("Fragment lengths for H3K27ac")+
    ylab("Fragment Length") +
    xlab("")
annotated_combo_df %>% 
  dplyr::filter(histone_mark=="H3K36me3") %>% 
  ggplot(., aes(x=Col1, y=Col2, color = sample))+
  geom_line(size=1)+
  # geom_point(size=2,alpha =0.7)+
  scale_x_continuous(breaks = seq(0, max(annotated_combo_df$Col1), by = 50))+
  facet_wrap(~sample)+
  labs(title = "Fragment length, H3K36me3",
       x = "Fragment Length (bp)",
       y = "Count",
       color= "Sample")+
  theme_minimal()

annotated_combo_df %>% 
  dplyr::filter(histone_mark=="H3K36me3") %>%
  ggplot(., aes(x=sample, y=Col1, weight = weight,fill = histone_mark))+
geom_violin(bw = 5) +
    scale_y_continuous(breaks = seq(0, 800, 50)) +
   theme_bw(base_size = 20) +
    ggpubr::rotate_x_text(angle = 90) +
  ggtitle("Fragment lengths for H3K36me3")+
    ylab("Fragment Length") +
    xlab("")
```
#### Peak count across files

```{r peakcount, fig.width=12}
peak_count_H3K27ac <- read_table("C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K27ac/macs2_output/H3K27ac_peak_count.txt", 
    col_names = FALSE) %>% 
  dplyr::filter(X2 != "total") %>% 
   mutate(X2=gsub("_peaks.narrowPeak","",X2)) %>% 
    mutate(histone="H3K27ac")

peak_count_H3K27me3 <- read_table("C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K27me3/macs2_output/H3K27me3_peak_count.txt", 
    col_names = FALSE)%>% 
  dplyr::filter(X2 != "total") %>% 
   mutate(X2=gsub("_peaks.broadPeak","",X2)) %>% 
   dplyr::filter(X2 != "H3K27me3_all") %>% 
  mutate(histone="H3K27me3")


peak_count_H3K9me3 <- read_table("C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K9me3/macs2_output/H3K9me3_peak_count.txt", 
    col_names = FALSE) %>% 
  dplyr::filter(X2 != "total") %>% 
   mutate(X2=gsub("_peaks.narrowPeak","",X2)) %>% 
    mutate(histone="H3K9me3")



peak_count_H3K36me3 <- read_table("C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K36me3/macs2_output/H3K36me3_peak_count.txt", 
    col_names = FALSE) %>% 
  dplyr::filter(X2 != "total") %>% 
  mutate(X2=gsub("_peaks.broadPeak","",X2)) %>% 
   dplyr::filter(X2 != "H3K36me3_all") %>% 
  mutate(histone="H3K36me3")
 


All_peak_count <- rbind(peak_count_H3K27ac,peak_count_H3K27me3,peak_count_H3K9me3,peak_count_H3K36me3)

All_peak_count %>% 
   ggplot(.,aes(x=X2, y=X1,fill=histone))+
   geom_col()+
   ylab("Count")+
   theme_classic()+
  # facet_wrap(~histone)+
  ggtitle("Peak number for all samples")+ 
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))

All_peak_count %>% 
  ggplot(., aes (x=histone, y = X1, fill = histone))+
  geom_boxplot()+
   ylab("Count")+
   theme_classic()+
  # facet_wrap(~histone)+
  ggtitle("Peak count across histones")

All_peak_count %>% 
  left_join(sampleinfo, by = c("X2"="library_ID", "histone"="histone_mark")) %>% 
  ggplot(., aes (x=trt, y = X1, fill = histone))+
  geom_boxplot()+
   ylab("Count")+
   theme_classic()+
  # facet_wrap(~histone)+
  ggtitle("Peak count across histones by trt")

All_peak_count %>% 
  left_join(sampleinfo, by = c("X2"="library_ID", "histone"="histone_mark")) %>% 
  ggplot(., aes (x=other_time, y = X1, fill = histone))+
  geom_boxplot()+
   ylab("Count")+
   theme_classic()+
  # facet_wrap(~histone)+
  ggtitle("Peak count across histones by time")


```

### H3K27ac analysis

```{r H3K27ac counts}
count_H3K27ac <- read_delim("C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K27ac/count_H3K27ac.txt", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)

noM_count_H3K27ac <- read_delim("C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K27ac/noM_count_H3K27ac", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)

Fragment_H3K27ac_txt <- read_delim("C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K27ac/count_H3K27ac.txt.summary", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE)

Fragment_H3K27ac_txt_noM <- read_delim("C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K27ac/noM_count_H3K27ac.summary", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE)
rename_list <- sampleinfo %>% 
  mutate(stem= "_final.bam") %>% 
  mutate(oldname=paste0(library_ID,stem)) %>% 
  mutate(newname=paste0(ind,"_",trt,"_",other_time,"_",histone_mark)) %>% 
  dplyr::select(oldname,newname)

noM_rename_list <- sampleinfo %>% 
  mutate(stem= "_noM_final.bam") %>% 
  mutate(oldname=paste0(library_ID,stem)) %>% 
  mutate(newname=paste0(ind,"_",trt,"_",other_time,"_",histone_mark)) %>% 
  dplyr::select(oldname,newname)
  
rename_vec <- setNames(rename_list$newname, rename_list$oldname)

rename_vec_noM <- setNames(noM_rename_list$newname, noM_rename_list$oldname)

names(count_H3K27ac)[names(count_H3K27ac) %in% names(rename_vec)] <- rename_vec[names(count_H3K27ac)[names(count_H3K27ac) %in% names(rename_vec)]]

names(noM_count_H3K27ac)[names(noM_count_H3K27ac) %in% names(rename_vec_noM)] <- rename_vec_noM[names(noM_count_H3K27ac)[names(noM_count_H3K27ac) %in% names(rename_vec_noM)]]


```


```{r, fig.width=10, fig.height=7}
H3K27ac_counts_raw <- count_H3K27ac %>% 
  dplyr::select(Geneid,`3_VEH_24r_H3K27ac`:`3_DOX_24r_H3K27ac`) %>% 
  column_to_rownames("Geneid") %>% 
  as.matrix()
H3K27ac_counts_raw_noM <- noM_count_H3K27ac %>% 
  dplyr::select(Geneid,`3_VEH_24r_H3K27ac`:`3_DOX_24r_H3K27ac`) %>% 
  column_to_rownames("Geneid") %>% 
  as.matrix()


first_H3K27ac_counts_cor <- count_H3K27ac %>% 
  dplyr::select(Geneid,`3_VEH_24r_H3K27ac`:`3_DOX_24r_H3K27ac`) %>% 
  column_to_rownames("Geneid") %>% 
  cpm(., log = TRUE) %>% 
  cor()

first_H3K27ac_counts_cor_noM <- noM_count_H3K27ac %>% 
  dplyr::select(Geneid,`3_VEH_24r_H3K27ac`:`3_DOX_24r_H3K27ac`) %>% 
  column_to_rownames("Geneid") %>% 
  cpm(., log = TRUE) %>% 
  cor()

annomat <- data.frame(sample=colnames(first_H3K27ac_counts_cor)) %>% 
  separate_wider_delim(sample,delim="_",names=c("ind","trt","time",NA),cols_remove = FALSE) %>% 
  mutate(trt=factor(trt, levels = c("VEH","5FU","DOX")),
         time=factor(time, levels =c("24t","24r","144r"))) %>% 
  column_to_rownames("sample")
heatmap_first <- ComplexHeatmap::HeatmapAnnotation(df = annomat)

Heatmap(first_H3K27ac_counts_cor, 
        top_annotation = heatmap_first,
        column_title="Unfiltered log2cpm H3K27ac")

Heatmap(first_H3K27ac_counts_cor_noM, 
        top_annotation = heatmap_first,
        column_title="Unfiltered log2cpm H3K27ac_noM")
```

### H3K27me3 analysis


```{r H3k27me3 import}
count_H3K27me3 <- read_delim("C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K27me3/H3K27me3_counts.txt", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)

noM_count_H3K27me3 <- read_delim("C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K27me3/noM_count_H3K27me3", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)


Fragment_H3K27me3_txt <- read_delim("C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K27me3/H3K27me3_counts.txt.summary", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE)

Fragment_H3K27me3_txt_noM <- read_delim("C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K27me3/noM_count_H3K27me3.summary", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE)
rename_list <- sampleinfo %>% 
  mutate(stem= "_final.bam") %>% 
  mutate(oldname=paste0(library_ID,stem)) %>% 
  mutate(newname=paste0(ind,"_",trt,"_",other_time,"_",histone_mark)) %>% 
  dplyr::select(oldname,newname)
  
rename_vec <- setNames(rename_list$newname, rename_list$oldname)

names(count_H3K27me3)[names(count_H3K27me3) %in% names(rename_vec)] <- rename_vec[names(count_H3K27me3)[names(count_H3K27me3) %in% names(rename_vec)]]

names(noM_count_H3K27me3)[names(noM_count_H3K27me3) %in% names(rename_vec_noM)] <- rename_vec_noM[names(noM_count_H3K27me3)[names(noM_count_H3K27me3) %in% names(rename_vec_noM)]]


```



```{r H3K27me3 heatmap,fig.width=10, fig.height=7}
H3K27me3_counts_raw <- count_H3K27me3 %>% 
  dplyr::select(Geneid,`1_VEH_24t_H3K27me3`:`5_VEH_24r_H3K27me3`) %>% 
  column_to_rownames("Geneid") %>% 
  as.matrix()

H3K27me3_counts_raw_noM <- noM_count_H3K27me3 %>% 
  dplyr::select(Geneid,`1_VEH_24t_H3K27me3`:`5_VEH_24r_H3K27me3`) %>% 
  column_to_rownames("Geneid") %>% 
  as.matrix()


first_H3K27me3_counts_cor <- count_H3K27me3 %>% 
  dplyr::select(Geneid,`1_VEH_24t_H3K27me3`:`5_VEH_24r_H3K27me3`) %>% 
  column_to_rownames("Geneid") %>% 
  cpm(., log = TRUE) %>% 
  cor()
first_H3K27me3_counts_cor_noM <- noM_count_H3K27me3 %>% 
  dplyr::select(Geneid,`1_VEH_24t_H3K27me3`:`5_VEH_24r_H3K27me3`) %>% 
  column_to_rownames("Geneid") %>% 
  cpm(., log = TRUE) %>% 
  cor()

annomat_H3K27me3 <- data.frame(sample=colnames(first_H3K27me3_counts_cor)) %>% 
  separate_wider_delim(sample,delim="_",names=c("ind","trt","time",NA),cols_remove = FALSE) %>% 
  mutate(trt=factor(trt, levels = c("VEH","5FU","DOX")),
         time=factor(time, levels =c("24t","24r","144r"))) %>% 
  column_to_rownames("sample")
heatmap_first_H3K27me3 <- ComplexHeatmap::HeatmapAnnotation(df = annomat_H3K27me3)

Heatmap(first_H3K27me3_counts_cor, 
        top_annotation = heatmap_first_H3K27me3,
        column_title="Unfiltered log2cpm H3K27me3")

Heatmap(first_H3K27me3_counts_cor_noM, 
        top_annotation = heatmap_first_H3K27me3,
        column_title="Unfiltered log2cpm H3K27me3 no mitoreads")
```






### H3K9me3 analysis

```{r}
count_H3K9me3 <- read_delim("C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K9me3/H3K9me3_counts.txt", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)

noM_count_H3K9me3 <- read_delim("C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K9me3/noM_count_H3K9me3", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)


Fragment_H3K9me3_txt <- read_delim("C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K9me3/H3K9me3_counts.txt.summary", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE)

Fragment_H3K9me3_txt_noM <- read_delim("C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K9me3/noM_count_H3K9me3.summary", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE)
rename_list <- sampleinfo %>% 
  mutate(stem= "_final.bam") %>% 
  mutate(oldname=paste0(library_ID,stem)) %>% 
  mutate(newname=paste0(ind,"_",trt,"_",other_time,"_",histone_mark)) %>% 
  dplyr::select(oldname,newname)
  
rename_vec <- setNames(rename_list$newname, rename_list$oldname)

names(count_H3K9me3)[names(count_H3K9me3) %in% names(rename_vec)] <- rename_vec[names(count_H3K9me3)[names(count_H3K9me3) %in% names(rename_vec)]]
names(noM_count_H3K9me3)[names(noM_count_H3K9me3) %in% names(rename_vec_noM)] <- rename_vec_noM[names(noM_count_H3K9me3)[names(noM_count_H3K9me3) %in% names(rename_vec_noM)]]

```




```{r H3K9me3 heatmap,fig.width=10, fig.height=7}
H3K9me3_counts_raw <- count_H3K9me3 %>% 
  dplyr::select(Geneid,`1_DOX_24t_H3K9me3`:`5_5FU_144r_H3K9me3`) %>% 
  column_to_rownames("Geneid") %>% 
  as.matrix()

H3K9me3_counts_raw <- noM_count_H3K9me3 %>% 
  dplyr::select(Geneid,`1_DOX_24t_H3K9me3`:`5_5FU_144r_H3K9me3`) %>% 
  column_to_rownames("Geneid") %>% 
  as.matrix()



first_H3K9me3_counts_cor <- count_H3K9me3 %>% 
  dplyr::select(Geneid,`1_DOX_24t_H3K9me3`:`5_5FU_144r_H3K9me3`) %>% 
  column_to_rownames("Geneid") %>% 
  cpm(., log = TRUE) %>% 
  cor()
first_H3K9me3_counts_cor_noM <- noM_count_H3K9me3 %>% 
  dplyr::select(Geneid,`1_DOX_24t_H3K9me3`:`5_5FU_144r_H3K9me3`) %>% 
  column_to_rownames("Geneid") %>% 
  cpm(., log = TRUE) %>% 
  cor()

annomat_H3K9me3 <- data.frame(sample=colnames(first_H3K9me3_counts_cor)) %>% 
  separate_wider_delim(sample,delim="_",names=c("ind","trt","time",NA),cols_remove = FALSE) %>% 
  mutate(trt=factor(trt, levels = c("VEH","5FU","DOX")),
         time=factor(time, levels =c("24t","24r","144r"))) %>% 
  column_to_rownames("sample")
heatmap_first_H3K9me3 <- ComplexHeatmap::HeatmapAnnotation(df = annomat_H3K9me3)

Heatmap(first_H3K9me3_counts_cor, 
        top_annotation = heatmap_first_H3K9me3,
        column_title="Unfiltered log2cpm H3K9me3")


Heatmap(first_H3K9me3_counts_cor_noM, 
        top_annotation = heatmap_first_H3K9me3,
        column_title="Unfiltered log2cpm H3K9me3 without mito reads")
```




### H3K36me3 analysis

```{r}
count_H3K36me3 <- read_delim("C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K36me3/H3K36me3_counts.txt", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)
noM_count_H3K36me3 <- read_delim("C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K36me3/noM_count_H3K36me3", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE, skip = 1)

Fragment_H3K36me3_txt <- read_delim("C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K36me3/H3K36me3_counts.txt.summary", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE)

Fragment_H3K36me3_txt_noM <- read_delim("C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K36me3/noM_count_H3K36me3.summary", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE)
rename_list <- sampleinfo %>% 
  mutate(stem= "_final.bam") %>% 
  mutate(oldname=paste0(library_ID,stem)) %>% 
  mutate(newname=paste0(ind,"_",trt,"_",other_time,"_",histone_mark)) %>% 
  dplyr::select(oldname,newname)
  
rename_vec <- setNames(rename_list$newname, rename_list$oldname)

names(count_H3K36me3)[names(count_H3K36me3) %in% names(rename_vec)] <- rename_vec[names(count_H3K36me3)[names(count_H3K36me3) %in% names(rename_vec)]]
names(noM_count_H3K36me3)[names(noM_count_H3K36me3) %in% names(rename_vec_noM)] <- rename_vec_noM[names(noM_count_H3K36me3)[names(noM_count_H3K36me3) %in% names(rename_vec_noM)]]

```




```{r H3K36me3 heatmap,fig.width=10, fig.height=7}
H3K36me3_counts_raw <- count_H3K36me3 %>% 
  dplyr::select(Geneid,`1_5FU_24t_H3K36me3`:`5_VEH_144r_H3K36me3`) %>% 
  column_to_rownames("Geneid") %>% 
  as.matrix()
H3K36me3_counts_raw_noM <- noM_count_H3K36me3 %>% 
  dplyr::select(Geneid,`1_5FU_24t_H3K36me3`:`5_VEH_144r_H3K36me3`) %>% 
  column_to_rownames("Geneid") %>% 
  as.matrix()


first_H3K36me3_counts_cor <- count_H3K36me3 %>% 
  dplyr::select(Geneid,`1_5FU_24t_H3K36me3`:`5_VEH_144r_H3K36me3`) %>% 
  column_to_rownames("Geneid") %>% 
  cpm(., log = TRUE) %>% 
  cor()

first_H3K36me3_counts_cor_noM <- noM_count_H3K36me3 %>% 
  dplyr::select(Geneid,`1_5FU_24t_H3K36me3`:`5_VEH_144r_H3K36me3`) %>% 
  column_to_rownames("Geneid") %>% 
  cpm(., log = TRUE) %>% 
  cor()

annomat_H3K36me3 <- data.frame(sample=colnames(first_H3K36me3_counts_cor)) %>% 
  separate_wider_delim(sample,delim="_",names=c("ind","trt","time",NA),cols_remove = FALSE) %>% 
  mutate(trt=factor(trt, levels = c("VEH","5FU","DOX")),
         time=factor(time, levels =c("24t","24r","144r"))) %>% 
  column_to_rownames("sample")
heatmap_first_H3K36me3 <- ComplexHeatmap::HeatmapAnnotation(df = annomat_H3K36me3)

Heatmap(first_H3K36me3_counts_cor, 
        top_annotation = heatmap_first_H3K36me3,
        column_title="Unfiltered log2cpm H3K36me3")

Heatmap(first_H3K36me3_counts_cor_noM, 
        top_annotation = heatmap_first_H3K36me3,
        column_title="Unfiltered log2cpm H3K36me3 without mito reads")
```



#### Fragment Width

```{r Frag width}

histone_list <- c("H3K27ac", "H3K27me3", "H3K9me3", "H3K36me3")
base_path <- "C:/Users/renee/Other_projects_data/5FU_data/prelim-data"

get_peak_widths_long_split <- function(histone_name, base_path) {
  path <- file.path(base_path, histone_name, "macs2_output")
  
  file_list <- list.files(path = path, pattern = "\\.(narrowPeak|broadPeak)$", full.names = TRUE)
  
  # Exclude files with 'all_peaks' in the name
  file_list <- file_list[!grepl("all_peaks", basename(file_list), ignore.case = TRUE)]
  
  # Split into noM and not noM groups
  noM_files <- file_list[grepl("noM", basename(file_list))]
  reg_files <- file_list[!grepl("noM", basename(file_list))]

  # Define internal peak processing function
  process_peaks <- function(file_path, group_label) {
    filename <- basename(file_path)
    peak_df <- fread(file_path, header = FALSE) %>%
      transmute(
        row = row_number(),
        width = abs(V3 - V2),
        file = paste0(histone_name, "-", filename),
        group = group_label
      )
    return(peak_df)
  }

  # Process each group
  peak_list <- list(
    map(noM_files, process_peaks, group_label = "noM"),
    map(reg_files, process_peaks, group_label = "regular")
  ) %>% flatten()

  bind_rows(peak_list)
}


```

```{r fragment visulization}

# To apply to all histones:
all_peak_widths <- purrr::map_df(histone_list, get_peak_widths_long_split, base_path = base_path)


anno_peak_width_long <- all_peak_widths %>% 
  dplyr::filter(group!="noM") %>% 
    mutate(file= gsub("_noM_peaks.narrowPeak","",file)) %>% 
    mutate(file= gsub("_noM_peaks.broadPeak","",file)) %>% 
  separate_wider_delim(., cols=file, delim="-",names=c("histone","sample")) %>% 
  left_join(sampleinfo, by =c("sample"="library_ID"))

anno_peak_width_long_noM <-
  all_peak_widths %>% 
  dplyr::filter(group=="noM") %>% 
    mutate(file= gsub("_noM_peaks.narrowPeak","",file)) %>% 
    mutate(file= gsub("_noM_peaks.broadPeak","",file)) %>% 
  separate_wider_delim(., cols=file, delim="-",names=c("histone","sample")) %>% 
  left_join(sampleinfo, by =c("sample"="library_ID"))

```

```{r}
anno_peak_width_long %>%
  # distinct(histone)
  ggplot(.,aes(x=histone, y = width, fill = histone))+
  geom_violin()+
  # scale_fill_viridis_a(discrete = TRUE, begin = 0.1, end = 0.55, option = "magma", alpha = 0.8) +
  #   scale_color_viridis_a(discrete = TRUE, begin = 0.1, end = 0.9) +
    scale_y_continuous(trans = "log", breaks = c(400, 3000, 22000)) +
    theme_bw(base_size = 18) +
    ylab("Width of Peaks") +
    xlab("")+
  ggtitle("Width of all peaks with MT reads")


anno_peak_width_long_noM %>%
  # distinct(histone)
  ggplot(.,aes(x=histone, y = width, fill = histone))+
  geom_violin()+
  # scale_fill_viridis_a(discrete = TRUE, begin = 0.1, end = 0.55, option = "magma", alpha = 0.8) +
  #   scale_color_viridis_a(discrete = TRUE, begin = 0.1, end = 0.9) +
    scale_y_continuous(trans = "log", breaks = c(400, 3000, 22000)) +
    theme_bw(base_size = 18) +
    ylab("Width of Peaks") +
    xlab("")+
  ggtitle("Width of all peaks without MT reads")
```
  The smaller fragment bits, specifically in H3K26 and K3k9me3 made me realize something was up.  After visualizing the top ranked peaks in some of the peak files by sample, I became aware of chrM contamination.  Below this mark is me trying to remap/and filter out the non-standard chromosomes.
  
  Code I used to check top peaks on files: 
```{bash echo=TRUE, eval=FALSE}
sort -k5,5rn MCW_CT_ROA_215_peaks.narrowPeak | head -n 30

```
```{r peakwidth full filtered peaks}
Fragment_H3K36me3_txt %>% 
  pivot_longer(., cols=!Status, names_to = "sample", values_to ="count") %>% 
  pivot_wider(., id_cols = sample, names_from = Status, values_from = count) %>% 
  rowwise() %>% 
  mutate(total = sum(c_across(Assigned:Unassigned_Ambiguity))) %>% 
  dplyr::select(sample,Assigned, Unassigned_NoFeatures,total) %>% 
  mutate(perc_assign= sprintf("%.2f",(Assigned/total)*100)) %>% 
  mutate(sample= gsub("_final.bam","",sample)) %>% 
  left_join(., sampleinfo, by = c("sample"="library_ID")) %>% 
  mutate(other_time=factor(other_time, levels = c ("24t", "24r", "144r"))) %>% 
  mutate(trt=factor(trt, levels = c("VEH","DOX","5FU"))) %>% 
  mutate(perc_assign=as.numeric(perc_assign)) %>% 
  ggplot(., aes(x=interaction(ind,trt,other_time), y=perc_assign, fill=trt, group = trt))+
  geom_col()+
  geom_text(aes(y = 25,label = sample), vjust = 0.2, size = 3, angle = 90)+

   theme_classic()+
  ggtitle("FRIP for H3K36me3")+
  ylab("Fraction of reads in peaks * 100")+
  xlab("samples")+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))


##############################noM
Fragment_H3K36me3_txt_noM %>% 
  pivot_longer(., cols=!Status, names_to = "sample", values_to ="count") %>% 
  pivot_wider(., id_cols = sample, names_from = Status, values_from = count) %>% 
  rowwise() %>% 
  mutate(total = sum(c_across(Assigned:Unassigned_Ambiguity))) %>% 
  dplyr::select(sample,Assigned, Unassigned_NoFeatures,total) %>% 
  mutate(perc_assign= sprintf("%.2f",(Assigned/total)*100)) %>% 
  mutate(sample= gsub("_noM_final.bam","",sample)) %>% 
  left_join(., sampleinfo, by = c("sample"="library_ID")) %>% 
  mutate(other_time=factor(other_time, levels = c ("24t", "24r", "144r"))) %>% 
  mutate(trt=factor(trt, levels = c("VEH","DOX","5FU"))) %>% 
  mutate(perc_assign=as.numeric(perc_assign)) %>% 
  ggplot(., aes(x=interaction(ind,trt,other_time), y=perc_assign, fill=trt, group = trt))+
  geom_col()+
  geom_text(aes(y = 25,label = sample), vjust = 0.2, size = 3, angle = 90)+

   theme_classic()+
  ggtitle("FRIP for H3K36me3 without mt reads")+
  ylab("Fraction of reads in peaks * 100")+
  xlab("samples")+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))
```

```{r}
Fragment_H3K9me3_txt %>% 
  pivot_longer(., cols=!Status, names_to = "sample", values_to ="count") %>% 
  pivot_wider(., id_cols = sample, names_from = Status, values_from = count) %>% 
  rowwise() %>% 
  mutate(total = sum(c_across(Assigned:Unassigned_Ambiguity))) %>% 
  dplyr::select(sample,Assigned, Unassigned_NoFeatures,total) %>% 
  mutate(perc_assign= sprintf("%.2f",(Assigned/total)*100)) %>% 
  mutate(sample= gsub("_final.bam","",sample)) %>% 
  left_join(., sampleinfo, by = c("sample"="library_ID")) %>% 
  mutate(other_time=factor(other_time, levels = c ("24t", "24r", "144r"))) %>% 
  mutate(trt=factor(trt, levels = c("VEH","DOX","5FU"))) %>% 
  mutate(perc_assign=as.numeric(perc_assign)) %>% 
  ggplot(., aes(x=interaction(ind,trt,other_time), y=perc_assign, fill=trt, group = trt))+
  geom_col()+
  geom_text(aes(y = 5,label = sample), vjust = 0.2, size = 3, angle = 90)+
   theme_classic()+
  ggtitle("FRIP for H3K9me3")+ 
  ylab("Fraction of reads in peaks * 100")+
  xlab("samples")+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))

###################################noM###########
Fragment_H3K9me3_txt_noM %>% 
  pivot_longer(., cols=!Status, names_to = "sample", values_to ="count") %>% 
  pivot_wider(., id_cols = sample, names_from = Status, values_from = count) %>% 
  rowwise() %>% 
  mutate(total = sum(c_across(Assigned:Unassigned_Ambiguity))) %>% 
  dplyr::select(sample,Assigned, Unassigned_NoFeatures,total) %>% 
  mutate(perc_assign= sprintf("%.2f",(Assigned/total)*100)) %>% 
  mutate(sample= gsub("_noM_final.bam","",sample)) %>% 
  left_join(., sampleinfo, by = c("sample"="library_ID")) %>% 
  mutate(other_time=factor(other_time, levels = c ("24t", "24r", "144r"))) %>% 
  mutate(trt=factor(trt, levels = c("VEH","DOX","5FU"))) %>% 
  mutate(perc_assign=as.numeric(perc_assign)) %>% 
  ggplot(., aes(x=interaction(ind,trt,other_time), y=perc_assign, fill=trt, group = trt))+
  geom_col()+
  geom_text(aes(y = 5,label = sample), vjust = 0.2, size = 3, angle = 90)+
   theme_classic()+
  ggtitle("FRIP for H3K9me3 without mt reads")+ 
  ylab("Fraction of reads in peaks * 100")+
  xlab("samples")+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))
```
```{r frip H3K27me3}
Fragment_H3K27me3_txt %>% 
  pivot_longer(., cols=!Status, names_to = "sample", values_to ="count") %>% 
  pivot_wider(., id_cols = sample, names_from = Status, values_from = count) %>% 
  rowwise() %>% 
  mutate(total = sum(c_across(Assigned:Unassigned_Ambiguity))) %>% 
  dplyr::select(sample,Assigned, Unassigned_NoFeatures,total) %>% 
  mutate(perc_assign= sprintf("%.2f",(Assigned/total)*100)) %>% 
  mutate(sample= gsub("_final.bam","",sample)) %>% 
  left_join(., sampleinfo, by = c("sample"="library_ID")) %>% 
  mutate(other_time=factor(other_time, levels = c ("24t", "24r", "144r"))) %>% 
  mutate(trt=factor(trt, levels = c("VEH","DOX","5FU"))) %>% 
  mutate(perc_assign=as.numeric(perc_assign)) %>% 
  ggplot(., aes(x=interaction(ind,trt,other_time), y=perc_assign, fill=trt, group = trt))+
  geom_col()+
  geom_text(aes(y = 25,label = sample), vjust = 0.2, size = 3, angle = 90)+
   theme_classic()+
  ylab("Fraction of reads in peaks * 100")+
  xlab("samples")+
  ggtitle("FRIP for H3K27me3")+ theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))
######################noM################################3
Fragment_H3K27me3_txt_noM %>% 
  pivot_longer(., cols=!Status, names_to = "sample", values_to ="count") %>% 
  pivot_wider(., id_cols = sample, names_from = Status, values_from = count) %>% 
  rowwise() %>% 
  mutate(total = sum(c_across(Assigned:Unassigned_Ambiguity))) %>% 
  dplyr::select(sample,Assigned, Unassigned_NoFeatures,total) %>% 
  mutate(perc_assign= sprintf("%.2f",(Assigned/total)*100)) %>% 
  mutate(sample= gsub("_noM_final.bam","",sample)) %>% 
  left_join(., sampleinfo, by = c("sample"="library_ID")) %>% 
  mutate(other_time=factor(other_time, levels = c ("24t", "24r", "144r"))) %>% 
  mutate(trt=factor(trt, levels = c("VEH","DOX","5FU"))) %>% 
  mutate(perc_assign=as.numeric(perc_assign)) %>% 
  ggplot(., aes(x=interaction(ind,trt,other_time), y=perc_assign, fill=trt, group = trt))+
  geom_col()+
  geom_text(aes(y = 25,label = sample), vjust = 0.2, size = 3, angle = 90)+
   theme_classic()+
  ylab("Fraction of reads in peaks * 100")+
  xlab("samples")+
  ggtitle("FRIP for H3K27me3 without mt reads")+ theme(axis.text.x=element_text(vjust = .2,angle=90))+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))
```

```{r frip H3K27ac}

Fragment_H3K27ac_txt %>% 
  pivot_longer(., cols=!Status, names_to = "sample", values_to ="count") %>% 
  pivot_wider(., id_cols = sample, names_from = Status, values_from = count) %>% 
  rowwise() %>% 
  mutate(total = sum(c_across(Assigned:Unassigned_Ambiguity))) %>% 
  dplyr::select(sample,Assigned, Unassigned_NoFeatures,total) %>% 
  mutate(perc_assign= sprintf("%.2f",(Assigned/total)*100)) %>% 
  mutate(sample= gsub("_final.bam","",sample)) %>% 
  left_join(., sampleinfo, by = c("sample"="library_ID")) %>% 
  mutate(other_time=factor(other_time, levels = c ("24t", "24r", "144r"))) %>% 
  mutate(trt=factor(trt, levels = c("VEH","DOX","5FU"))) %>% 
  mutate(perc_assign=as.numeric(perc_assign)) %>% 
  ggplot(., aes(x=interaction(ind,trt,other_time), y=perc_assign, fill=trt, group = trt))+
  geom_col()+
  geom_text(aes(y = 15,label = sample), vjust = 0.2, size = 3, angle = 90)+
   theme_classic()+
  ggtitle("FRIP for H3K27ac")+ theme(axis.text.x=element_text(vjust = .2,angle=90))+
  ylab("Fraction of reads in peaks * 100")+
  xlab("samples")+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))

#######################noM################################
Fragment_H3K27ac_txt_noM %>% 
  pivot_longer(., cols=!Status, names_to = "sample", values_to ="count") %>% 
  pivot_wider(., id_cols = sample, names_from = Status, values_from = count) %>% 
  rowwise() %>% 
  mutate(total = sum(c_across(Assigned:Unassigned_Ambiguity))) %>% 
  dplyr::select(sample,Assigned, Unassigned_NoFeatures,total) %>% 
  mutate(perc_assign= sprintf("%.2f",(Assigned/total)*100)) %>% 
  mutate(sample= gsub("_noM_final.bam","",sample)) %>% 
  left_join(., sampleinfo, by = c("sample"="library_ID")) %>% 
  mutate(other_time=factor(other_time, levels = c ("24t", "24r", "144r"))) %>% 
  mutate(trt=factor(trt, levels = c("VEH","DOX","5FU"))) %>% 
  mutate(perc_assign=as.numeric(perc_assign)) %>% 
  ggplot(., aes(x=interaction(ind,trt,other_time), y=perc_assign, fill=trt, group = trt))+
  geom_col()+
  geom_text(aes(y = 15,label = sample), vjust = 0.2, size = 3, angle = 90)+
   theme_classic()+
  ggtitle("FRIP for H3K27ac without mt reads")+ theme(axis.text.x=element_text(vjust = .2,angle=90))+
  ylab("Fraction of reads in peaks * 100")+
  xlab("samples")+
  scale_y_continuous( expand = expansion(mult = c(0, .1)))

```
### Peak width of master peaks
```{r Width of master peaks}
H3K27ac_masterpeak <- readBed(file="C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K27ac/macs2_output/filt_H3k27ac_all_merged.bed", remove.unusual = FALSE)

# H3K27ac_masterpeak %>% 
#          as.data.frame() %>% mutate(standard= if_else(seqnames %in% standard_chroms, "yes","no")) %>% group_by(standard) %>% tally


H3K27me3_masterpeak <- readBed(file="C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K27me3/macs2_output/filt_H3k27me3_all_merged.bed", remove.unusual = FALSE)
       # H3K27me3_masterpeak %>% 
       #   as.data.frame() %>% mutate(standard= if_else(seqnames %in% standard_chroms, "yes","no")) %>% group_by(standard) %>% tally
       
H3K9me3_masterpeak <- readBed(file="C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K9me3/macs2_output/filt_H3k9me3_all_merged.bed", remove.unusual = FALSE)
       # H3K9me3_masterpeak %>% 
       #   as.data.frame() %>% mutate(standard= if_else(seqnames %in% standard_chroms, "yes","no")) %>% group_by(standard) %>% tally  
       
H3K36me3_masterpeak <- readBed(file="C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K36me3/macs2_output/filt_H3k36me3_all_merged.bed", remove.unusual = FALSE)
       # H3K36me3_masterpeak %>% 
       #   as.data.frame() %>% mutate(standard= if_else(seqnames %in% standard_chroms, "yes","no")) %>% group_by(standard) %>% tally         
       


master_peak_width <- data_frame(histone_width=width(H3K27me3_masterpeak),histone="H3K27me3") %>% 
  rbind(data_frame(histone_width=width(H3K27ac_masterpeak),histone="H3K27ac")) %>% 
  rbind(data_frame(histone_width=width(H3K9me3_masterpeak),histone="H3K9me3")) %>% 
  rbind(data_frame(histone_width=width(H3K36me3_masterpeak), histone = "H3K36me3"))

master_peak_width %>% 
  ggplot(.,aes(x=histone, y = histone_width, fill = histone))+
  geom_violin()+
    scale_y_continuous(trans = "log", breaks = c(400, 3000, 22000)) +
    theme_bw(base_size = 18) +
    ylab("Width of Peaks") +
    xlab("")+
  ggtitle("Master peak list widths with mt reads")
  
```
```{r}
H3K27ac_masterpeak_noM <- readBed(file="C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K27ac/macs2_output/filt_H3k27ac_noM_all_merged.bed", remove.unusual = FALSE)

H3K27me3_masterpeak_noM <- readBed(file="C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K27me3/macs2_output/filt_H3k27me3_noM_all_merged.bed", remove.unusual = FALSE)
       
H3K9me3_masterpeak_noM <- readBed(file="C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K9me3/macs2_output/filt_H3k9me3_noM_all_merged.bed", remove.unusual = FALSE)
       
       
H3K36me3_masterpeak_noM <- readBed(file="C:/Users/renee/Other_projects_data/5FU_data/prelim-data/H3K36me3/macs2_output/filt_H3k36me3_noM_all_merged.bed", remove.unusual = FALSE)
      


master_peak_width_noM <- data_frame(histone_width=width(H3K27me3_masterpeak_noM),histone="H3K27me3") %>% 
  rbind(data_frame(histone_width=width(H3K27ac_masterpeak_noM),histone="H3K27ac")) %>% 
  rbind(data_frame(histone_width=width(H3K9me3_masterpeak_noM),histone="H3K9me3")) %>% 
  rbind(data_frame(histone_width=width(H3K36me3_masterpeak_noM), histone = "H3K36me3"))

master_peak_width_noM %>% 
  ggplot(.,aes(x=histone, y = histone_width, fill = histone))+
  geom_violin()+
    scale_y_continuous(trans = "log", breaks = c(400, 3000, 22000)) +
    theme_bw(base_size = 18) +
    ylab("Width of Peaks") +
    xlab("")+
  ggtitle("Master Peak list widths without MT reads")
  
```
